{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/mvapontes/KEGGAPI.jl/blob/main/examples/Case1.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "gEUClGc1JTAY"
      },
      "source": [
        "# <img src=\"https://github.com/JuliaLang/julia-logo-graphics/raw/master/images/julia-logo-color.png\" height=\"100\" /> _Colab Notebook Template_\n",
        "\n",
        "## Instructions\n",
        "1. Work on a copy of this notebook: _File_ > _Save a copy in Drive_ (you will need a Google account). Alternatively, you can download the notebook using _File_ > _Download .ipynb_, then upload it to [Colab](https://colab.research.google.com/).\n",
        "2. If you need a GPU: _Runtime_ > _Change runtime type_ > _Hardware accelerator_ = _GPU_.\n",
        "3. Execute the following cell (click on it and press Ctrl+Enter) to install Julia, IJulia and other packages (if needed, update `JULIA_VERSION` and the other parameters). This takes a couple of minutes.\n",
        "4. Reload this page (press Ctrl+R, or ⌘+R, or the F5 key) and continue to the next section.\n",
        "\n",
        "_Notes_:\n",
        "* If your Colab Runtime gets reset (e.g., due to inactivity), repeat steps 2, 3 and 4.\n",
        "* After installation, if you want to change the Julia version or activate/deactivate the GPU, you will need to reset the Runtime: _Runtime_ > _Factory reset runtime_ and repeat steps 3 and 4."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ZC527yn-JRaI",
        "outputId": "5b7d9e6f-dbfe-41ac-e211-9de2cc568e5f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Installing Julia 1.8.2 on the current Colab Runtime...\n",
            "2023-08-04 08:46:10 URL:https://storage.googleapis.com/julialang2/bin/linux/x64/1.8/julia-1.8.2-linux-x86_64.tar.gz [135859273/135859273] -> \"/tmp/julia.tar.gz\" [1]\n",
            "Installing Julia package IJulia...\n",
            "Installing Julia package BenchmarkTools...\n",
            "Installing IJulia kernel...\n",
            "\u001b[36m\u001b[1m[ \u001b[22m\u001b[39m\u001b[36m\u001b[1mInfo: \u001b[22m\u001b[39mInstalling julia kernelspec in /root/.local/share/jupyter/kernels/julia-1.8\n",
            "\n",
            "Successfully installed julia version 1.8.2!\n",
            "Please reload this page (press Ctrl+R, ⌘+R, or the F5 key) then\n",
            "jump to the 'Checking the Installation' section.\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": []
          },
          "metadata": {},
          "execution_count": 1
        }
      ],
      "source": [
        "%%shell\n",
        "set -e\n",
        "\n",
        "#---------------------------------------------------#\n",
        "JULIA_VERSION=\"1.8.2\" # any version ≥ 0.7.0\n",
        "JULIA_PACKAGES=\"IJulia BenchmarkTools\"\n",
        "JULIA_PACKAGES_IF_GPU=\"CUDA\" # or CuArrays for older Julia versions\n",
        "JULIA_NUM_THREADS=2\n",
        "#---------------------------------------------------#\n",
        "\n",
        "if [ -z `which julia` ]; then\n",
        "  # Install Julia\n",
        "  JULIA_VER=`cut -d '.' -f -2 <<< \"$JULIA_VERSION\"`\n",
        "  echo \"Installing Julia $JULIA_VERSION on the current Colab Runtime...\"\n",
        "  BASE_URL=\"https://julialang-s3.julialang.org/bin/linux/x64\"\n",
        "  URL=\"$BASE_URL/$JULIA_VER/julia-$JULIA_VERSION-linux-x86_64.tar.gz\"\n",
        "  wget -nv $URL -O /tmp/julia.tar.gz # -nv means \"not verbose\"\n",
        "  tar -x -f /tmp/julia.tar.gz -C /usr/local --strip-components 1\n",
        "  rm /tmp/julia.tar.gz\n",
        "\n",
        "  # Install Packages\n",
        "  nvidia-smi -L &> /dev/null && export GPU=1 || export GPU=0\n",
        "  if [ $GPU -eq 1 ]; then\n",
        "    JULIA_PACKAGES=\"$JULIA_PACKAGES $JULIA_PACKAGES_IF_GPU\"\n",
        "  fi\n",
        "  for PKG in `echo $JULIA_PACKAGES`; do\n",
        "    echo \"Installing Julia package $PKG...\"\n",
        "    julia -e 'using Pkg; pkg\"add '$PKG'; precompile;\"' &> /dev/null\n",
        "  done\n",
        "\n",
        "  # Install kernel and rename it to \"julia\"\n",
        "  echo \"Installing IJulia kernel...\"\n",
        "  julia -e 'using IJulia; IJulia.installkernel(\"julia\", env=Dict(\n",
        "      \"JULIA_NUM_THREADS\"=>\"'\"$JULIA_NUM_THREADS\"'\"))'\n",
        "  KERNEL_DIR=`julia -e \"using IJulia; print(IJulia.kerneldir())\"`\n",
        "  KERNEL_NAME=`ls -d \"$KERNEL_DIR\"/julia*`\n",
        "  mv -f $KERNEL_NAME \"$KERNEL_DIR\"/julia\n",
        "\n",
        "  echo ''\n",
        "  echo \"Successfully installed `julia -v`!\"\n",
        "  echo \"Please reload this page (press Ctrl+R, ⌘+R, or the F5 key) then\"\n",
        "  echo \"jump to the 'Checking the Installation' section.\"\n",
        "fi"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "aHkU6NpPJgBi",
        "outputId": "e4408af6-dfb6-47d5-9749-7f63043d16f6"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Julia Version 1.8.2\n",
            "Commit 36034abf260 (2022-09-29 15:21 UTC)\n",
            "Platform Info:\n",
            "  OS: Linux (x86_64-linux-gnu)\n",
            "  CPU: 2 × Intel(R) Xeon(R) CPU @ 2.20GHz\n",
            "  WORD_SIZE: 64\n",
            "  LIBM: libopenlibm\n",
            "  LLVM: libLLVM-13.0.1 (ORCJIT, broadwell)\n",
            "  Threads: 2 on 2 virtual cores\n",
            "Environment:\n",
            "  LD_LIBRARY_PATH = /usr/local/nvidia/lib:/usr/local/nvidia/lib64\n",
            "  JULIA_NUM_THREADS = 2\n"
          ]
        }
      ],
      "source": [
        "versioninfo()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "Ff0GHbJoJjG6"
      },
      "outputs": [],
      "source": [
        "using Pkg"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "_HDCtvfPJodU",
        "outputId": "6c6901a4-172c-4aee-d0e9-dfe08b156cb5"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m registry at `~/.julia/registries/General.toml`\n",
            "\u001b[32m\u001b[1m   Resolving\u001b[22m\u001b[39m package versions...\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m Requires ─────────── v1.3.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m OrderedCollections ─ v1.6.2\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m LoweredCodeUtils ─── v2.3.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m CodeTracking ─────── v1.3.2\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m JuliaInterpreter ─── v0.9.23\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m Revise ───────────── v3.5.3\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/.julia/environments/v1.8/Project.toml`\n",
            " \u001b[90m [295af30f] \u001b[39m\u001b[92m+ Revise v3.5.3\u001b[39m\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/.julia/environments/v1.8/Manifest.toml`\n",
            " \u001b[90m [da1fd8a2] \u001b[39m\u001b[92m+ CodeTracking v1.3.2\u001b[39m\n",
            " \u001b[90m [aa1ae85d] \u001b[39m\u001b[92m+ JuliaInterpreter v0.9.23\u001b[39m\n",
            " \u001b[90m [6f1432cf] \u001b[39m\u001b[92m+ LoweredCodeUtils v2.3.0\u001b[39m\n",
            " \u001b[90m [bac558e1] \u001b[39m\u001b[92m+ OrderedCollections v1.6.2\u001b[39m\n",
            " \u001b[90m [ae029012] \u001b[39m\u001b[92m+ Requires v1.3.0\u001b[39m\n",
            " \u001b[90m [295af30f] \u001b[39m\u001b[92m+ Revise v3.5.3\u001b[39m\n",
            " \u001b[90m [8ba89e20] \u001b[39m\u001b[92m+ Distributed\u001b[39m\n",
            "\u001b[32m\u001b[1mPrecompiling\u001b[22m\u001b[39m project...\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mRequires\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mOrderedCollections\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mCodeTracking\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mJuliaInterpreter\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mLoweredCodeUtils\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39mRevise\n",
            "  6 dependencies successfully precompiled in 21 seconds. 20 already precompiled.\n",
            "\u001b[32m\u001b[1m   Resolving\u001b[22m\u001b[39m package versions...\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m TableTraits ───────────────── v1.0.1\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m DataAPI ───────────────────── v1.15.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m PooledArrays ──────────────── v1.4.2\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m Crayons ───────────────────── v4.1.1\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m Tables ────────────────────── v1.10.1\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m SentinelArrays ────────────── v1.4.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m IteratorInterfaceExtensions ─ v1.0.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m InlineStrings ─────────────── v1.4.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m DataValueInterfaces ───────── v1.0.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m LaTeXStrings ──────────────── v1.3.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m InvertedIndices ───────────── v1.3.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m Reexport ──────────────────── v1.2.2\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m Compat ────────────────────── v4.9.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m PrettyTables ──────────────── v2.2.7\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m SortingAlgorithms ─────────── v1.1.1\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m DataFrames ────────────────── v1.6.1\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m DataStructures ────────────── v0.18.15\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m Missings ──────────────────── v1.1.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m StringManipulation ────────── v0.3.0\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/.julia/environments/v1.8/Project.toml`\n",
            " \u001b[90m [a93c6f00] \u001b[39m\u001b[92m+ DataFrames v1.6.1\u001b[39m\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/.julia/environments/v1.8/Manifest.toml`\n",
            " \u001b[90m [34da2185] \u001b[39m\u001b[92m+ Compat v4.9.0\u001b[39m\n",
            " \u001b[90m [a8cc5b0e] \u001b[39m\u001b[92m+ Crayons v4.1.1\u001b[39m\n",
            " \u001b[90m [9a962f9c] \u001b[39m\u001b[92m+ DataAPI v1.15.0\u001b[39m\n",
            " \u001b[90m [a93c6f00] \u001b[39m\u001b[92m+ DataFrames v1.6.1\u001b[39m\n",
            " \u001b[90m [864edb3b] \u001b[39m\u001b[92m+ DataStructures v0.18.15\u001b[39m\n",
            " \u001b[90m [e2d170a0] \u001b[39m\u001b[92m+ DataValueInterfaces v1.0.0\u001b[39m\n",
            " \u001b[90m [842dd82b] \u001b[39m\u001b[92m+ InlineStrings v1.4.0\u001b[39m\n",
            " \u001b[90m [41ab1584] \u001b[39m\u001b[92m+ InvertedIndices v1.3.0\u001b[39m\n",
            " \u001b[90m [82899510] \u001b[39m\u001b[92m+ IteratorInterfaceExtensions v1.0.0\u001b[39m\n",
            " \u001b[90m [b964fa9f] \u001b[39m\u001b[92m+ LaTeXStrings v1.3.0\u001b[39m\n",
            " \u001b[90m [e1d29d7a] \u001b[39m\u001b[92m+ Missings v1.1.0\u001b[39m\n",
            " \u001b[90m [2dfb63ee] \u001b[39m\u001b[92m+ PooledArrays v1.4.2\u001b[39m\n",
            " \u001b[90m [08abe8d2] \u001b[39m\u001b[92m+ PrettyTables v2.2.7\u001b[39m\n",
            " \u001b[90m [189a3867] \u001b[39m\u001b[92m+ Reexport v1.2.2\u001b[39m\n",
            " \u001b[90m [91c51154] \u001b[39m\u001b[92m+ SentinelArrays v1.4.0\u001b[39m\n",
            " \u001b[90m [a2af1166] \u001b[39m\u001b[92m+ SortingAlgorithms v1.1.1\u001b[39m\n",
            " \u001b[90m [892a3eda] \u001b[39m\u001b[92m+ StringManipulation v0.3.0\u001b[39m\n",
            " \u001b[90m [3783bdb8] \u001b[39m\u001b[92m+ TableTraits v1.0.1\u001b[39m\n",
            " \u001b[90m [bd369af6] \u001b[39m\u001b[92m+ Tables v1.10.1\u001b[39m\n",
            " \u001b[90m [9fa8497b] \u001b[39m\u001b[92m+ Future\u001b[39m\n",
            "\u001b[32m\u001b[1mPrecompiling\u001b[22m\u001b[39m project...\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mDataValueInterfaces\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mCompat\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mLaTeXStrings\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mInvertedIndices\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mIteratorInterfaceExtensions\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mReexport\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mDataAPI\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mSentinelArrays\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mCrayons\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mStringManipulation\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mInlineStrings\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mTableTraits\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mMissings\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mPooledArrays\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mTables\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mDataStructures\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mSortingAlgorithms\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mPrettyTables\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39mDataFrames\n",
            "  19 dependencies successfully precompiled in 62 seconds. 26 already precompiled.\n",
            "\u001b[32m\u001b[1m   Resolving\u001b[22m\u001b[39m package versions...\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m FilePathsBase ────── v0.9.20\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m WorkerUtilities ──── v1.6.1\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m CodecZlib ────────── v0.7.2\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m WeakRefStrings ───── v1.4.2\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m TranscodingStreams ─ v0.9.13\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m CSV ──────────────── v0.10.11\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/.julia/environments/v1.8/Project.toml`\n",
            " \u001b[90m [336ed68f] \u001b[39m\u001b[92m+ CSV v0.10.11\u001b[39m\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/.julia/environments/v1.8/Manifest.toml`\n",
            " \u001b[90m [336ed68f] \u001b[39m\u001b[92m+ CSV v0.10.11\u001b[39m\n",
            " \u001b[90m [944b1d66] \u001b[39m\u001b[92m+ CodecZlib v0.7.2\u001b[39m\n",
            " \u001b[90m [48062228] \u001b[39m\u001b[92m+ FilePathsBase v0.9.20\u001b[39m\n",
            " \u001b[90m [3bb67fe8] \u001b[39m\u001b[92m+ TranscodingStreams v0.9.13\u001b[39m\n",
            " \u001b[90m [ea10d353] \u001b[39m\u001b[92m+ WeakRefStrings v1.4.2\u001b[39m\n",
            " \u001b[90m [76eceee3] \u001b[39m\u001b[92m+ WorkerUtilities v1.6.1\u001b[39m\n",
            "\u001b[32m\u001b[1mPrecompiling\u001b[22m\u001b[39m project...\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mWorkerUtilities\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mTranscodingStreams\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mWeakRefStrings\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mCodecZlib\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mFilePathsBase\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39mCSV\n",
            "  6 dependencies successfully precompiled in 19 seconds. 45 already precompiled.\n",
            "\u001b[32m\u001b[1m   Resolving\u001b[22m\u001b[39m package versions...\n",
            "\u001b[32m\u001b[1m  No Changes\u001b[22m\u001b[39m to `~/.julia/environments/v1.8/Project.toml`\n",
            "\u001b[32m\u001b[1m  No Changes\u001b[22m\u001b[39m to `~/.julia/environments/v1.8/Manifest.toml`\n",
            "\u001b[32m\u001b[1m     Cloning\u001b[22m\u001b[39m git-repo `https://github.com/bwbioinfo/KEGGAPI.jl`\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m git-repo `https://github.com/bwbioinfo/KEGGAPI.jl`\n",
            "\u001b[32m\u001b[1m   Resolving\u001b[22m\u001b[39m package versions...\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m SimpleBufferStream ── v1.1.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m OpenSSL_jll ───────── v3.0.10+0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m LoggingExtras ─────── v1.0.0\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m ConcurrentUtilities ─ v2.2.1\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m BitFlags ──────────── v0.1.7\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m URIs ──────────────── v1.4.2\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m ExceptionUnwrapping ─ v0.1.9\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m OpenSSL ───────────── v1.4.1\n",
            "\u001b[32m\u001b[1m   Installed\u001b[22m\u001b[39m HTTP ──────────────── v1.9.14\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/.julia/environments/v1.8/Project.toml`\n",
            " \u001b[90m [e8256861] \u001b[39m\u001b[92m+ KEGGAPI v1.0.0-DEV `https://github.com/bwbioinfo/KEGGAPI.jl#main`\u001b[39m\n",
            "\u001b[32m\u001b[1m    Updating\u001b[22m\u001b[39m `~/.julia/environments/v1.8/Manifest.toml`\n",
            " \u001b[90m [d1d4a3ce] \u001b[39m\u001b[92m+ BitFlags v0.1.7\u001b[39m\n",
            " \u001b[90m [f0e56b4a] \u001b[39m\u001b[92m+ ConcurrentUtilities v2.2.1\u001b[39m\n",
            " \u001b[90m [460bff9d] \u001b[39m\u001b[92m+ ExceptionUnwrapping v0.1.9\u001b[39m\n",
            " \u001b[90m [cd3eb016] \u001b[39m\u001b[92m+ HTTP v1.9.14\u001b[39m\n",
            " \u001b[90m [e8256861] \u001b[39m\u001b[92m+ KEGGAPI v1.0.0-DEV `https://github.com/bwbioinfo/KEGGAPI.jl#main`\u001b[39m\n",
            " \u001b[90m [e6f89c97] \u001b[39m\u001b[92m+ LoggingExtras v1.0.0\u001b[39m\n",
            " \u001b[90m [4d8831e6] \u001b[39m\u001b[92m+ OpenSSL v1.4.1\u001b[39m\n",
            " \u001b[90m [777ac1f9] \u001b[39m\u001b[92m+ SimpleBufferStream v1.1.0\u001b[39m\n",
            " \u001b[90m [5c2747f8] \u001b[39m\u001b[92m+ URIs v1.4.2\u001b[39m\n",
            " \u001b[90m [458c3c95] \u001b[39m\u001b[92m+ OpenSSL_jll v3.0.10+0\u001b[39m\n",
            "\u001b[32m\u001b[1mPrecompiling\u001b[22m\u001b[39m project...\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mSimpleBufferStream\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mBitFlags\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mURIs\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mLoggingExtras\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mExceptionUnwrapping\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mConcurrentUtilities\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mOpenSSL_jll\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mOpenSSL\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39m\u001b[90mHTTP\u001b[39m\n",
            "\u001b[32m  ✓ \u001b[39mKEGGAPI\n",
            "  10 dependencies successfully precompiled in 10 seconds. 51 already precompiled.\n"
          ]
        }
      ],
      "source": [
        "Pkg.add(\"Revise\")\n",
        "Pkg.add(\"DataFrames\")\n",
        "Pkg.add(\"CSV\")\n",
        "Pkg.add(\"BenchmarkTools\")\n",
        "Pkg.add(\"FastaIO\")\n",
        "Pkg.add(url=\"https://github.com/bwbioinfo/KEGGAPI.jl\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "nu1xa60HLsF9"
      },
      "outputs": [],
      "source": [
        "using Revise\n",
        "using DataFrames\n",
        "using CSV\n",
        "using FastaIO\n",
        "using BenchmarkTools\n",
        "using KEGGAPI"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "6NZMFaIwKabP"
      },
      "source": [
        "# Case 1: From Swissprot ID to Kegg information"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "V9h88j2GKpu5"
      },
      "source": [
        "### 1. Convert outside Database ID to Kegg ID and vice versa\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "\n",
        "| Database       | DB Identifier\n",
        "|:---------------|:-----------------|\n",
        "|Uniprot ID      | \"uniprotid\"      |\n",
        "|NCBI Gene ID    | \"ncbi-geneid\"    |\n",
        "|NCBI Protein ID | \"ncbi-proteinid\" |\n",
        "|KEGG ID         | \"genes\"          |\n",
        "    "
      ],
      "metadata": {
        "id": "PObI5fbwPTuz"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "### 1.1 Outside identifiers directly use as input\n",
        "\n",
        "To determine if a protein/gene is in KEGG database, the function conv uses as input the KEGG identifier and the gene of interest with the DB identifier.\n",
        "\n",
        "Only those outside identifiers with a hit in KEGG database are return"
      ],
      "metadata": {
        "id": "LhP60aGNNabw"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 137
        },
        "id": "olX5LvJJKhec",
        "outputId": "10d580e1-cc0a-47da-f902-5bb5299ecaf2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.535926 seconds (226 allocations: 12.391 KiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m1×2 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m Target ID     \u001b[0m\u001b[1m Source ID    \u001b[0m\n",
              "     │\u001b[90m String        \u001b[0m\u001b[90m String       \u001b[0m\n",
              "─────┼─────────────────────────────\n",
              "   1 │ up:A0A072UR65  mtr:25493984"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>1×2 DataFrame</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">Target ID</th><th style = \"text-align: left;\">Source ID</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">up:A0A072UR65</td><td style = \"text-align: left;\">mtr:25493984</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& Target ID & Source ID\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & up:A0A072UR65 & mtr:25493984 \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 10
        }
      ],
      "source": [
        "@time kegg_conv_uniprot = KEGGAPI.conv(\"genes\", \"uniprot:A0A072UR65\")\n",
        "DataFrame(\n",
        "    kegg_conv_uniprot.data,\n",
        "    kegg_conv_uniprot.colnames\n",
        "    )"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### 1.2 Outside database identifiers from a file as input\n",
        "\n",
        "Several identifiers from the same database can be run at once. Either as input from a file or several identifiers join by \"+\" sign.\n",
        "\n",
        "Only those outside identifiers with a hit in KEGG database are return.\n",
        "\n",
        "The selected dataset belong to Uniprot proteins Review dataset. User can download the data and upload it to their session. https://www.kaggle.com/datasets/andreylovyagin/uniprot-proteins-reviewed-swissprot?select=data.csv"
      ],
      "metadata": {
        "id": "H-RZzmNHL6VS"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "df = DataFrame(CSV.File(\"subset_data.csv\", header=1, delim=\",\"))"
      ],
      "metadata": {
        "id": "dmGoXiKgIkMT",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "0c9f6884-7b34-4ba8-d9e5-c350076538bf"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m49×112 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m Column1 \u001b[0m\u001b[1m Entry      \u001b[0m\u001b[1m Reviewed \u001b[0m\u001b[1m Entry Name  \u001b[0m\u001b[1m Protein names              \u001b[0m ⋯\n",
              "     │\u001b[90m Int64   \u001b[0m\u001b[90m String15   \u001b[0m\u001b[90m String15 \u001b[0m\u001b[90m String15    \u001b[0m\u001b[90m String                     \u001b[0m ⋯\n",
              "─────┼──────────────────────────────────────────────────────────────────────────\n",
              "   1 │       0  A0A024B7W1  reviewed  POLG_ZIKVF   Genome polyprotein [Cleaved ⋯\n",
              "   2 │       1  A0A024SC78  reviewed  CUTI1_HYPJR  Cutinase, EC 3.1.1.74\n",
              "   3 │       2  A0A024SH76  reviewed  GUX2_HYPJR   Exoglucanase 2, EC 3.2.1.91\n",
              "   4 │       3  A0A026W182  reviewed  ORCO_OOCBI   Odorant receptor coreceptor\n",
              "   5 │       4  A0A044RE18  reviewed  BLI_ONCVO    Endoprotease bli, EC 3.4.21 ⋯\n",
              "   6 │       5  A0A059TC02  reviewed  CCR1_PETHY   Cinnamoyl-CoA reductase 1,\n",
              "   7 │       6  A0A060A682  reviewed  HAP2_TETTH   Hapless 2 (Generative cell\n",
              "   8 │       7  A0A061ACU2  reviewed  PIEZ1_CAEEL  Piezo-type mechanosensitive\n",
              "   9 │       8  A0A061I403  reviewed  FICD_CRIGR   Protein adenylyltransferase ⋯\n",
              "  10 │       9  A0A067CMC7  reviewed  HTP3_SAPPC   Endonuclease Htp3, EC 3.1.3\n",
              "  11 │      10  A0A067XGX8  reviewed  AROG2_PETHY  Phospho-2-dehydro-3-deoxyhe\n",
              "  ⋮  │    ⋮         ⋮          ⋮           ⋮                       ⋮           ⋱\n",
              "  40 │      39  A0A0A1C930  reviewed  HMR2A_PANGI  3-hydroxy-3-methylglutaryl\n",
              "  41 │      40  A0A0A1H8I4  reviewed  AIS_PSESP    Aconitate isomerase, AI, EC ⋯\n",
              "  42 │      41  A0A0A1HA03  reviewed  708C1_FAGES  UDP-glycosyltransferase 708\n",
              "  43 │      42  A0A0A1I6E7  reviewed  NDB4S_ANDCR  Antimicrobial peptide AcrAP\n",
              "  44 │      43  A0A0A1I6N9  reviewed  NDB4T_ANDCR  Antimicrobial peptide AcrAP\n",
              "  45 │      44  A0A0A7EPL0  reviewed  PIAL1_ARATH  E4 SUMO-protein ligase PIAL ⋯\n",
              "  46 │      45  A0A0A7EQR3  reviewed  GCE_CERUI    4-O-methyl-glucuronoyl meth\n",
              "  47 │      46  A0A0A7GEY4  reviewed  GGPPS_GEOAI  Geranylgeranyl pyrophosphat\n",
              "  48 │      47  A0A0A7HFE1  reviewed  CAS10_STRTR  CRISPR system single-strand\n",
              "  49 │      48  A0A0A7HIF0  reviewed  CSM3_STRTR   CRISPR system Cms endoribon ⋯\n",
              "\u001b[36m                                                 108 columns and 28 rows omitted\u001b[0m"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>49×112 DataFrame</span></div><div style = \"float: right;\"><span style = \"font-style: italic;\">12 columns and 24 rows omitted</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">Column1</th><th style = \"text-align: left;\">Entry</th><th style = \"text-align: left;\">Reviewed</th><th style = \"text-align: left;\">Entry Name</th><th style = \"text-align: left;\">Protein names</th><th style = \"text-align: left;\">Gene Names</th><th style = \"text-align: left;\">Organism</th><th style = \"text-align: left;\">Length</th><th style = \"text-align: left;\">Gene Names (ordered locus)</th><th style = \"text-align: left;\">Gene Names (ORF)</th><th style = \"text-align: left;\">Gene Names (primary)</th><th style = \"text-align: left;\">Gene Names (synonym)</th><th style = \"text-align: left;\">Organism (ID)</th><th style = \"text-align: left;\">Proteomes</th><th style = \"text-align: left;\">Taxonomic lineage</th><th style = \"text-align: left;\">Taxonomic lineage (Ids)</th><th style = \"text-align: left;\">Virus hosts</th><th style = \"text-align: left;\">Alternative products (isoforms)</th><th style = \"text-align: left;\">Alternative sequence</th><th style = \"text-align: left;\">Erroneous gene model prediction</th><th style = \"text-align: left;\">Fragment</th><th style = \"text-align: left;\">Gene encoded by</th><th style = \"text-align: left;\">Mass</th><th style = \"text-align: left;\">Mass spectrometry</th><th style = \"text-align: left;\">Natural variant</th><th style = \"text-align: left;\">Non-adjacent residues</th><th style = \"text-align: left;\">Non-standard residue</th><th style = \"text-align: left;\">Non-terminal residue</th><th style = \"text-align: left;\">Polymorphism</th><th style = \"text-align: left;\">RNA Editing</th><th style = \"text-align: left;\">Sequence</th><th style = \"text-align: left;\">Sequence caution</th><th style = \"text-align: left;\">Sequence conflict</th><th style = \"text-align: left;\">Sequence uncertainty</th><th style = \"text-align: left;\">Sequence version</th><th style = \"text-align: left;\">Absorption</th><th style = \"text-align: left;\">Active site</th><th style = \"text-align: left;\">Binding site</th><th style = \"text-align: left;\">Catalytic activity</th><th style = \"text-align: left;\">Cofactor</th><th style = \"text-align: left;\">DNA binding</th><th style = \"text-align: left;\">EC number</th><th style = \"text-align: left;\">Function [CC]</th><th style = \"text-align: left;\">Activity regulation</th><th style = \"text-align: left;\">Kinetics</th><th style = \"text-align: left;\">Pathway</th><th style = \"text-align: left;\">pH dependence</th><th style = \"text-align: left;\">Redox potential</th><th style = \"text-align: left;\">Rhea ID</th><th style = \"text-align: left;\">Site</th><th style = \"text-align: left;\">Temperature dependence</th><th style = \"text-align: left;\">Annotation</th><th style = \"text-align: left;\">Caution</th><th style = \"text-align: left;\">Keywords</th><th style = \"text-align: left;\">Keyword ID</th><th style = \"text-align: left;\">Miscellaneous [CC]</th><th style = \"text-align: left;\">Protein existence</th><th style = \"text-align: left;\">Tools</th><th style = \"text-align: left;\">UniParc</th><th style = \"text-align: left;\">Comments</th><th style = \"text-align: left;\">Features</th><th style = \"text-align: left;\">Interacts with</th><th style = \"text-align: left;\">Subunit structure</th><th style = \"text-align: left;\">Developmental stage</th><th style = \"text-align: left;\">Tissue specificity</th><th style = \"text-align: left;\">Induction</th><th style = \"text-align: left;\">Gene Ontology (biological process)</th><th style = \"text-align: left;\">Gene Ontology (cellular component)</th><th style = \"text-align: left;\">Gene Ontology (molecular function)</th><th style = \"text-align: left;\">Gene Ontology IDs</th><th style = \"text-align: left;\">Gene Ontology (GO)</th><th style = \"text-align: left;\">Allergenic Properties</th><th style = \"text-align: left;\">Biotechnological use</th><th style = \"text-align: left;\">Disruption phenotype</th><th style = \"text-align: left;\">Involvement in disease</th><th style = \"text-align: left;\">Mutagenesis</th><th style = \"text-align: left;\">Pharmaceutical use</th><th style = \"text-align: left;\">Toxic dose</th><th style = \"text-align: left;\">Intramembrane</th><th style = \"text-align: left;\">Subcellular location [CC]</th><th style = \"text-align: left;\">Topological domain</th><th style = \"text-align: left;\">Transmembrane</th><th style = \"text-align: left;\">Chain</th><th style = \"text-align: left;\">Cross-link</th><th style = \"text-align: left;\">Disulfide bond</th><th style = \"text-align: left;\">Glycosylation</th><th style = \"text-align: left;\">Initiator methionine</th><th style = \"text-align: left;\">Lipidation</th><th style = \"text-align: left;\">Modified residue</th><th style = \"text-align: left;\">Peptide</th><th style = \"text-align: left;\">Post-translational modification</th><th style = \"text-align: left;\">Propeptide</th><th style = \"text-align: left;\">Signal peptide</th><th style = \"text-align: left;\">Transit peptide</th><th style = \"text-align: left;\">3D</th><th style = \"text-align: left;\">Beta strand</th><th style = \"text-align: left;\">Helix</th><th style = \"text-align: left;\">Turn</th><th style = \"text-align: left;\">PubMed ID</th><th style = \"text-align: left;\">Date of creation</th><th style = \"text-align: right;\">&ctdot;</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"Int64\" style = \"text-align: left;\">Int64</th><th title = \"String15\" style = \"text-align: left;\">String15</th><th title = \"String15\" style = \"text-align: left;\">String15</th><th title = \"String15\" style = \"text-align: left;\">String15</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Int64\" style = \"text-align: left;\">Int64</th><th title = \"Union{Missing, String15}\" style = \"text-align: left;\">String15?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String15}\" style = \"text-align: left;\">String15?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Int64\" style = \"text-align: left;\">Int64</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Union{Missing, String31}\" style = \"text-align: left;\">String31?</th><th title = \"Int64\" style = \"text-align: left;\">Int64</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Int64\" style = \"text-align: left;\">Int64</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Float64\" style = \"text-align: left;\">Float64</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"String31\" style = \"text-align: left;\">String31</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"String15\" style = \"text-align: left;\">String15</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Union{Missing, String7}\" style = \"text-align: left;\">String7?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Missing\" style = \"text-align: left;\">Missing</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Union{Missing, String}\" style = \"text-align: left;\">String?</th><th title = \"Dates.Date\" style = \"text-align: left;\">Date</th><th title = \"Dates.Date\" style = \"text-align: right;\">&ctdot;</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: right;\">0</td><td style = \"text-align: left;\">A0A024B7W1</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">POLG_ZIKVF</td><td style = \"text-align: left;\">Genome polyprotein [Cleaved into: Capsid protein C (Capsid protein) (Core protein); Protein prM (Precursor membrane protein); Peptide pr (Peptide precursor); Small envelope protein M (Matrix protein); Envelope protein E; Non-structural protein 1, NS1; Non-structural protein 2A, NS2A; Serine protease subunit NS2B (Flavivirin protease NS2B regulatory subunit) (Non-structural protein 2B); Serine protease NS3, EC 3.4.21.91, EC 3.6.1.15, EC 3.6.4.13 (Flavivirin protease NS3 catalytic subunit) (Non-structural protein 3); Non-structural protein 4A, NS4A; Peptide 2k; Non-structural protein 4B, NS4B; RNA-directed RNA polymerase NS5, EC 2.1.1.56, EC 2.1.1.57, EC 2.7.7.48 (NS5) ]</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Zika virus (isolate ZIKV/Human/French Polynesia/10087PF/2013) (ZIKV)</td><td style = \"text-align: right;\">3423</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">2043570</td><td style = \"text-align: left;\">UP000112691: Genome; UP000137079: Genome; UP000151151: Genome; UP000168269: Genome</td><td style = \"text-align: left;\">Zika virus (species), Flavivirus (genus), Flaviviridae (family), Amarillovirales (order), Flasuviricetes (class), Kitrinoviricota (phylum), Orthornavirae (kingdom), Riboviria (no rank), Viruses (superkingdom)</td><td style = \"text-align: left;\">64320 (species), 11051 (genus), 11050 (family), 2732545 (order), 2732462 (class), 2732406 (phylum), 2732396 (kingdom), 2559587 (no rank), 10239 (superkingdom)</td><td style = \"text-align: left;\">Aedes aegypti (Yellowfever mosquito) (Culex aegypti) [TaxID: 7159]; Aedes albopictus (Asian tiger mosquito) (Stegomyia albopicta) [TaxID: 7160]; Homo sapiens (Human) [TaxID: 9606]; Macaca mulatta (Rhesus macaque) [TaxID: 9544]</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">379113</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MKNPKKKSGGFRIVNMLKRGVARVSPFGGLKRLPAGLLLGHGPIRMVLAILAFLRFTAIKPSLGLINRWGSVGKKEAMEIIKKFKKDLAAMLRIINARKEKKRRGADTSVGIVGLLLTTAMAAEVTRRGSAYYMYLDRNDAGEAISFPTTLGMNKCYIQIMDLGHMCDATMSYECPMLDEGVEPDDVDCWCNTTSTWVVYGTCHHKKGEARRSRRAVTLPSHSTRKLQTRSQTWLESREYTKHLIRVENWIFRNPGFALAAAAIAWLLGSSTSQKVIYLVMILLIAPAYSIRCIGVSNRDFVEGMSGGTWVDVVLEHGGCVTVMAQDKPTVDIELVTTTVSNMAEVRSYCYEASISDMASDSRCPTQGEAYLDKQSDTQYVCKRTLVDRGWGNGCGLFGKGSLVTCAKFACSKKMTGKSIQPENLEYRIMLSVHGSQHSGMIVNDTGHETDENRAKVEITPNSPRAEATLGGFGSLGLDCEPRTGLDFSDLYYLTMNNKHWLVHKEWFHDIPLPWHAGADTGTPHWNNKEALVEFKDAHAKRQTVVVLGSQEGAVHTALAGALEAEMDGAKGRLSSGHLKCRLKMDKLRLKGVSYSLCTAAFTFTKIPAETLHGTVTVEVQYAGTDGPCKVPAQMAVDMQTLTPVGRLITANPVITESTENSKMMLELDPPFGDSYIVIGVGEKKITHHWHRSGSTIGKAFEATVRGAKRMAVLGDTAWDFGSVGGALNSLGKGIHQIFGAAFKSLFGGMSWFSQILIGTLLMWLGLNTKNGSISLMCLALGGVLIFLSTAVSADVGCSVDFSKKETRCGTGVFVYNDVEAWRDRYKYHPDSPRRLAAAVKQAWEDGICGISSVSRMENIMWRSVEGELNAILEENGVQLTVVVGSVKNPMWRGPQRLPVPVNELPHGWKAWGKSYFVRAAKTNNSFVVDGDTLKECPLKHRAWNSFLVEDHGFGVFHTSVWLKVREDYSLECDPAVIGTAVKGKEAVHSDLGYWIESEKNDTWRLKRAHLIEMKTCEWPKSHTLWTDGIEESDLIIPKSLAGPLSHHNTREGYRTQMKGPWHSEELEIRFEECPGTKVHVEETCGTRGPSLRSTTASGRVIEEWCCRECTMPPLSFRAKDGCWYGMEIRPRKEPESNLVRSMVTAGSTDHMDHFSLGVLVILLMVQEGLKKRMTTKIIISTSMAVLVAMILGGFSMSDLAKLAILMGATFAEMNTGGDVAHLALIAAFKVRPALLVSFIFRANWTPRESMLLALASCLLQTAISALEGDLMVLINGFALAWLAIRAMVVPRTDNITLAILAALTPLARGTLLVAWRAGLATCGGFMLLSLKGKGSVKKNLPFVMALGLTAVRLVDPINVVGLLLLTRSGKRSWPPSEVLTAVGLICALAGGFAKADIEMAGPMAAVGLLIVSYVVSGKSVDMYIERAGDITWEKDAEVTGNSPRLDVALDESGDFSLVEDDGPPMREIILKVVLMTICGMNPIAIPFAAGAWYVYVKTGKRSGALWDVPAPKEVKKGETTDGVYRVMTRRLLGSTQVGVGVMQEGVFHTMWHVTKGSALRSGEGRLDPYWGDVKQDLVSYCGPWKLDAAWDGHSEVQLLAVPPGERARNIQTLPGIFKTKDGDIGAVALDYPAGTSGSPILDKCGRVIGLYGNGVVIKNGSYVSAITQGRREEETPVECFEPSMLKKKQLTVLDLHPGAGKTRRVLPEIVREAIKTRLRTVILAPTRVVAAEMEEALRGLPVRYMTTAVNVTHSGTEIVDLMCHATFTSRLLQPIRVPNYNLYIMDEAHFTDPSSIAARGYISTRVEMGEAAAIFMTATPPGTRDAFPDSNSPIMDTEVEVPERAWSSGFDWVTDHSGKTVWFVPSVRNGNEIAACLTKAGKRVIQLSRKTFETEFQKTKHQEWDFVVTTDISEMGANFKADRVIDSRRCLKPVILDGERVILAGPMPVTHASAAQRRGRIGRNPNKPGDEYLYGGGCAETDEDHAHWLEARMLLDNIYLQDGLIASLYRPEADKVAAIEGEFKLRTEQRKTFVELMKRGDLPVWLAYQVASAGITYTDRRWCFDGTTNNTIMEDSVPAEVWTRHGEKRVLKPRWMDARVCSDHAALKSFKEFAAGKRGAAFGVMEALGTLPGHMTERFQEAIDNLAVLMRAETGSRPYKAAAAQLPETLETIMLLGLLGTVSLGIFFVLMRNKGIGKMGFGMVTLGASAWLMWLSEIEPARIACVLIVVFLLLVVLIPEPEKQRSPQDNQMAIIIMVAVGLLGLITANELGWLERTKSDLSHLMGRREEGATIGFSMDIDLRPASAWAIYAALTTFITPAVQHAVTTSYNNYSLMAMATQAGVLFGMGKGMPFYAWDFGVPLLMIGCYSQLTPLTLIVAIILLVAHYMYLIPGLQAAAARAAQKRTAAGIMKNPVVDGIVVTDIDTMTIDPQVEKKMGQVLLIAVAVSSAILSRTAWGWGEAGALITAATSTLWEGSPNKYWNSSTATSLCNIFRGSYLAGASLIYTVTRNAGLVKRRGGGTGETLGEKWKARLNQMSALEFYSYKKSGITEVCREEARRALKDGVATGGHAVSRGSAKLRWLVERGYLQPYGKVIDLGCGRGGWSYYAATIRKVQEVKGYTKGGPGHEEPMLVQSYGWNIVRLKSGVDVFHMAAEPCDTLLCDIGESSSSPEVEEARTLRVLSMVGDWLEKRPGAFCIKVLCPYTSTMMETLERLQRRYGGGLVRVPLSRNSTHEMYWVSGAKSNTIKSVSTTSQLLLGRMDGPRRPVKYEEDVNLGSGTRAVVSCAEAPNMKIIGNRIERIRSEHAETWFFDENHPYRTWAYHGSYEAPTQGSASSLINGVVRLLSKPWDVVTGVTGIAMTDTTPYGQQRVFKEKVDTRVPDPQEGTRQVMSMVSSWLWKELGKHKRPRVCTKEEFINKVRSNAALGAIFEEEKEWKTAVEAVNDPRFWALVDKEREHHLRGECQSCVYNMMGKREKKQGEFGKAKGSRAIWYMWLGARFLEFEALGFLNEDHWMGRENSGGGVEGLGLQRLGYVLEEMSRIPGGRMYADDTAGWDTRISRFDLENEALITNQMEKGHRALALAIIKYTYQNKVVKVLRPAEKGKTVMDIISRQDQRGSGQVVTYALNTFTNLVVQLIRNMEAEEVLEMQDLWLLRRSEKVTNWLQSNGWDRLKRMAVSGDDCVVKPIDDRFAHALRFLNDMGKVRKDTQEWKPSTGWDNWEEVPFCSHHFNKLHLKDGRSIVVPCRHQDELIGRARVSPGAGWSIRETACLAKSYAQMWQLLYFHRRDLRLMANAICSSVPVDWVPTGRTTWSIHGKGEWMTTEDMLVVWNRVWIEENDHMEDKTPVTKWTDIPYLGKREDLWCGSLIGHRPRTTWAENIKNTVNMVRRIIGDEEKYMDYLSTQVRYLGEEGSTPGVL</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 1553; /note=&quot;Charge relay system; for serine protease NS3 activity&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00860&quot;; ACT_SITE 1577; /note=&quot;Charge relay system; for serine protease NS3 activity&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00860&quot;; ACT_SITE 1637; /note=&quot;Charge relay system; for serine protease NS3 activity&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00860&quot;; ACT_SITE 2581; /note=&quot;For 2&apos;-O-MTase activity&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q6YMS4&quot;; ACT_SITE 2666; /note=&quot;For 2&apos;-O-MTase activity&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q6YMS4&quot;; ACT_SITE 2702; /note=&quot;For 2&apos;-O-MTase activity&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q6YMS4&quot;; ACT_SITE 2738; /note=&quot;For 2&apos;-O-MTase activity&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q6YMS4&quot;</td><td style = \"text-align: left;\">BINDING 1696..1703; /ligand=&quot;ATP&quot;; /ligand_id=&quot;ChEBI:CHEBI:30616&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00541&quot;; BINDING 2533..2539; /ligand=&quot;GTP&quot;; /ligand_id=&quot;ChEBI:CHEBI:37565&quot;; /evidence=&quot;ECO:0000269|PubMed:27866982&quot;; BINDING 2576; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924, ECO:0000269|PubMed:27866982&quot;; BINDING 2606; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924, ECO:0000269|PubMed:27633330, ECO:0000269|PubMed:27866982, ECO:0000269|PubMed:28031359&quot;; BINDING 2607; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924, ECO:0000269|PubMed:27633330, ECO:0000269|PubMed:27866982, ECO:0000269|PubMed:28031359&quot;; BINDING 2624; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924, ECO:0000269|PubMed:27866982&quot;; BINDING 2625; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924, ECO:0000269|PubMed:27633330, ECO:0000269|PubMed:27866982, ECO:0000269|PubMed:28031359&quot;; BINDING 2630; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000269|PubMed:27633330, ECO:0000269|PubMed:27866982, ECO:0000269|PubMed:28031359&quot;; BINDING 2631; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000269|PubMed:27633330, ECO:0000269|PubMed:27866982, ECO:0000269|PubMed:28031359&quot;; BINDING 2651; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924, ECO:0000269|PubMed:27633330, ECO:0000269|PubMed:27866982, ECO:0000269|PubMed:28031359&quot;; BINDING 2652; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924, ECO:0000269|PubMed:27633330, ECO:0000269|PubMed:27866982, ECO:0000269|PubMed:28031359&quot;; BINDING 2666; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000269|PubMed:27633330, ECO:0000269|PubMed:27866982, ECO:0000269|PubMed:28031359&quot;; BINDING 2667; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; BINDING 2669..2675; /ligand=&quot;GTP&quot;; /ligand_id=&quot;ChEBI:CHEBI:37565&quot;; /evidence=&quot;ECO:0000269|PubMed:27866982&quot;; BINDING 2733..2735; /ligand=&quot;GTP&quot;; /ligand_id=&quot;ChEBI:CHEBI:37565&quot;; /evidence=&quot;ECO:0000269|PubMed:27866982&quot;; BINDING 2740; /ligand=&quot;S-adenosyl-L-methionine&quot;; /ligand_id=&quot;ChEBI:CHEBI:59789&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; BINDING 2959; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; BINDING 2963; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; BINDING 2968; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; BINDING 2971; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; BINDING 3234; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /ligand_label=&quot;2&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; BINDING 3250; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /ligand_label=&quot;2&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; BINDING 3369; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /ligand_label=&quot;2&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P14335&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: [RNA-directed RNA polymerase NS5]: Reaction=a 5&apos;-end (5&apos;-triphosphoguanosine)-(ribonucleoside) in mRNA + S-adenosyl-L-methionine = a 5&apos;-end (N(7)-methyl 5&apos;-triphosphoguanosine)-ribonucleoside in mRNA + S-adenosyl-L-homocysteine; Xref=Rhea:RHEA:67008, Rhea:RHEA-COMP:17166, Rhea:RHEA-COMP:17167, ChEBI:CHEBI:57856, ChEBI:CHEBI:59789, ChEBI:CHEBI:156461, ChEBI:CHEBI:167617; EC=2.1.1.56; Evidence={ECO:0000255|PROSITE-ProRule:PRU00924}; CATALYTIC ACTIVITY: [RNA-directed RNA polymerase NS5]: Reaction=a 5&apos;-end (N(7)-methyl 5&apos;-triphosphoguanosine)-ribonucleoside in mRNA + S-adenosyl-L-methionine = a 5&apos;-end (N(7)-methyl 5&apos;-triphosphoguanosine)-(2&apos;-O-methyl-ribonucleoside) in mRNA + H(+) + S-adenosyl-L-homocysteine; Xref=Rhea:RHEA:67020, Rhea:RHEA-COMP:17167, Rhea:RHEA-COMP:17168, ChEBI:CHEBI:15378, ChEBI:CHEBI:57856, ChEBI:CHEBI:59789, ChEBI:CHEBI:156461, ChEBI:CHEBI:167609; EC=2.1.1.57; Evidence={ECO:0000255|PROSITE-ProRule:PRU00924}; CATALYTIC ACTIVITY: Reaction=a ribonucleoside 5&apos;-triphosphate + RNA(n) = diphosphate + RNA(n+1); Xref=Rhea:RHEA:21248, Rhea:RHEA-COMP:14527, Rhea:RHEA-COMP:17342, ChEBI:CHEBI:33019, ChEBI:CHEBI:61557, ChEBI:CHEBI:140395; EC=2.7.7.48; Evidence={ECO:0000255|PROSITE-ProRule:PRU00539, ECO:0000269|PubMed:30951555}; CATALYTIC ACTIVITY: Reaction=Selective hydrolysis of -Xaa-Xaa-|-Yaa- bonds in which each of the Xaa can be either Arg or Lys and Yaa can be either Ser or Ala.; EC=3.4.21.91; Evidence={ECO:0000250|UniProtKB:Q32ZE1}; CATALYTIC ACTIVITY: Reaction=a ribonucleoside 5&apos;-triphosphate + H2O = a ribonucleoside 5&apos;-diphosphate + H(+) + phosphate; Xref=Rhea:RHEA:23680, ChEBI:CHEBI:15377, ChEBI:CHEBI:15378, ChEBI:CHEBI:43474, ChEBI:CHEBI:57930, ChEBI:CHEBI:61557; EC=3.6.1.15; Evidence={ECO:0000250|UniProtKB:Q32ZE1}; CATALYTIC ACTIVITY: Reaction=ATP + H2O = ADP + H(+) + phosphate; Xref=Rhea:RHEA:13065, ChEBI:CHEBI:15377, ChEBI:CHEBI:15378, ChEBI:CHEBI:30616, ChEBI:CHEBI:43474, ChEBI:CHEBI:456216; EC=3.6.4.13; Evidence={ECO:0000250|UniProtKB:Q9Q6P4};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.1.1.56; 2.1.1.57; 2.7.7.48; 3.4.21.91; 3.6.1.15; 3.6.4.13</td><td style = \"text-align: left;\">FUNCTION: [Capsid protein C]: Plays a role in virus budding by binding to the cell membrane and gathering the viral RNA into a nucleocapsid that forms the core of the mature virus particle. During virus entry, may induce genome penetration into the host cytoplasm after hemifusion induced by the surface proteins. Can migrate to the cell nucleus where it modulates host functions. {ECO:0000250|UniProtKB:P17763}.; FUNCTION: [Capsid protein C]: Inhibits RNA silencing by interfering with host Dicer. {ECO:0000250|UniProtKB:P03314}.; FUNCTION: [Peptide pr]: Prevents premature fusion activity of envelope proteins in trans-Golgi by binding to envelope protein E at pH 6.0. After virion release in extracellular space, gets dissociated from E dimers. {ECO:0000250|UniProtKB:P17763}.; FUNCTION: [Protein prM]: Plays a role in host immune defense modulation and protection of envelope protein E during virion synthesis. PrM-E cleavage is inefficient, many virions are only partially matured and immature prM-E proteins could play a role in immune evasion. Contributes to fetal microcephaly in humans. Acts as a chaperone for envelope protein E during intracellular virion assembly by masking and inactivating envelope protein E fusion peptide. prM is the only viral peptide matured by host furin in the trans-Golgi network probably to avoid catastrophic activation of the viral fusion activity in acidic Golgi compartment prior to virion release. {ECO:0000250|UniProtKB:P17763}.; FUNCTION: [Small envelope protein M]: May play a role in virus budding. Exerts cytotoxic effects by activating a mitochondrial apoptotic pathway through M ectodomain. May display a viroporin activity. {ECO:0000250|UniProtKB:P17763}.; FUNCTION: [Envelope protein E]: Binds to host cell surface receptors and mediates fusion between viral and cellular membranes. Efficient virus attachment to cell is, at least in part, mediated by host HAVCR1 in a cell-type specific manner (By similarity). In addition, host NCAM1 can also be used as entry receptor (By similarity).Interaction with host HSPA5 plays an important role in the early stages of infection as well (By similarity). Envelope protein is synthesized in the endoplasmic reticulum and forms a heterodimer with protein prM. The heterodimer plays a role in virion budding in the ER, and the newly formed immature particle is covered with 60 spikes composed of heterodimers between precursor prM and envelope protein E. The virion is transported to the Golgi apparatus where the low pH causes the dissociation of PrM-E heterodimers and formation of E homodimers. PrM-E cleavage is inefficient, many virions are only partially matured and immature prM-E proteins could play a role in immune evasion (By similarity). {ECO:0000250|UniProtKB:A0A142I5B9, ECO:0000250|UniProtKB:P17763}.; FUNCTION: [Non-structural protein 1]: Plays a role in the inhibition of host RLR-induced interferon-beta activation by targeting TANK-binding kinase 1/TBK1. In addition, recruits the host deubiquitinase USP8 to cleave &apos;Lys-11&apos;-linked polyubiquitin chains from caspase-1/CASP1 thus inhibiting its proteasomal degradation. In turn, stabilized CASP1 promotes cleavage of cGAS, which inhibits its ability to recognize mitochondrial DNA release and initiate type I interferon signaling. {ECO:0000250|UniProtKB:Q32ZE1}.; FUNCTION: [Non-structural protein 2A]: Component of the viral RNA replication complex that recruits genomic RNA, the structural protein prM/E complex, and the NS2B/NS3 protease complex to the virion assembly site and orchestrates virus morphogenesis (By similarity). Antagonizes also the host MDA5-mediated induction of alpha/beta interferon antiviral response (By similarity). May disrupt adherens junction formation and thereby impair proliferation of radial cells in the host cortex (By similarity). {ECO:0000250|UniProtKB:A0A142I5B9, ECO:0000250|UniProtKB:Q32ZE1}.; FUNCTION: [Serine protease subunit NS2B]: Required cofactor for the serine protease function of NS3. {ECO:0000250|UniProtKB:Q32ZE1}.; FUNCTION: [Serine protease NS3]: Displays three enzymatic activities: serine protease, NTPase and RNA helicase. NS3 serine protease, in association with NS2B, performs its autocleavage and cleaves the polyprotein at dibasic sites in the cytoplasm: C-prM, NS2A-NS2B, NS2B-NS3, NS3-NS4A, NS4A-2K and NS4B-NS5. NS3 RNA helicase binds RNA and unwinds dsRNA in the 3&apos; to 5&apos; direction (By similarity). Leads to translation arrest when expressed ex vivo (PubMed:28592527). {ECO:0000250|UniProtKB:Q32ZE1, ECO:0000269|PubMed:28592527}.; FUNCTION: [Non-structural protein 4A]: Regulates the ATPase activity of the NS3 helicase activity (By similarity). NS4A allows NS3 helicase to conserve energy during unwinding (By similarity). Cooperatively with NS4B suppresses the Akt-mTOR pathway and leads to cellular dysregulation (PubMed:27524440). By inhibiting host ANKLE2 functions, may cause defects in brain development, such as microcephaly (PubMed:30550790). Antagonizes also the host MDA5-mediated induction of alpha/beta interferon antiviral response (By similarity). Leads to translation arrest when expressed ex vivo (PubMed:28592527). {ECO:0000250|UniProtKB:Q32ZE1, ECO:0000250|UniProtKB:Q9Q6P4, ECO:0000269|PubMed:27524440, ECO:0000269|PubMed:28592527, ECO:0000269|PubMed:30550790}.; FUNCTION: [Peptide 2k]: Functions as a signal peptide for NS4B and is required for the interferon antagonism activity of the latter. {ECO:0000250|UniProtKB:P17763}.; FUNCTION: [Non-structural protein 4B]: Induces the formation of ER-derived membrane vesicles where the viral replication takes place (By similarity). Also plays a role in the inhibition of host RLR-induced interferon-beta production at TANK-binding kinase 1/TBK1 level (By similarity). Cooperatively with NS4A suppresses the Akt-mTOR pathway and leads to cellular dysregulation (PubMed:27524440). {ECO:0000250|UniProtKB:Q32ZE1, ECO:0000250|UniProtKB:Q9Q6P4, ECO:0000269|PubMed:27524440}.; FUNCTION: [RNA-directed RNA polymerase NS5]: Replicates the viral (+) and (-) RNA genome, and performs the capping of genomes in the cytoplasm (PubMed:30951555). Methylates viral RNA cap at guanine N-7 and ribose 2&apos;-O positions. Once sufficient NS5 is expressed, binds to the cap-proximal structure and inhibits further translation of the viral genome (By similarity). Besides its role in RNA genome replication, also prevents the establishment of a cellular antiviral state by blocking the interferon-alpha/beta (IFN-alpha/beta) signaling pathway. Mechanistically, interferes with host kinases TBK1 and IKKE upstream of interferon regulatory factor 3/IRF3 to inhibit the RIG-I pathway (By similarity). Antagonizes also type I interferon signaling by targeting STAT2 for degradation by the proteasome thereby preventing activation of JAK-STAT signaling pathway (By similarity). Within the host nucleus, disrupts host SUMO1 and STAT2 co-localization with PML, resulting in PML degradation (PubMed:32699085). May also reduce immune responses by preventing the recruitment of the host PAF1 complex to interferon-responsive genes (PubMed:30550790). {ECO:0000250|UniProtKB:Q32ZE1, ECO:0000269|PubMed:30550790, ECO:0000269|PubMed:30951555, ECO:0000269|PubMed:32699085}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:67008 RHEA-COMP:17166 RHEA-COMP:17167 RHEA:67020 RHEA-COMP:17167 RHEA-COMP:17168 RHEA:21248 RHEA-COMP:14527 RHEA-COMP:17342 RHEA:23680 RHEA:13065</td><td style = \"text-align: left;\">SITE 104..105; /note=&quot;Cleavage; by viral protease NS3&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; SITE 122..123; /note=&quot;Cleavage; by host signal peptidase&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; SITE 139; /note=&quot;Fetal microcephaly&quot;; SITE 215..216; /note=&quot;Cleavage; by host furin&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; SITE 290..291; /note=&quot;Cleavage; by host signal peptidase&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 794..795; /note=&quot;Cleavage; by host signal peptidase&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 1146..1147; /note=&quot;Cleavage; by host&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 1372..1373; /note=&quot;Cleavage; by viral protease NS3&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 1502..1503; /note=&quot;Cleavage; by autolysis&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 1958; /note=&quot;Involved in NS3 ATPase and RTPase activities&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P14335&quot;; SITE 1961; /note=&quot;Involved in NS3 ATPase and RTPase activities&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P14335&quot;; SITE 2119..2120; /note=&quot;Cleavage; by autolysis&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 2246..2247; /note=&quot;Cleavage; by viral protease NS3&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 2269..2270; /note=&quot;Cleavage; by host signal peptidase&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 2520..2521; /note=&quot;Cleavage; by viral protease NS3&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P06935&quot;; SITE 2533; /note=&quot;mRNA cap binding&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2536; /note=&quot;mRNA cap binding; via carbonyl oxygen&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2537; /note=&quot;mRNA cap binding&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2539; /note=&quot;mRNA cap binding; via carbonyl oxygen&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2544; /note=&quot;mRNA cap binding&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2548; /note=&quot;mRNA cap binding&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2581; /note=&quot;Essential for 2&apos;-O-methyltransferase activity&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2666; /note=&quot;Essential for 2&apos;-O-methyltransferase and N-7 methyltransferase activity&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2670; /note=&quot;mRNA cap binding&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2702; /note=&quot;Essential for 2&apos;-O-methyltransferase activity&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2733; /note=&quot;mRNA cap binding&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2735; /note=&quot;mRNA cap binding&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;; SITE 2738; /note=&quot;Essential for 2&apos;-O-methyltransferase activity&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00924&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;4Fe-4S;Activation of host autophagy by virus;ATP-binding;Capsid protein;Clathrin-mediated endocytosis of virus by host;Cleavage on pair of basic residues;Disulfide bond;Fusion of virus membrane with host endosomal membrane;Fusion of virus membrane with host membrane;Glycoprotein;GTP-binding;Helicase;Host cytoplasm;Host endoplasmic reticulum;Host membrane;Host nucleus;Host-virus interaction;Hydrolase;Inhibition of host innate immune response by virus;Inhibition of host interferon signaling pathway by virus;Inhibition of host STAT1 by virus;Inhibition of host STAT2 by virus;Inhibition of host TYK2 by virus;Iron;Iron-sulfur;Isopeptide bond;Membrane;Metal-binding;Methyltransferase;mRNA capping;mRNA processing;Nucleotide-binding;Nucleotidyltransferase;Phosphoprotein;Protease;RNA-binding;RNA-directed RNA polymerase;S-adenosyl-L-methionine;Secreted;Serine protease;Suppressor of RNA silencing;Transcription;Transcription regulation;Transferase;Transmembrane;Transmembrane helix;Ubl conjugation;Viral attachment to host cell;Viral envelope protein;Viral immunoevasion;Viral penetration into host cytoplasm;Viral RNA replication;Virion;Virus endocytosis by host;Virus entry into host cell;Zinc</td><td style = \"text-align: left;\">KW-0002; KW-0004; KW-1072; KW-0067; KW-0167; KW-1165; KW-0165; KW-1015; KW-1170; KW-1168; KW-0325; KW-0342; KW-0347; KW-1035; KW-1038; KW-1043; KW-1048; KW-0945; KW-0378; KW-1090; KW-1114; KW-1105; KW-1106; KW-1112; KW-0408; KW-0411; KW-1017; KW-0472; KW-0479; KW-0489; KW-0506; KW-0507; KW-0547; KW-0548; KW-0597; KW-0645; KW-0694; KW-0696; KW-0949; KW-0964; KW-0720; KW-0941; KW-0804; KW-0805; KW-0808; KW-0812; KW-1133; KW-0832; KW-1161; KW-0261; KW-0899; KW-1162; KW-0693; KW-0946; KW-1164; KW-1160; KW-0862</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI0004592572</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY (6); DOMAIN (5); FUNCTION (14); PTM (7); SIMILARITY (1); SUBCELLULAR LOCATION (10); SUBUNIT (10); WEB RESOURCE (1)</td><td style = \"text-align: left;\">Active site (7); Beta strand (102); Binding site (23); Chain (13); Cross-link (2); Disulfide bond (10); Domain (5); Glycosylation (4); Helix (82); Intramembrane (4); Modified residue (1); Motif (2); Mutagenesis (5); Peptide (1); Propeptide (1); Region (7); Site (28); Topological domain (22); Transmembrane (17); Turn (26)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBUNIT: [Capsid protein C]: Homodimer. {ECO:0000250|UniProtKB:P17763}.; SUBUNIT: [Protein prM]: Forms heterodimers with envelope protein E in the endoplasmic reticulum and Golgi (By similarity). Interacts with non-structural protein 2A (By similarity). {ECO:0000250|UniProtKB:A0A142I5B9, ECO:0000250|UniProtKB:P17763}.; SUBUNIT: [Envelope protein E]: Homodimer; in the endoplasmic reticulum and Golgi (By similarity). Interacts with host TYRO3, AXL and DC-SIGN proteins (PubMed:26085147). Interacts with non-structural protein 2A (By similarity). Interacts with host HAVCR1; this interaction likely mediates virus attachment to host cell (By similarity). Interacts with host NCAM1 (By similarity). Interacts with host HSPA5 (By similarity). {ECO:0000250|UniProtKB:A0A142I5B9, ECO:0000250|UniProtKB:P17763, ECO:0000269|PubMed:26085147}.; SUBUNIT: [Non-structural protein 1]: Homodimer; Homohexamer when secreted (By similarity). Interacts with host TBK1 (By similarity). Interacts with host USP8 (By similarity). Interacts with envelope protein E (By similarity). {ECO:0000250|UniProtKB:P17763, ECO:0000250|UniProtKB:Q32ZE1}.; SUBUNIT: [Non-structural protein 2A]: Interacts with the structural protein prM/E complex, and the NS2B/NS3 protease complex. {ECO:0000250|UniProtKB:A0A142I5B9}.; SUBUNIT: [Serine protease subunit NS2B]: Forms a heterodimer with serine protease NS3 (By similarity). May form homooligomers (By similarity). Interacts with human SPCS1 (By similarity). Interacts with non-structural protein 2A (By similarity). {ECO:0000250|UniProtKB:A0A142I5B9, ECO:0000250|UniProtKB:P17763, ECO:0000250|UniProtKB:Q32ZE1}.; SUBUNIT: [Serine protease NS3]: Forms a heterodimer with NS2B (By similarity). Interacts with NS4B (By similarity). Interacts with unphosphorylated RNA-directed RNA polymerase NS5; this interaction stimulates RNA-directed RNA polymerase NS5 guanylyltransferase activity (By similarity). Interacts with non-structural protein 2A (By similarity). Interacts with host SHFL; this interaction promotes NS3 degradation via a lysosome-dependent pathway (By similarity). {ECO:0000250|UniProtKB:A0A142I5B9, ECO:0000250|UniProtKB:P17763, ECO:0000250|UniProtKB:Q32ZE1}.; SUBUNIT: [Non-structural protein 4A]: May interact with host ANKLE2; the interaction may cause defects in brain development, such as microcephaly (PubMed:30550790). May interact with host SRPRA and SEC61G (PubMed:30550790). {ECO:0000269|PubMed:30550790}.; SUBUNIT: [Non-structural protein 4B]: Interacts with serine protease NS3. Interacts with NS1 (By similarity). {ECO:0000250|UniProtKB:P17763}.; SUBUNIT: [RNA-directed RNA polymerase NS5]: Homodimer; dimerization may negatively regulate the GTase activity, a crucial step in the capping process (PubMed:30951555). Interacts with host STAT2; this interaction inhibits the phosphorylation of the latter, and, when all viral proteins are present (polyprotein), targets STAT2 for degradation (PubMed:30550790, PubMed:32699085). Interacts with host TBK1 and IKBKE; these interactions lead to the inhibition of the host RIG-I signaling pathway (By similarity). Interacts with host PAF1 complex; the interaction may prevent the recruitment of the host PAF1 complex to interferon-responsive genes, and thus reduces the immune response (PubMed:30550790). Interacts with serine protease NS3 (By similarity). Interacts with host KPNA2 (By similarity). {ECO:0000250|UniProtKB:P17763, ECO:0000250|UniProtKB:Q32ZE1, ECO:0000269|PubMed:30550790, ECO:0000269|PubMed:30951555, ECO:0000269|PubMed:32699085}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">clathrin-dependent endocytosis of virus by host cell [GO:0075512]; fusion of virus membrane with host endosome membrane [GO:0039654]; induction by virus of host autophagy [GO:0039520]; proteolysis [GO:0006508]; suppression by virus of host JAK-STAT cascade via inhibition of host TYK2 activity [GO:0039574]; suppression by virus of host JAK-STAT cascade via inhibition of STAT1 activity [GO:0039563]; suppression by virus of host JAK-STAT cascade via inhibition of STAT2 activity [GO:0039564]; suppression by virus of host transcription [GO:0039653]; suppression by virus of host type I interferon-mediated signaling pathway [GO:0039502]; viral RNA genome replication [GO:0039694]; virion attachment to host cell [GO:0019062]</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; host cell endoplasmic reticulum membrane [GO:0044167]; host cell nucleus [GO:0042025]; host cell perinuclear region of cytoplasm [GO:0044220]; integral component of membrane [GO:0016021]; viral capsid [GO:0019028]; viral envelope [GO:0019031]; virion membrane [GO:0055036]</td><td style = \"text-align: left;\">4 iron, 4 sulfur cluster binding [GO:0051539]; ATP binding [GO:0005524]; ATP hydrolysis activity [GO:0016887]; double-stranded RNA binding [GO:0003725]; exogenous protein binding [GO:0140272]; GTP binding [GO:0005525]; metal ion binding [GO:0046872]; mRNA (guanine-N7-)-methyltransferase activity [GO:0004482]; mRNA (nucleoside-2&apos;-O-)-methyltransferase activity [GO:0004483]; protein dimerization activity [GO:0046983]; RNA helicase activity [GO:0003724]; RNA-directed 5&apos;-3&apos; RNA polymerase activity [GO:0003968]; serine-type endopeptidase activity [GO:0004252]; structural molecule activity [GO:0005198]</td><td style = \"text-align: left;\">GO:0003724; GO:0003725; GO:0003968; GO:0004252; GO:0004482; GO:0004483; GO:0005198; GO:0005524; GO:0005525; GO:0005576; GO:0006508; GO:0016021; GO:0016887; GO:0019028; GO:0019031; GO:0019062; GO:0039502; GO:0039520; GO:0039563; GO:0039564; GO:0039574; GO:0039653; GO:0039654; GO:0039694; GO:0042025; GO:0044167; GO:0044220; GO:0046872; GO:0046983; GO:0051539; GO:0055036; GO:0075512; GO:0140272</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; host cell endoplasmic reticulum membrane [GO:0044167]; host cell nucleus [GO:0042025]; host cell perinuclear region of cytoplasm [GO:0044220]; integral component of membrane [GO:0016021]; viral capsid [GO:0019028]; viral envelope [GO:0019031]; virion membrane [GO:0055036]; 4 iron, 4 sulfur cluster binding [GO:0051539]; ATP binding [GO:0005524]; ATP hydrolysis activity [GO:0016887]; double-stranded RNA binding [GO:0003725]; exogenous protein binding [GO:0140272]; GTP binding [GO:0005525]; metal ion binding [GO:0046872]; mRNA (guanine-N7-)-methyltransferase activity [GO:0004482]; mRNA (nucleoside-2&apos;-O-)-methyltransferase activity [GO:0004483]; protein dimerization activity [GO:0046983]; RNA helicase activity [GO:0003724]; RNA-directed 5&apos;-3&apos; RNA polymerase activity [GO:0003968]; serine-type endopeptidase activity [GO:0004252]; structural molecule activity [GO:0005198]; clathrin-dependent endocytosis of virus by host cell [GO:0075512]; fusion of virus membrane with host endosome membrane [GO:0039654]; induction by virus of host autophagy [GO:0039520]; proteolysis [GO:0006508]; suppression by virus of host JAK-STAT cascade via inhibition of host TYK2 activity [GO:0039574]; suppression by virus of host JAK-STAT cascade via inhibition of STAT1 activity [GO:0039563]; suppression by virus of host JAK-STAT cascade via inhibition of STAT2 activity [GO:0039564]; suppression by virus of host transcription [GO:0039653]; suppression by virus of host type I interferon-mediated signaling pathway [GO:0039502]; viral RNA genome replication [GO:0039694]; virion attachment to host cell [GO:0019062]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MUTAGEN 444; /note=&quot;N-&gt;Q: Improves attachment, assembly and infectivity in cell culture. Attenuates the virus in mouse and mosquitoes.&quot;; /evidence=&quot;ECO:0000269|PubMed:29091758&quot;; MUTAGEN 2545; /note=&quot;Y-&gt;A: Complete loss of dimer formation and about three times increased polymerase activity; when associated with S-2548 and A-2549.&quot;; /evidence=&quot;ECO:0000269|PubMed:30951555&quot;; MUTAGEN 2548; /note=&quot;K-&gt;S: Complete loss of dimer formation and about three times increased polymerase activity; when associated with A-2545 and A-2549.&quot;; /evidence=&quot;ECO:0000269|PubMed:30951555&quot;; MUTAGEN 2549; /note=&quot;K-&gt;A: Complete loss of dimer formation and about three times increased polymerase activity; when associated with A-2545 and S-2548.&quot;; /evidence=&quot;ECO:0000269|PubMed:30951555&quot;; MUTAGEN 2772; /note=&quot;K-&gt;R: Loss of sumoylation and more than 80% loss of binding to host STAT2.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699085&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">INTRAMEM 1473..1493; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; INTRAMEM 2196..2216; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; INTRAMEM 2255..2269; /note=&quot;Helical; Note=Signal for NS4B&quot;; /evidence=&quot;ECO:0000305&quot;; INTRAMEM 2308..2328; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: [Capsid protein C]: Virion {ECO:0000250|UniProtKB:P17763}. Host nucleus {ECO:0000250|UniProtKB:P17763}. Host cytoplasm {ECO:0000250|UniProtKB:P06935}. Host cytoplasm, host perinuclear region {ECO:0000250|UniProtKB:P06935}.; SUBCELLULAR LOCATION: [Peptide pr]: Secreted {ECO:0000250|UniProtKB:P17763}.; SUBCELLULAR LOCATION: [Small envelope protein M]: Virion membrane {ECO:0000250|UniProtKB:P17763}; Multi-pass membrane protein {ECO:0000255}. Host endoplasmic reticulum membrane {ECO:0000250|UniProtKB:P17763}; Multi-pass membrane protein {ECO:0000255}.; SUBCELLULAR LOCATION: [Envelope protein E]: Virion membrane {ECO:0000250|UniProtKB:P17763}; Multi-pass membrane protein {ECO:0000255}. Host endoplasmic reticulum membrane {ECO:0000250|UniProtKB:P17763}; Multi-pass membrane protein {ECO:0000255}.; SUBCELLULAR LOCATION: [Non-structural protein 1]: Secreted {ECO:0000250|UniProtKB:P17763}. Host endoplasmic reticulum membrane {ECO:0000250|UniProtKB:Q32ZE1}; Peripheral membrane protein {ECO:0000250|UniProtKB:Q32ZE1}; Lumenal side {ECO:0000250|UniProtKB:P17763}. Note=Located in RE-derived vesicles hosting the replication complex. {ECO:0000250|UniProtKB:Q9Q6P4}.; SUBCELLULAR LOCATION: [Non-structural protein 2A]: Host endoplasmic reticulum membrane {ECO:0000250|UniProtKB:P17763}; Multi-pass membrane protein {ECO:0000250|UniProtKB:P17763}.; SUBCELLULAR LOCATION: [Serine protease NS3]: Host endoplasmic reticulum membrane {ECO:0000255|PROSITE-ProRule:PRU00860}; Peripheral membrane protein {ECO:0000255|PROSITE-ProRule:PRU00860}; Cytoplasmic side {ECO:0000255|PROSITE-ProRule:PRU00860}. Note=Remains non-covalently associated to serine protease subunit NS2B. {ECO:0000255|PROSITE-ProRule:PRU00860}.; SUBCELLULAR LOCATION: [Non-structural protein 4A]: Host endoplasmic reticulum membrane {ECO:0000250|UniProtKB:P17763}; Multi-pass membrane protein {ECO:0000250|UniProtKB:P17763}. Note=Located in RE-associated vesicles hosting the replication complex. {ECO:0000250|UniProtKB:P17763}.; SUBCELLULAR LOCATION: [Non-structural protein 4B]: Host endoplasmic reticulum membrane {ECO:0000250|UniProtKB:P17763}; Multi-pass membrane protein {ECO:0000250|UniProtKB:P17763}. Note=Located in RE-derived vesicles hosting the replication complex. {ECO:0000250|UniProtKB:Q9Q6P4}.; SUBCELLULAR LOCATION: [RNA-directed RNA polymerase NS5]: Host endoplasmic reticulum membrane {ECO:0000250|UniProtKB:Q32ZE1}; Peripheral membrane protein {ECO:0000250|UniProtKB:Q32ZE1}; Cytoplasmic side {ECO:0000250|UniProtKB:Q32ZE1}. Host nucleus {ECO:0000269|PubMed:32699085}. Note=Located in RE-associated vesicles hosting the replication complex. NS5 protein is mainly localized in the nucleus rather than in ER vesicles. {ECO:0000250|UniProtKB:P17763}.</td><td style = \"text-align: left;\">TOPO_DOM 1..104; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 126..249; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 270..274; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 291..745; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 768..773; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 795..1177; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1199..1220; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1242..1270; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1292..1295; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1317..1345; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1367..1373; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1395..1397; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1419..1472; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1494..2170; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2192..2195; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2217..2218; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2240..2254; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2270..2307; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2329..2344; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2366..2375; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2397..2441; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2463..3423; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;</td><td style = \"text-align: left;\">TRANSMEM 105..125; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 250..269; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 275..290; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000305&quot;; TRANSMEM 746..767; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 774..794; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1178..1198; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1221..1241; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1271..1291; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1296..1316; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1346..1366; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1374..1394; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1398..1418; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 2171..2191; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 2219..2239; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 2345..2365; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 2376..2396; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 2442..2462; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"text-align: left;\">CHAIN 1..3423; /note=&quot;Genome polyprotein&quot;; /id=&quot;PRO_0000443018&quot;; CHAIN 1..104; /note=&quot;Capsid protein C&quot;; /id=&quot;PRO_0000443019&quot;; CHAIN 123..290; /note=&quot;Protein prM&quot;; /id=&quot;PRO_0000443021&quot;; CHAIN 123..215; /note=&quot;Peptide pr&quot;; /id=&quot;PRO_0000443022&quot;; CHAIN 216..290; /note=&quot;Small envelope protein M&quot;; /id=&quot;PRO_0000443023&quot;; CHAIN 291..794; /note=&quot;Envelope protein E&quot;; /id=&quot;PRO_0000443024&quot;; CHAIN 795..1146; /note=&quot;Non-structural protein 1&quot;; /id=&quot;PRO_0000443025&quot;; CHAIN 1147..1372; /note=&quot;Non-structural protein 2A&quot;; /id=&quot;PRO_0000443026&quot;; CHAIN 1373..1502; /note=&quot;Serine protease subunit NS2B&quot;; /id=&quot;PRO_0000443027&quot;; CHAIN 1503..2119; /note=&quot;Serine protease NS3&quot;; /id=&quot;PRO_0000443028&quot;; CHAIN 2120..2246; /note=&quot;Non-structural protein 4A&quot;; /id=&quot;PRO_0000443029&quot;; CHAIN 2270..2520; /note=&quot;Non-structural protein 4B&quot;; /id=&quot;PRO_0000443031&quot;; CHAIN 2521..3423; /note=&quot;RNA-directed RNA polymerase NS5&quot;; /id=&quot;PRO_0000443032&quot;</td><td style = \"text-align: left;\">CROSSLNK 328; /note=&quot;Glycyl lysine isopeptide (Lys-Gly) (interchain with G-Cter in ubiquitin)&quot;; /evidence=&quot;ECO:0000250|UniProtKB:A0A142I5B9&quot;; CROSSLNK 571; /note=&quot;Glycyl lysine isopeptide (Lys-Gly) (interchain with G-Cter in ubiquitin)&quot;; /evidence=&quot;ECO:0000250|UniProtKB:A0A142I5B9&quot;</td><td style = \"text-align: left;\">DISULFID 350..406; /evidence=&quot;ECO:0000250|UniProtKB:P17763&quot;; DISULFID 382..411; /evidence=&quot;ECO:0000250|UniProtKB:P17763&quot;; DISULFID 480..581; /evidence=&quot;ECO:0000250|UniProtKB:P17763&quot;; DISULFID 598..629; /evidence=&quot;ECO:0000250|UniProtKB:P17763&quot;; DISULFID 798..809; /evidence=&quot;ECO:0000250|UniProtKB:Q9Q6P4&quot;; DISULFID 849..937; /evidence=&quot;ECO:0000250|UniProtKB:Q9Q6P4&quot;; DISULFID 973..1017; /evidence=&quot;ECO:0000250|UniProtKB:Q9Q6P4&quot;; DISULFID 1074..1123; /evidence=&quot;ECO:0000250|UniProtKB:Q9Q6P4&quot;; DISULFID 1085..1106; /evidence=&quot;ECO:0000250|UniProtKB:Q9Q6P4&quot;; DISULFID 1107..1110; /evidence=&quot;ECO:0000250|UniProtKB:Q9Q6P4&quot;</td><td style = \"text-align: left;\">CARBOHYD 192; /note=&quot;N-linked (GlcNAc...) asparagine; by host&quot;; /evidence=&quot;ECO:0000255&quot;; CARBOHYD 444; /note=&quot;N-linked (GlcNAc...) asparagine; by host&quot;; /evidence=&quot;ECO:0000269|PubMed:27093288, ECO:0000269|PubMed:27338953, ECO:0000269|PubMed:27882950, ECO:0000269|PubMed:29091758&quot;; CARBOHYD 924; /note=&quot;N-linked (GlcNAc...) asparagine; by host&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;; CARBOHYD 1001; /note=&quot;N-linked (GlcNAc...) asparagine; by host&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q32ZE1&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">MOD_RES 2576; /note=&quot;Phosphoserine&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P03314&quot;</td><td style = \"text-align: left;\">PEPTIDE 2247..2269; /note=&quot;Peptide 2k&quot;; /id=&quot;PRO_0000443030&quot;</td><td style = \"text-align: left;\">PTM: [Genome polyprotein]: Specific enzymatic cleavages in vivo yield mature proteins. Cleavages in the lumen of endoplasmic reticulum are performed by host signal peptidase, whereas cleavages in the cytoplasmic side are performed by serine protease NS3. Signal cleavage at the 2K-4B site requires a prior NS3 protease-mediated cleavage at the 4A-2K site. {ECO:0000250|UniProtKB:P17763}.; PTM: [Protein prM]: Cleaved in post-Golgi vesicles by a host furin, releasing the mature small envelope protein M, and peptide pr. This cleavage is incomplete as up to 30% of viral particles still carry uncleaved prM. {ECO:0000250|UniProtKB:P17763}.; PTM: [Envelope protein E]: N-glycosylation plays a role in virulence in mammalian and mosquito hosts, but may have no effect on neurovirulence. {ECO:0000269|PubMed:29091758}.; PTM: [Envelope protein E]: Ubiquitination by host TRIM7 promotes virus attachment and fusion of the virus and the host endosome membrane. {ECO:0000250|UniProtKB:A0A142I5B9}.; PTM: [Non-structural protein 1]: N-glycosylated. The excreted form is glycosylated, which is required for efficient secretion of the protein from infected cells. {ECO:0000250|UniProtKB:P17763}.; PTM: [RNA-directed RNA polymerase NS5]: Phosphorylated on serines residues. This phosphorylation may trigger NS5 nuclear localization. {ECO:0000250|UniProtKB:P17763}.; PTM: [RNA-directed RNA polymerase NS5]: Sumoylated, required for regulating IFN induced interferon stimulated genes/ISGs. {ECO:0000269|PubMed:32699085}.</td><td style = \"text-align: left;\">PROPEP 105..122; /note=&quot;ER anchor for capsid protein C, removed in mature form by serine protease NS3&quot;; /id=&quot;PRO_0000443020&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Electron microscopy (13); X-ray crystallography (30)</td><td style = \"text-align: left;\">STRAND 234..236; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; STRAND 271..273; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; STRAND 297..303; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 305..307; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 310..316; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 320..324; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 326..328; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; STRAND 331..340; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 345..362; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 380..390; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 399..418; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 425..433; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 454..460; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 461..463; /evidence=&quot;ECO:0007829|PDB:5LBS&quot;; STRAND 465..469; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 474..481; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 483..486; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; STRAND 491..496; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 499..504; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 515..517; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 533..537; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 543..547; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 562..568; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 571..573; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 578..584; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; STRAND 602..606; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 612..614; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; STRAND 616..622; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 628..630; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 633..637; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 644..648; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 650..652; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 662..669; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 672..682; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 686..692; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; STRAND 721..723; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; STRAND 1068..1072; /evidence=&quot;ECO:0007829|PDB:7BSD&quot;; STRAND 1078..1081; /evidence=&quot;ECO:0007829|PDB:7BSD&quot;; STRAND 1093..1095; /evidence=&quot;ECO:0007829|PDB:7BSD&quot;; STRAND 1104..1109; /evidence=&quot;ECO:0007829|PDB:7BSD&quot;; STRAND 1115..1119; /evidence=&quot;ECO:0007829|PDB:7BSD&quot;; STRAND 1122..1125; /evidence=&quot;ECO:0007829|PDB:7BSD&quot;; STRAND 1422..1429; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1522..1531; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1534..1544; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1547..1550; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1560..1562; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1565..1567; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1569..1573; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1578..1584; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1593..1595; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1597..1601; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1609..1613; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1616..1620; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1623..1627; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1640..1642; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1648..1651; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; STRAND 1678..1680; /evidence=&quot;ECO:0007829|PDB:5JMT&quot;; STRAND 1691..1694; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1721..1727; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1743..1745; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1760..1764; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1774..1776; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1782..1787; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1813..1817; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1835..1839; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1848..1850; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1861..1864; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1885..1888; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1906..1910; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1923..1927; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1930..1937; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1941..1949; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 1971..1975; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 2081..2083; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 2089..2091; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 2095..2098; /evidence=&quot;ECO:0007829|PDB:5Y6N&quot;; STRAND 2102..2104; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; STRAND 2553..2556; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2569..2571; /evidence=&quot;ECO:0007829|PDB:5NJV&quot;; STRAND 2595..2600; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2617..2623; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2644..2647; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2661..2665; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2697..2704; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2725..2727; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2739..2742; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2765..2767; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2772..2775; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; STRAND 2822..2832; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 2973..2975; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 2996..2998; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3025..3027; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3046..3049; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3095..3104; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3110..3118; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3180..3183; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3186..3189; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3210..3213; /evidence=&quot;ECO:0007829|PDB:6LD3&quot;; STRAND 3233..3239; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3245..3250; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; STRAND 3326..3329; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;</td><td style = \"text-align: left;\">HELIX 222..225; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 243..253; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 257..264; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 274..285; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 373..376; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; HELIX 422..424; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; HELIX 439..441; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; HELIX 471..473; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; HELIX 488..490; /evidence=&quot;ECO:0007829|PDB:5LCV&quot;; HELIX 505..509; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; HELIX 529..532; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; HELIX 552..558; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; HELIX 585..587; /evidence=&quot;ECO:0007829|PDB:5LBS&quot;; HELIX 696..714; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 717..720; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 731..744; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 752..767; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 779..789; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; HELIX 1552..1555; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; HELIX 1634..1636; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; HELIX 1685..1687; /evidence=&quot;ECO:0007829|PDB:5Y6N&quot;; HELIX 1706..1717; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1728..1737; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1765..1773; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1794..1808; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1852..1855; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1868..1880; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1893..1902; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1912..1915; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1952..1959; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 1983..1985; /evidence=&quot;ECO:0007829|PDB:5Y6N&quot;; HELIX 1988..1996; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 2002..2004; /evidence=&quot;ECO:0007829|PDB:5Y6N&quot;; HELIX 2011..2016; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 2028..2039; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 2045..2053; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 2062..2064; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 2069..2071; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 2099..2101; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 2105..2115; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; HELIX 2528..2538; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2541..2547; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2558..2565; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2578..2587; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2606..2612; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2641..2643; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2652..2654; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2674..2692; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2709..2722; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2749..2762; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; HELIX 2795..2808; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 2844..2848; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 2851..2855; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 2857..2860; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 2869..2878; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 2889..2906; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 2917..2924; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 2943..2947; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 2950..2964; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3000..3010; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3012..3015; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3021..3024; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3033..3044; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3059..3062; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3065..3071; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3072..3077; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3080..3093; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3106..3108; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3128..3147; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3154..3157; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3163..3178; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3194..3198; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3201..3205; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3225..3227; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3253..3260; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3270..3287; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3292..3304; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3331..3339; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3355..3357; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3363..3368; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3376..3383; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; HELIX 3385..3396; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;</td><td style = \"text-align: left;\">TURN 237..241; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; TURN 265..267; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; TURN 292..295; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; TURN 391..394; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; TURN 448..450; /evidence=&quot;ECO:0007829|PDB:5LCV&quot;; TURN 539..541; /evidence=&quot;ECO:0007829|PDB:5LBS&quot;; TURN 559..561; /evidence=&quot;ECO:0007829|PDB:5LBV&quot;; TURN 591..594; /evidence=&quot;ECO:0007829|PDB:5LCV&quot;; TURN 639..641; /evidence=&quot;ECO:0007829|PDB:5KVE&quot;; TURN 657..661; /evidence=&quot;ECO:0007829|PDB:6PLK&quot;; TURN 745..747; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; TURN 776..778; /evidence=&quot;ECO:0007829|PDB:6CO8&quot;; TURN 1574..1577; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; TURN 1652..1654; /evidence=&quot;ECO:0007829|PDB:7M1V&quot;; TURN 1702..1705; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; TURN 1738..1740; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; TURN 1788..1790; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; TURN 1890..1892; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; TURN 1938..1940; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; TURN 2021..2024; /evidence=&quot;ECO:0007829|PDB:6S0J&quot;; TURN 2548..2551; /evidence=&quot;ECO:0007829|PDB:5KQR&quot;; TURN 2603..2605; /evidence=&quot;ECO:0007829|PDB:5GP1&quot;; TURN 2809..2812; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; TURN 2907..2909; /evidence=&quot;ECO:0007829|PDB:6LD2&quot;; TURN 3016..3019; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;; TURN 3340..3342; /evidence=&quot;ECO:0007829|PDB:6LD1&quot;</td><td style = \"text-align: left;\">24903869; 26085147; 27524440; 29091758; 28592527; 30550790; 32699085; 27866982; 27475895; 27633330; 27882950; 27093288; 27338953; 27172988; 27033547; 28357511; 28031359; 28300075; 28067914; 28487506; 29423037; 29958768; 30951555</td><td style = \"text-align: left;\">2018-01-31</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">2</td><td style = \"text-align: right;\">1</td><td style = \"text-align: left;\">A0A024SC78</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">CUTI1_HYPJR</td><td style = \"text-align: left;\">Cutinase, EC 3.1.1.74</td><td style = \"text-align: left;\">M419DRAFT_76732</td><td style = \"text-align: left;\">Hypocrea jecorina (strain ATCC 56765 / BCRC 32924 / NRRL 11460 / Rut C-30) (Trichoderma reesei)</td><td style = \"text-align: right;\">248</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">M419DRAFT_76732</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">1344414</td><td style = \"text-align: left;\">UP000024376: Unassembled WGS sequence</td><td style = \"text-align: left;\">Hypocrea jecorina (species), Trichoderma (genus), Hypocreaceae (family), Hypocreales (order), Hypocreomycetidae (subclass), Sordariomycetes (class), sordariomyceta (no rank), leotiomyceta (no rank), Pezizomycotina (subphylum), saccharomyceta (no rank), Ascomycota (phylum), Dikarya (subkingdom), Fungi (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">51453 (species), 5543 (genus), 5129 (family), 5125 (order), 222543 (subclass), 147550 (class), 715989 (no rank), 716546 (no rank), 147538 (subphylum), 716545 (no rank), 4890 (phylum), 451864 (subkingdom), 4751 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">25924</td><td style = \"text-align: left;\">MASS SPECTROMETRY: Mass=23748; Method=MALDI; Evidence={ECO:0000269|PubMed:25219509};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MRSLAILTTLLAGHAFAYPKPAPQSVNRRDWPSINEFLSELAKVMPIGDTITAACDLISDGEDAAASLFGISETENDPCGDVTVLFARGTCDPGNVGVLVGPWFFDSLQTALGSRTLGVKGVPYPASVQDFLSGSVQNGINMANQIKSVLQSCPNTKLVLGGYSQGSMVVHNAASNLDAATMSKISAVVLFGDPYYGKPVANFDAAKTLVVCHDGDNICQGGDIILLPHLTYAEDADTAAAFVVPLVS</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 164; /note=&quot;Nucleophile&quot;; /evidence=&quot;ECO:0000269|PubMed:25219509, ECO:0007744|PDB:4PSE&quot;; ACT_SITE 216; /evidence=&quot;ECO:0000250|UniProtKB:P00590&quot;; ACT_SITE 229; /note=&quot;Proton donor/acceptor&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P00590&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=cutin + H2O = cutin monomers.; EC=3.1.1.74; Evidence={ECO:0000269|PubMed:25219509};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3.1.1.74</td><td style = \"text-align: left;\">FUNCTION: Catalyzes the hydrolysis of complex carboxylic polyesters found in the cell wall of plants (PubMed:25219509). Degrades cutin, a macromolecule that forms the structure of the plant cuticle (PubMed:25219509). {ECO:0000269|PubMed:25219509}.</td><td style = \"text-align: left;\">ACTIVITY REGULATION: Weakly inhibited by n-undecyl phosphonate (C11Y4) (PubMed:25219509). Activity unaffected by paraoxon (PubMed:25219509). {ECO:0000269|PubMed:25219509}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  pH dependence: Optimum pH is 4-7. {ECO:0000269|PubMed:25219509};</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SITE 90; /note=&quot;Transition state stabilizer&quot;; /evidence=&quot;ECO:0000269|PubMed:25219509, ECO:0007744|PDB:4PSE&quot;; SITE 165; /note=&quot;Transition state stabilizer&quot;; /evidence=&quot;ECO:0000269|PubMed:25219509, ECO:0007744|PDB:4PSE&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;Disulfide bond;Hydrolase;Secreted;Serine esterase;Signal</td><td style = \"text-align: left;\">KW-0002; KW-1015; KW-0378; KW-0964; KW-0719; KW-0732</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI0001EF138F</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); BIOPHYSICOCHEMICAL PROPERTIES (1); CATALYTIC ACTIVITY (1); DOMAIN (1); FUNCTION (1); MASS SPECTROMETRY (1); PTM (1); SIMILARITY (1); SUBCELLULAR LOCATION (1)</td><td style = \"text-align: left;\">Active site (3); Chain (1); Disulfide bond (3); Propeptide (1); Region (1); Signal (1); Site (2)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">extracellular region [GO:0005576]</td><td style = \"text-align: left;\">cutinase activity [GO:0050525]</td><td style = \"text-align: left;\">GO:0005576; GO:0050525</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; cutinase activity [GO:0050525]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Secreted {ECO:0000255|RuleBase:RU361263}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 29..248; /note=&quot;Cutinase&quot;; /evidence=&quot;ECO:0000305&quot;; /id=&quot;PRO_5005101809&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISULFID 55..91; /evidence=&quot;ECO:0000269|PubMed:25219509, ECO:0007744|PDB:4PSC, ECO:0007744|PDB:4PSD, ECO:0007744|PDB:4PSE&quot;; DISULFID 79..153; /evidence=&quot;ECO:0000269|PubMed:25219509, ECO:0007744|PDB:4PSC, ECO:0007744|PDB:4PSD, ECO:0007744|PDB:4PSE&quot;; DISULFID 212..219; /evidence=&quot;ECO:0000269|PubMed:25219509, ECO:0007744|PDB:4PSC, ECO:0007744|PDB:4PSD, ECO:0007744|PDB:4PSE&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PTM: The 2 disulfide bonds play a critical role in holding the catalytic residues in juxta-position; reduction of the disulfide bridges results in the complete inactivation of the enzyme. {ECO:0000250|UniProtKB:P11373}.</td><td style = \"text-align: left;\">PROPEP 18..28; /evidence=&quot;ECO:0000269|PubMed:25219509&quot;; /id=&quot;PRO_0000455277&quot;</td><td style = \"text-align: left;\">SIGNAL 1..17; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">X-ray crystallography (3)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">25219509</td><td style = \"text-align: left;\">2022-05-25</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">3</td><td style = \"text-align: right;\">2</td><td style = \"text-align: left;\">A0A024SH76</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">GUX2_HYPJR</td><td style = \"text-align: left;\">Exoglucanase 2, EC 3.2.1.91 (1,4-beta-cellobiohydrolase) (Cellobiohydrolase 6A, Cel6A) (Exocellobiohydrolase II, CBHII) (Exoglucanase II)</td><td style = \"text-align: left;\">cbh2 M419DRAFT_122470</td><td style = \"text-align: left;\">Hypocrea jecorina (strain ATCC 56765 / BCRC 32924 / NRRL 11460 / Rut C-30) (Trichoderma reesei)</td><td style = \"text-align: right;\">471</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">M419DRAFT_122470</td><td style = \"text-align: left;\">cbh2</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">1344414</td><td style = \"text-align: left;\">UP000024376: Unassembled WGS sequence</td><td style = \"text-align: left;\">Hypocrea jecorina (species), Trichoderma (genus), Hypocreaceae (family), Hypocreales (order), Hypocreomycetidae (subclass), Sordariomycetes (class), sordariomyceta (no rank), leotiomyceta (no rank), Pezizomycotina (subphylum), saccharomyceta (no rank), Ascomycota (phylum), Dikarya (subkingdom), Fungi (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">51453 (species), 5543 (genus), 5129 (family), 5125 (order), 222543 (subclass), 147550 (class), 715989 (no rank), 716546 (no rank), 147538 (subphylum), 716545 (no rank), 4890 (phylum), 451864 (subkingdom), 4751 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">49653</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MIVGILTTLATLATLAASVPLEERQACSSVWGQCGGQNWSGPTCCASGSTCVYSNDYYSQCLPGAASSSSSTRAASTTSRVSPTTSRSSSATPPPGSTTTRVPPVGSGTATYSGNPFVGVTPWANAYYASEVSSLAIPSLTGAMATAAAAVAKVPSFMWLDTLDKTPLMEQTLADIRTANKNGGNYAGQFVVYDLPDRDCAALASNGEYSIADGGVAKYKNYIDTIRQIVVEYSDIRTLLVIEPDSLANLVTNLGTPKCANAQSAYLECINYAVTQLNLPNVAMYLDAGHAGWLGWPANQDPAAQLFANVYKNASSPRALRGLATNVANYNGWNITSPPSYTQGNAVYNEKLYIHAIGPLLANHGWSNAFFITDQGRSGKQPTGQQQWGDWCNVIGTGFGIRPSANTGDSLLDSFVWVKPGGECDGTSDSSAPRFDSHCALPDALQPAPQAGAWFQAYFVQLLTNANPSFL</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 245; /note=&quot;Proton donor&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=Hydrolysis of (1-&gt;4)-beta-D-glucosidic linkages in cellulose and cellotetraose, releasing cellobiose from the non-reducing ends of the chains.; EC=3.2.1.91; Evidence={ECO:0000250|UniProtKB:P07987};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3.2.1.91</td><td style = \"text-align: left;\">FUNCTION: Exocellobiohydrolases (CBH) that catalyzes the hydrolysis of 1,4-beta-D-glucosidic bonds in cellulose to release the disaccharide cellobiose. The degradation of cellulose involves an interplay between different cellulolytic enzymes. Hydrolysis starts with endoglucanases (EGs), which cut internal beta-1,4-glucosidic bonds in cellulose to reduce the polymerization degree of the substrate and create new chain ends for exocellobiohydrolases (CBHs). The CBHs release the disaccharide cellobiose from the non-reducing end of the cellulose polymer chain. Finally, beta-1,4-glucosidases hydrolyze the cellobiose and other short cello-oligosaccharides into glucose units. {ECO:0000250|UniProtKB:P07987}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SITE 38; /note=&quot;Not glycosylated&quot;; /evidence=&quot;ECO:0000269|PubMed:12499406&quot;; SITE 199; /note=&quot;Transition state stabilizer that also modulates the pKa of Asp-245 and may act as a proton acceptor through a water chain&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Carbohydrate metabolism;Cellulose degradation;Disulfide bond;Glycoprotein;Glycosidase;Hydrolase;Polysaccharide degradation;Pyrrolidone carboxylic acid;Secreted;Signal</td><td style = \"text-align: left;\">KW-0119; KW-0136; KW-1015; KW-0325; KW-0326; KW-0378; KW-0624; KW-0873; KW-0964; KW-0732</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000002B8F5</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY (1); DOMAIN (1); FUNCTION (1); PTM (1); SIMILARITY (1); SUBCELLULAR LOCATION (1)</td><td style = \"text-align: left;\">Active site (1); Chain (1); Disulfide bond (2); Domain (1); Glycosylation (9); Modified residue (1); Propeptide (1); Region (3); Signal (1); Site (2)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">cellulose catabolic process [GO:0030245]</td><td style = \"text-align: left;\">extracellular region [GO:0005576]</td><td style = \"text-align: left;\">cellulose 1,4-beta-cellobiosidase activity [GO:0016162]; cellulose binding [GO:0030248]</td><td style = \"text-align: left;\">GO:0005576; GO:0016162; GO:0030245; GO:0030248</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; cellulose 1,4-beta-cellobiosidase activity [GO:0016162]; cellulose binding [GO:0030248]; cellulose catabolic process [GO:0030245]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Secreted {ECO:0000250|UniProtKB:P07987}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 25..471; /note=&quot;Exoglucanase 2&quot;; /id=&quot;PRO_5005101780&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISULFID 200..259; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; DISULFID 392..439; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;</td><td style = \"text-align: left;\">CARBOHYD 111; /note=&quot;O-linked (Man...) threonine&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; CARBOHYD 121; /note=&quot;O-linked (Man...) threonine&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; CARBOHYD 130; /note=&quot;O-linked (Man...) serine&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; CARBOHYD 133; /note=&quot;O-linked (Man...) serine&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; CARBOHYD 134; /note=&quot;O-linked (Man...) serine&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; CARBOHYD 139; /note=&quot;O-linked (Man...) serine&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; CARBOHYD 146; /note=&quot;O-linked (Man...) threonine&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; CARBOHYD 313; /note=&quot;N-linked (GlcNAc) asparagine&quot;; /evidence=&quot;ECO:0000269|PubMed:12499406&quot;; CARBOHYD 334; /note=&quot;N-linked (GlcNAc...) (high mannose) asparagine&quot;; /evidence=&quot;ECO:0000269|PubMed:12499406&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">MOD_RES 25; /note=&quot;Pyrrolidone carboxylic acid&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PTM: Asn-334 contains mainly a high-mannose-type glycan (Hex(7-9)GlcNAc(2)) in a 3:1 ration with a single GlcNAc. Asn-313 was primarily unglycosylated with a small fraction (18%) bearing a single GlcNAc at this site. {ECO:0000269|PubMed:12499406}.</td><td style = \"text-align: left;\">PROPEP 19..24; /evidence=&quot;ECO:0000250|UniProtKB:P07987&quot;; /id=&quot;PRO_0000441279&quot;</td><td style = \"text-align: left;\">SIGNAL 1..18; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">12499406</td><td style = \"text-align: left;\">2017-08-30</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">4</td><td style = \"text-align: right;\">3</td><td style = \"text-align: left;\">A0A026W182</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">ORCO_OOCBI</td><td style = \"text-align: left;\">Odorant receptor coreceptor</td><td style = \"text-align: left;\">Orco X777_12371</td><td style = \"text-align: left;\">Ooceraea biroi (Clonal raider ant) (Cerapachys biroi)</td><td style = \"text-align: right;\">478</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">X777_12371</td><td style = \"text-align: left;\">Orco</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">2015173</td><td style = \"text-align: left;\">UP000053097: Unassembled WGS sequence</td><td style = \"text-align: left;\">Ooceraea (genus), Dorylinae (subfamily), Formicidae (family), Formicoidea (superfamily), Aculeata (infraorder), Apocrita (suborder), Hymenoptera (order), Endopterygota (cohort), Neoptera (infraclass), Pterygota (subclass), Dicondylia (no rank), Insecta (class), Hexapoda (subphylum), Pancrustacea (no rank), Mandibulata (no rank), Arthropoda (phylum), Panarthropoda (no rank), Ecdysozoa (no rank), Protostomia (no rank), Bilateria (no rank), Eumetazoa (no rank), Metazoa (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">2015172 (genus), 213859 (subfamily), 36668 (family), 2153479 (superfamily), 7434 (infraorder), 7400 (suborder), 7399 (order), 33392 (cohort), 33340 (infraclass), 7496 (subclass), 85512 (no rank), 50557 (class), 6960 (subphylum), 197562 (no rank), 197563 (no rank), 6656 (phylum), 88770 (no rank), 1206794 (no rank), 33317 (no rank), 33213 (no rank), 6072 (no rank), 33208 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">54109</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MMKMKQQGLVADLLPNIRVMKTFGHFVFNYYNDNSSKYLHKVYCCVNLFMLLLQFGLCAVNLIVESADVDDLTANTITLLFFTHSIVKICYFAIRSKYFYRTWAIWNNPNSHPLFAESNARYHAIALKKMRLLLFLVGGTTMLAAVAWTVLTFFEHPIRKIVDPVTNETEIIELPQLLIRSFYPFDAGKGITHVLVLVYQFYWVLFMLIDANSLDVLFCSWLLFACEQLQHLKQIMKPLMELSATLDTVVPNSSELFKAGSADHLRDGDNPPPPPPPQSDNMLDLDLRNIYSNRQDFTATFRPTAGMTFNGGVGPNGLTKKQEALVRSAIKYWVERHKHIVRLVTAVGDAYGFALLLHMLTTTITLTLLAYQATKVNGINVYAASTIGYILYTFGQVFLFCIFGNRLIEESTSVMEAAYSCHWYDGSEEAKTFVQIVCQQCQKAMSISGAKFFTVSLDLFASVLGAVVTYFMVLVQLK</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">FUNCTION: Odorant coreceptor which complexes with conventional odorant receptors (ORs) to form odorant-sensing units, providing sensitive and prolonged odorant signaling and calcium permeability (By similarity). Obligate coreceptor of all odorant receptors (By similarity). Orco is a universal and integral part of the functional odorant receptor, involved in the dendritic localization of other olfactory receptors. Can form functional ion channels in the absence of an odor-binding odorant receptor (By similarity). Plays a central role in the perception of olfactory stimuli in ants and is essential for ant social organization (PubMed:28802042). Required for pheromone sensing (PubMed:28802042). Also required for the development and maintenance of odorant receptor neurons (ORNs) and of antennal lobe glomeruli (PubMed:28802042). {ECO:0000250|UniProtKB:Q7QCC7, ECO:0000269|PubMed:28802042}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Behavior;Cell membrane;Glycoprotein;Membrane;Olfaction;Receptor;Reference proteome;Sensory transduction;Transducer;Transmembrane;Transmembrane helix</td><td style = \"text-align: left;\">KW-0085; KW-1003; KW-0325; KW-0472; KW-0552; KW-0675; KW-1185; KW-0716; KW-0807; KW-0812; KW-1133</td><td style = \"text-align: left;\">MISCELLANEOUS: In contrast to other ant species, C.biroi ants reproduce via parthenogenesis, enabling stable germline modifications from the clonal progeny of injected individuals without laboratory crosses. {ECO:0000269|PubMed:28802042}.</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000454DD45</td><td style = \"text-align: left;\">DISRUPTION PHENOTYPE (1); FUNCTION (1); MISCELLANEOUS (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); SUBUNIT (1); TISSUE SPECIFICITY (1); WEB RESOURCE (1)</td><td style = \"text-align: left;\">Chain (1); Glycosylation (1); Region (1); Topological domain (8); Transmembrane (7)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBUNIT: Heterodimer with conventional odorant receptors (ORs). {ECO:0000250|UniProtKB:Q7QCC7}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Present in antennae (at protein level). {ECO:0000269|PubMed:28802042}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">antennal development [GO:0007469]; detection of chemical stimulus involved in sensory perception of smell [GO:0050911]; detection of pheromone [GO:0043695]; olfactory behavior [GO:0042048]; response to pheromone [GO:0019236]; signal transduction [GO:0007165]; social behavior [GO:0035176]</td><td style = \"text-align: left;\">integral component of membrane [GO:0016021]; plasma membrane [GO:0005886]</td><td style = \"text-align: left;\">odorant binding [GO:0005549]; olfactory receptor activity [GO:0004984]</td><td style = \"text-align: left;\">GO:0004984; GO:0005549; GO:0005886; GO:0007165; GO:0007469; GO:0016021; GO:0019236; GO:0035176; GO:0042048; GO:0043695; GO:0050911</td><td style = \"text-align: left;\">integral component of membrane [GO:0016021]; plasma membrane [GO:0005886]; odorant binding [GO:0005549]; olfactory receptor activity [GO:0004984]; antennal development [GO:0007469]; detection of chemical stimulus involved in sensory perception of smell [GO:0050911]; detection of pheromone [GO:0043695]; olfactory behavior [GO:0042048]; response to pheromone [GO:0019236]; signal transduction [GO:0007165]; social behavior [GO:0035176]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISRUPTION PHENOTYPE: Impaired behavior and fitness, probably due to loss of pheromone sensing. Ants also display gross neuroanatomical defects in the antennal lobe, with a dramatic decrease in the number of antennal lobe glomeruli. {ECO:0000269|PubMed:28802042}.</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Cell membrane {ECO:0000305}; Multi-pass membrane protein {ECO:0000255}.</td><td style = \"text-align: left;\">TOPO_DOM 1..43; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 65..73; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 95..133; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 155..190; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 212..349; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 371..382; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 404..454; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 476..478; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;</td><td style = \"text-align: left;\">TRANSMEM 44..64; /note=&quot;Helical; Name=1&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 74..94; /note=&quot;Helical; Name=2&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 134..154; /note=&quot;Helical; Name=3&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 191..211; /note=&quot;Helical; Name=4&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 350..370; /note=&quot;Helical; Name=5&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 383..403; /note=&quot;Helical; Name=6&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 455..475; /note=&quot;Helical; Name=7&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"text-align: left;\">CHAIN 1..478; /note=&quot;Odorant receptor coreceptor&quot;; /id=&quot;PRO_0000442003&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CARBOHYD 167; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">24508170; 28802042</td><td style = \"text-align: left;\">2017-10-25</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">5</td><td style = \"text-align: right;\">4</td><td style = \"text-align: left;\">A0A044RE18</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">BLI_ONCVO</td><td style = \"text-align: left;\">Endoprotease bli, EC 3.4.21.75 (Blisterase)</td><td style = \"text-align: left;\">Bli</td><td style = \"text-align: left;\">Onchocerca volvulus</td><td style = \"text-align: right;\">693</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Bli</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">6282</td><td style = \"text-align: left;\">UP000024404: Unassembled WGS sequence</td><td style = \"text-align: left;\">Onchocerca (genus), Onchocercidae (family), Filarioidea (superfamily), Spiruromorpha (infraorder), Spirurina (suborder), Rhabditida (order), Chromadorea (class), Nematoda (phylum), Ecdysozoa (no rank), Protostomia (no rank), Bilateria (no rank), Eumetazoa (no rank), Metazoa (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">6281 (genus), 6296 (family), 6295 (superfamily), 2072716 (infraorder), 6274 (suborder), 6236 (order), 119089 (class), 6231 (phylum), 1206794 (no rank), 33317 (no rank), 33213 (no rank), 6072 (no rank), 33208 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">76800</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MYWQLVRILVLFDCLQKILAIEHDSICIADVDDACPEPSHTVMRLRERNDKKAHLIAKQHGLEIRGQPFLDGKSYFVTHISKQRSRRRKREIISRLQEHPDILSIEEQRPRVRRKRDFLYPDIAHELAGSSTNIRHTGLISNTEPRIDFIQHDAPVLPFPDPLYKEQWYLNNGAQGGFDMNVQAAWLLGYAGRNISVSILDDGIQRDHPDLAANYDPLASTDINGHDDDPTPQDDGDNKHGTRCAGEVASIAGNVYCGVGVAFHAKIGGVRMLDGPVSDSVEAASLSLNRHHIDIYSASWGPEDDGRTFDGPGPLAREAFYRGVKAGRGGKGSIFVWASGNGGSRQDSCSADGYTTSVYTLSVSSATIDNRSPWYLEECPSTIATTYSSANMNQPAIITVDVPHGCTRSHTGTSASAPLAAGIIALALEANPNLTWRDMQHIVLRTANPVPLLNNPGWSVNGVGRRINNKFGYGLMDAGALVKLALIWKTVPEQHICTYDYKLEKPNPRPITGNFQMNFSLEVNGCESGTPVLYLEHVQVLATFRFGKRGDLKLTLFSPRGTSSVLLPPRPQDFNSNGIHKWPFLSVQTWGEDPRGKWTLMVESVSTNRNVGGTFHDWSLLLYGTAEPAQPNDPRHSSVVPSSVSAESPFDRITQHIASQEKKKKQRDSRDWQPKKVENKKSLLVSAQPELRV</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CONFLICT 661..693; /note=&quot;EKKKKQRDSRDWQPKKVENKKSLLVSAQPELRV -&gt; VHGPQSFFFSFLCLYNS (in Ref. 1; AAO12507)&quot;; /evidence=&quot;ECO:0000305&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 201; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU01240&quot;; ACT_SITE 240; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU01240&quot;; ACT_SITE 414; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU01240&quot;</td><td style = \"text-align: left;\">BINDING 161; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 202; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 210; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 222; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;2&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 227; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;2&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 229; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;2&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 237..238; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 251; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 254; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 256; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 258; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 282; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 299..304; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 304; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;3&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 310; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 338..341; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 347; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;3&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 352; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 354; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 377; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /ligand_label=&quot;3&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; BINDING 414; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=Release of mature proteins from their proproteins by cleavage of -Arg-Xaa-Yaa-Arg-|-Zaa- bonds, where Xaa can be any amino acid and Yaa is Arg or Lys. Releases albumin, complement component C3 and von Willebrand factor from their respective precursors.; EC=3.4.21.75; Evidence={ECO:0000269|PubMed:12855702};</td><td style = \"text-align: left;\">COFACTOR: Name=Ca(2+); Xref=ChEBI:CHEBI:29108; Evidence={ECO:0000269|PubMed:12855702}; Note=Binds 3 calcium ions per subunit. {ECO:0000250|UniProtKB:P09958};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3.4.21.75</td><td style = \"text-align: left;\">FUNCTION: Serine endoprotease which cleaves substrates at the RX(K/R)R consensus motif. {ECO:0000269|PubMed:12855702}.</td><td style = \"text-align: left;\">ACTIVITY REGULATION: Inhibited by the propeptide before the second cleavage. Inhibited by ethylenediaminetetraacetic acid (EDTA), ZnSO(4) and chloroketone DEC-RVKR-CMK. {ECO:0000269|PubMed:12855702}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  pH dependence: Optimum pH is 7.0. Active from pH 7.0 to 8.5. {ECO:0000269|PubMed:12855702};</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SITE 84..85; /note=&quot;Cleavage, second; by autolysis&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P09958&quot;; SITE 116..117; /note=&quot;Cleavage, first; by autolysis&quot;; /evidence=&quot;ECO:0000305|PubMed:12855702&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Autocatalytic cleavage;Calcium;Cleavage on pair of basic residues;Disulfide bond;Glycoprotein;Hydrolase;Metal-binding;Protease;Reference proteome;Secreted;Serine protease;Signal;Zymogen</td><td style = \"text-align: left;\">KW-0068; KW-0106; KW-0165; KW-1015; KW-0325; KW-0378; KW-0479; KW-0645; KW-1185; KW-0964; KW-0720; KW-0732; KW-0865</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00043BA27E</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); BIOPHYSICOCHEMICAL PROPERTIES (1); CATALYTIC ACTIVITY (1); COFACTOR (1); FUNCTION (1); PTM (2); SIMILARITY (1); SUBCELLULAR LOCATION (1)</td><td style = \"text-align: left;\">Active site (3); Binding site (21); Chain (1); Compositional bias (3); Disulfide bond (3); Domain (2); Glycosylation (3); Propeptide (1); Region (2); Sequence conflict (1); Signal (1); Site (2)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">dibasic protein processing [GO:0090472]; zymogen activation [GO:0031638]</td><td style = \"text-align: left;\">extracellular region [GO:0005576]</td><td style = \"text-align: left;\">metal ion binding [GO:0046872]; serine-type endopeptidase activity [GO:0004252]</td><td style = \"text-align: left;\">GO:0004252; GO:0005576; GO:0031638; GO:0046872; GO:0090472</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; metal ion binding [GO:0046872]; serine-type endopeptidase activity [GO:0004252]; dibasic protein processing [GO:0090472]; zymogen activation [GO:0031638]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Secreted {ECO:0000305|PubMed:12855702}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 117..693; /note=&quot;Endoprotease bli&quot;; /evidence=&quot;ECO:0000305|PubMed:12855702&quot;; /id=&quot;PRO_5004295169&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISULFID 257..406; /evidence=&quot;ECO:0000250|UniProtKB:P23188&quot;; DISULFID 349..379; /evidence=&quot;ECO:0000250|UniProtKB:P23188&quot;; DISULFID 497..526; /evidence=&quot;ECO:0000250|UniProtKB:P23188&quot;</td><td style = \"text-align: left;\">CARBOHYD 194; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 433; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 518; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PTM: N-glycosylated. {ECO:0000269|PubMed:12855702}.; PTM: The inhibition peptide, which plays the role of an intramolecular chaperone, is probably autocatalytically removed in the endoplasmic reticulum (ER) and remains non-covalently bound as a potent autoinhibitor. Probably following transport to the trans Golgi, a second cleavage within the inhibition propeptide results in propeptide dissociation and bli activation. {ECO:0000269|PubMed:12855702}.</td><td style = \"text-align: left;\">PROPEP 21..116; /note=&quot;Inhibition peptide&quot;; /evidence=&quot;ECO:0000305|PubMed:12855702&quot;; /id=&quot;PRO_0000439880&quot;</td><td style = \"text-align: left;\">SIGNAL 1..20; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">12855702</td><td style = \"text-align: left;\">2017-05-10</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">6</td><td style = \"text-align: right;\">5</td><td style = \"text-align: left;\">A0A059TC02</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">CCR1_PETHY</td><td style = \"text-align: left;\">Cinnamoyl-CoA reductase 1, PhCCR1, EC 1.2.1.44 (Coniferylaldehyde synthase, EC 1.2.1.-)</td><td style = \"text-align: left;\">CCR1</td><td style = \"text-align: left;\">Petunia hybrida (Petunia)</td><td style = \"text-align: right;\">333</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CCR1</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">4102</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Petunia (genus), Petunioideae (subfamily), Solanaceae (family), Solanales (order), lamiids (no rank), asterids (no rank), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">4101 (genus), 424555 (subfamily), 4070 (family), 4069 (order), 91888 (no rank), 71274 (no rank), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">36886</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MRSVSGQVVCVTGAGGFIASWLVKILLEKGYTVRGTVRNPDDPKNGHLRELEGAKERLTLCKADLLDYQSLREAINGCDGVFHTASPVTDDPEQMVEPAVIGTKNVINAAAEANVRRVVFTSSIGAVYMDPNRDPETVVDETCWSDPDFCKNTKNWYCYGKMVAEQAAWEEAKEKGVDLVVINPVLVQGPLLQTTVNASVLHILKYLTGSAKTYANSVQAYVDVKDVALAHILLYETPEASGRYLCAESVLHRGDVVEILSKFFPEYPIPTKCSDVTKPRVKPYKFSNQKLKDLGLEFTPVKQCLYETVKSLQEKGHLPIPTQKDEPIIRIQP</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 161; /note=&quot;Proton donor&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12068&quot;</td><td style = \"text-align: left;\">BINDING 13..19; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;; BINDING 38; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;; BINDING 44; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;; BINDING 64..65; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;; BINDING 84..86; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;; BINDING 157; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;; BINDING 161; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;; BINDING 184..187; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;; BINDING 199; /ligand=&quot;NADP(+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:58349&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1S&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=(E)-coniferaldehyde + CoA + NADP(+) = (E)-feruloyl-CoA + H(+) + NADPH; Xref=Rhea:RHEA:64648, ChEBI:CHEBI:15378, ChEBI:CHEBI:16547, ChEBI:CHEBI:57287, ChEBI:CHEBI:57783, ChEBI:CHEBI:58349, ChEBI:CHEBI:87305; EC=1.2.1.44; Evidence={ECO:0000269|PubMed:24985707, ECO:0000269|PubMed:25217505}; PhysiologicalDirection=right-to-left; Xref=Rhea:RHEA:64650; Evidence={ECO:0000269|PubMed:24985707, ECO:0000269|PubMed:25217505}; CATALYTIC ACTIVITY: Reaction=(E)-4-coumaraldehyde + CoA + NADP(+) = (E)-4-coumaroyl-CoA + H(+) + NADPH; Xref=Rhea:RHEA:64652, ChEBI:CHEBI:15378, ChEBI:CHEBI:28353, ChEBI:CHEBI:57287, ChEBI:CHEBI:57783, ChEBI:CHEBI:58349, ChEBI:CHEBI:85008; EC=1.2.1.44; Evidence={ECO:0000269|PubMed:24985707, ECO:0000269|PubMed:25217505}; PhysiologicalDirection=right-to-left; Xref=Rhea:RHEA:64654; Evidence={ECO:0000269|PubMed:24985707, ECO:0000269|PubMed:25217505}; CATALYTIC ACTIVITY: Reaction=(E)-sinapaldehyde + CoA + NADP(+) = (E)-sinapoyl-CoA + H(+) + NADPH; Xref=Rhea:RHEA:64656, ChEBI:CHEBI:15378, ChEBI:CHEBI:27949, ChEBI:CHEBI:57287, ChEBI:CHEBI:57393, ChEBI:CHEBI:57783, ChEBI:CHEBI:58349; EC=1.2.1.44; Evidence={ECO:0000269|PubMed:24985707, ECO:0000269|PubMed:25217505}; PhysiologicalDirection=right-to-left; Xref=Rhea:RHEA:64658; Evidence={ECO:0000269|PubMed:24985707, ECO:0000269|PubMed:25217505}; CATALYTIC ACTIVITY: Reaction=(E)-cinnamaldehyde + CoA + NADP(+) = (E)-cinnamoyl-CoA + H(+) + NADPH; Xref=Rhea:RHEA:10620, ChEBI:CHEBI:15378, ChEBI:CHEBI:16731, ChEBI:CHEBI:57252, ChEBI:CHEBI:57287, ChEBI:CHEBI:57783, ChEBI:CHEBI:58349; EC=1.2.1.44; Evidence={ECO:0000250|UniProtKB:Q9S9N9}; PhysiologicalDirection=right-to-left; Xref=Rhea:RHEA:10622; Evidence={ECO:0000250|UniProtKB:Q9S9N9};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">1.2.1.-; 1.2.1.44</td><td style = \"text-align: left;\">FUNCTION: Involved in the latter stages of lignin biosynthesis (PubMed:24985707). Catalyzes one of the last steps of monolignol biosynthesis, the conversion of cinnamoyl-CoAs into their corresponding cinnamaldehydes (PubMed:25217505, PubMed:24985707). Mediates the conversion of feruloyl CoA to coniferylaldehyde (PubMed:25217505, PubMed:24985707). Also active toward p-coumaroyl-CoA and sinapoyl-CoA (PubMed:25217505, PubMed:24985707). Involved in the production of floral volatile phenylpropanoids in flowers of fragrant cultivars (e.g. cv. Mitchell and cv. V26) from cinnamic acid, a common precursor with the anthocyanin biosynthesis pathway involved in flower pigmentation (PubMed:24985707). {ECO:0000269|PubMed:24985707, ECO:0000269|PubMed:25217505}.</td><td style = \"text-align: left;\">ACTIVITY REGULATION: Inhibited by sodium iodide-mediated oxidation. {ECO:0000269|PubMed:25217505}.</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  Kinetic parameters: KM=208.6 uM for p-coumaroyl-CoA {ECO:0000269|PubMed:25217505}; KM=307.6 uM for feruloyl-CoA {ECO:0000269|PubMed:25217505}; KM=270.3 uM for sinapoyl-CoA {ECO:0000269|PubMed:25217505}; Vmax=1235.7 nmol/sec/mg enzyme with p-coumaroyl-CoA as substrate {ECO:0000269|PubMed:25217505}; Vmax=5713 nmol/sec/mg enzyme with feruloyl-CoA as substrate {ECO:0000269|PubMed:25217505}; Vmax=3384.7 nmol/sec/mg enzyme with sinapoyl-CoA as substrate {ECO:0000269|PubMed:25217505}; Note=kcat is 1.2 sec(-1) with p-coumaroyl-CoA as substrate (PubMed:25217505). kcat is 5.8 sec(-1) with feruloyl-CoA as substrate (PubMed:25217505). kcat is 3.4 sec(-1) with sinapoyl-CoA as substrate (PubMed:25217505). {ECO:0000269|PubMed:25217505};</td><td style = \"text-align: left;\">PATHWAY: Aromatic compound metabolism; phenylpropanoid biosynthesis. {ECO:0000269|PubMed:24985707, ECO:0000269|PubMed:25217505}.</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  pH dependence: Optimum pH is 6. {ECO:0000269|PubMed:25217505};</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:64648 RHEA:64652 RHEA:64656 RHEA:10620</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;Cytoplasm;Disulfide bond;Lignin biosynthesis;NADP;Nucleotide-binding;Oxidoreductase;Phenylpropanoid metabolism</td><td style = \"text-align: left;\">KW-0002; KW-0963; KW-1015; KW-0438; KW-0521; KW-0547; KW-0560; KW-0587</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00049A4F3A</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); BIOPHYSICOCHEMICAL PROPERTIES (1); CATALYTIC ACTIVITY (4); DEVELOPMENTAL STAGE (1); DISRUPTION PHENOTYPE (1); FUNCTION (1); INDUCTION (1); PATHWAY (1); PTM (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Active site (1); Beta strand (14); Binding site (9); Chain (1); Disulfide bond (1); Helix (14); Mutagenesis (3); Turn (2)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DEVELOPMENTAL STAGE: Highly expressed in the scent-producing corollas and tubes of two days old flowers, reaching a maximum at day three post-anthesis, and, to a lower extent, in other floral tissues (e.g. ovary, pistil, sepals and stamen). {ECO:0000269|PubMed:24985707}.</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Expressed in flowers, leaves and stems. {ECO:0000269|PubMed:24985707}.</td><td style = \"text-align: left;\">INDUCTION: Circadian-regulation with peak levels occurring in flowers during the light period, in the afternoon. {ECO:0000269|PubMed:24985707}.</td><td style = \"text-align: left;\">circadian rhythm [GO:0007623]; green leaf volatile biosynthetic process [GO:0010597]; lignin biosynthetic process [GO:0009809]; phenylpropanoid biosynthetic process [GO:0009699]</td><td style = \"text-align: left;\">cytoplasm [GO:0005737]</td><td style = \"text-align: left;\">cinnamoyl-CoA reductase activity [GO:0016621]; nucleotide binding [GO:0000166]</td><td style = \"text-align: left;\">GO:0000166; GO:0005737; GO:0007623; GO:0009699; GO:0009809; GO:0010597; GO:0016621</td><td style = \"text-align: left;\">cytoplasm [GO:0005737]; cinnamoyl-CoA reductase activity [GO:0016621]; nucleotide binding [GO:0000166]; circadian rhythm [GO:0007623]; green leaf volatile biosynthetic process [GO:0010597]; lignin biosynthetic process [GO:0009809]; phenylpropanoid biosynthetic process [GO:0009699]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISRUPTION PHENOTYPE: No visible effect on the emission and internal pools of phenylpropene and benzenoid compounds, but increased accumulation of feruloyl-CoA, p-coumaroyl-CoA and vanillin (PubMed:24985707). Reduced lignin content (PubMed:24985707). {ECO:0000269|PubMed:24985707}.</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MUTAGEN 150; /note=&quot;C-&gt;A,S: Increased activity.&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505&quot;; MUTAGEN 158; /note=&quot;C-&gt;A: Increased activity.&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505&quot;; MUTAGEN 158; /note=&quot;C-&gt;S: Reduced activity.&quot;; /evidence=&quot;ECO:0000269|PubMed:25217505&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:24985707}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 1..333; /note=&quot;Cinnamoyl-CoA reductase 1&quot;; /id=&quot;PRO_0000451497&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISULFID 150..158; /evidence=&quot;ECO:0000269|PubMed:25217505, ECO:0007744|PDB:4R1T&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PTM: The formation of a reversible disulfide bond reduces activity by perturbing the positioning of nearby catalytic residues. {ECO:0000269|PubMed:25217505}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">X-ray crystallography (2)</td><td style = \"text-align: left;\">STRAND 8..12; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 32..38; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 58..62; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 79..83; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 116..121; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 131..133; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 179..184; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 186..189; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 192..195; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 212..214; /evidence=&quot;ECO:0007829|PDB:4R1T&quot;; STRAND 218..223; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 242..246; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 249..252; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; STRAND 272..274; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;</td><td style = \"text-align: left;\">HELIX 17..28; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 43..45; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 46..49; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 54..57; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 68..75; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 92..112; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 124..126; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 147..152; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 156..175; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 198..207; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 224..236; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 253..263; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 289..293; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;; HELIX 301..314; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;</td><td style = \"text-align: left;\">TURN 3..6; /evidence=&quot;ECO:0007829|PDB:4R1T&quot;; TURN 13..15; /evidence=&quot;ECO:0007829|PDB:4R1S&quot;</td><td style = \"text-align: left;\">25217505; 24985707</td><td style = \"text-align: left;\">2020-12-02</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">7</td><td style = \"text-align: right;\">6</td><td style = \"text-align: left;\">A0A060A682</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">HAP2_TETTH</td><td style = \"text-align: left;\">Hapless 2 (Generative cell specific-1)</td><td style = \"text-align: left;\">HAP2 GCS1</td><td style = \"text-align: left;\">Tetrahymena thermophila</td><td style = \"text-align: right;\">742</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">HAP2</td><td style = \"text-align: left;\">GCS1</td><td style = \"text-align: right;\">5911</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Tetrahymena (genus), Tetrahymenidae (family), Tetrahymenina (suborder), Hymenostomatida (order), Oligohymenophorea (class), Intramacronucleata (subphylum), Ciliophora (phylum), Alveolata (no rank), Sar (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">5890 (genus), 291294 (family), 37093 (suborder), 31277 (order), 6020 (class), 431838 (subphylum), 5878 (phylum), 33630 (no rank), 2698737 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">82663</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MKFLAFGLIYFHFCILNRCEYITSSTIQKCYNSSNEPNNCSQKAVIVLSLENGQIANTEQVVATLNQLSDSGVNKQLQNSFIFEVTKSPVTALFPLIYLQDFNSQPLEQVIATTLFSCKDGFYDSSPTCKFQYDSKGQKILDSQGYCCYCSLSDILGMGNDLSRGKVCYALNLGAGSATAHCLKFSPLWYSAFKIQQYQLYFEVNINIYTVDSQNQKNLKQTLKLSTSNPTMKSSDNSTISKIIGTFTPTQPPADLSSYYLVKPSFPATDPRVLQGISSWMFVDKTMFTLDGTQCNKIGVSYSGFRQQSSSCSQPVGSCLQNQLENLYQSDLILLSQNKQPKYLLESQGNFNQVQFQGQTILQQGLSGSASTLITIEIDAAQIKFVTNLGIGCISQCSINNFESHSGNGKLVALVQNQGNYSAEFVLGFNCSSNVQPIQGQKLFLTANQLYNFNCSVSVNSDISAINNNCTINLYDAIGNQLDSKNILFNTTSTNHTSNQGNNTGQQQSSQEYKSSQSCSDKCSSFWSFWCYFSAGCIKEAFKSIASIAGVASALALVIFLAKNGYLVPIIRFLCCCCCKSKKKENEKNKDKTDKKSIQESCSYDRSCCSHSISQSYQVENKNKYKRSKIQRSFSSESCQDKSKKIINELSNLEETFEANKLYANIDKNSSIFEYFGFKKSFTFILYERNDILFLPQNSTILDMIGALQPQKGSYLAQKFLEIVNKNALKVVSTSPLYLLIE</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">FUNCTION: During fertilization, required for the formation of intercellular membrane pores and subsequent exchange of gametic pronuclei between cells. Probably initiates the formation of intercellular membrane pores by inserting part of its extracellular domain into the cell membrane of the adjoining cell in the mating pair. Mating requires the presence of HAP2 on at least one of the two cells. Mating efficiency is high when HAP2 is present on both cells, and is strongly reduced when HAP2 is present on only one of the two cells. {ECO:0000269|PubMed:25155508, ECO:0000269|PubMed:28238660}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Cell junction;Cell membrane;Disulfide bond;Fertilization;Lipid-binding;Membrane;Signal;Transmembrane;Transmembrane helix</td><td style = \"text-align: left;\">KW-0965; KW-1003; KW-1015; KW-0278; KW-0446; KW-0472; KW-0732; KW-0812; KW-1133</td><td style = \"text-align: left;\">MISCELLANEOUS: HAP2/GCS1 family members mediate membrane fusion between gametes in a broad range of eukaryotes, ranging from algae and higher plants to protozoans and cnidaria, suggesting they are derived from an ancestral gamete fusogen (PubMed:20080406). They function similar to viral fusogens, by inserting part of their extracellular domain into the lipid bilayer of an adjoining cell (PubMed:28238660). {ECO:0000269|PubMed:28238660, ECO:0000303|PubMed:20080406}.</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI0003F3AEBF</td><td style = \"text-align: left;\">DEVELOPMENTAL STAGE (1); DOMAIN (1); FUNCTION (1); MISCELLANEOUS (1); SIMILARITY (1); SUBCELLULAR LOCATION (1)</td><td style = \"text-align: left;\">Chain (1); Disulfide bond (7); Mutagenesis (5); Region (1); Signal (1); Topological domain (2); Transmembrane (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DEVELOPMENTAL STAGE: Detected in all seven mating types. {ECO:0000269|PubMed:25155508}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">fertilization [GO:0009566]; plasma membrane fusion [GO:0045026]; single fertilization [GO:0007338]</td><td style = \"text-align: left;\">cell-cell junction [GO:0005911]; integral component of membrane [GO:0016021]; plasma membrane [GO:0005886]</td><td style = \"text-align: left;\">fusogenic activity [GO:0140522]; lipid binding [GO:0008289]</td><td style = \"text-align: left;\">GO:0005886; GO:0005911; GO:0007338; GO:0008289; GO:0009566; GO:0016021; GO:0045026; GO:0140522</td><td style = \"text-align: left;\">cell-cell junction [GO:0005911]; integral component of membrane [GO:0016021]; plasma membrane [GO:0005886]; fusogenic activity [GO:0140522]; lipid binding [GO:0008289]; fertilization [GO:0009566]; plasma membrane fusion [GO:0045026]; single fertilization [GO:0007338]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MUTAGEN 131..133; /note=&quot;FQY-&gt;AAA: No effect.&quot;; /evidence=&quot;ECO:0000269|PubMed:28238660&quot;; MUTAGEN 147..148; /note=&quot;CC-&gt;SS: Loss of function in mediating cell fusion.&quot;; /evidence=&quot;ECO:0000269|PubMed:28238660&quot;; MUTAGEN 152..179; /note=&quot;Missing: Loss of function in mediating cell fusion.&quot;; /evidence=&quot;ECO:0000269|PubMed:28238660&quot;; MUTAGEN 164; /note=&quot;R-&gt;A: No effect.&quot;; /evidence=&quot;ECO:0000269|PubMed:28238660&quot;; MUTAGEN 171..173; /note=&quot;LNL-&gt;AAA: No effect.&quot;; /evidence=&quot;ECO:0000269|PubMed:28238660&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Cell membrane {ECO:0000305|PubMed:25155508, ECO:0000305|PubMed:28238660}; Single-pass type I membrane protein {ECO:0000255}. Cell junction {ECO:0000269|PubMed:25155508, ECO:0000269|PubMed:28238660}. Note=Detected at the mating junction. {ECO:0000269|PubMed:25155508, ECO:0000269|PubMed:28238660}.</td><td style = \"text-align: left;\">TOPO_DOM 20..540; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 562..742; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;</td><td style = \"text-align: left;\">TRANSMEM 541..561; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"text-align: left;\">CHAIN 20..742; /note=&quot;Hapless 2&quot;; /evidence=&quot;ECO:0000255&quot;; /id=&quot;PRO_5001584807&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISULFID 30..40; /evidence=&quot;ECO:0000250|UniProtKB:A4GRC6&quot;; DISULFID 118..147; /evidence=&quot;ECO:0000250|UniProtKB:A4GRC6&quot;; DISULFID 129..182; /evidence=&quot;ECO:0000250|UniProtKB:A4GRC6&quot;; DISULFID 148..312; /evidence=&quot;ECO:0000250|UniProtKB:A4GRC6&quot;; DISULFID 150..168; /evidence=&quot;ECO:0000250|UniProtKB:A4GRC6&quot;; DISULFID 295..319; /evidence=&quot;ECO:0000250|UniProtKB:A4GRC6&quot;; DISULFID 431..470; /evidence=&quot;ECO:0000250|UniProtKB:A4GRC6&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SIGNAL 1..19; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">25155508; 20080406; 28238660</td><td style = \"text-align: left;\">2017-05-10</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">8</td><td style = \"text-align: right;\">7</td><td style = \"text-align: left;\">A0A061ACU2</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">PIEZ1_CAEEL</td><td style = \"text-align: left;\">Piezo-type mechanosensitive ion channel component 1</td><td style = \"text-align: left;\">pezo-1 C10C5.1</td><td style = \"text-align: left;\">Caenorhabditis elegans</td><td style = \"text-align: right;\">2442</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">C10C5.1</td><td style = \"text-align: left;\">pezo-1</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">6239</td><td style = \"text-align: left;\">UP000001940: Chromosome IV</td><td style = \"text-align: left;\">Caenorhabditis (genus), Peloderinae (subfamily), Rhabditidae (family), Rhabditoidea (superfamily), Rhabditomorpha (infraorder), Rhabditina (suborder), Rhabditida (order), Chromadorea (class), Nematoda (phylum), Ecdysozoa (no rank), Protostomia (no rank), Bilateria (no rank), Eumetazoa (no rank), Metazoa (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">6237 (genus), 55885 (subfamily), 6243 (family), 55879 (superfamily), 2301119 (infraorder), 2301116 (suborder), 6236 (order), 119089 (class), 6231 (phylum), 1206794 (no rank), 33317 (no rank), 33213 (no rank), 6072 (no rank), 33208 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  Event=Alternative splicing, Alternative initiation; Named isoforms=12; Name=g {ECO:0000312|WormBase:C10C5.1g}; IsoId=A0A061ACU2-1; Sequence=Displayed; Name=a {ECO:0000312|WormBase:C10C5.1a}; IsoId=A0A061ACU2-2; Sequence=VSP_060811, VSP_060812; Name=b {ECO:0000312|WormBase:C10C5.1b}; IsoId=A0A061ACU2-3; Sequence=VSP_060810, VSP_060811, VSP_060812; Name=c {ECO:0000312|WormBase:C10C5.1c}; IsoId=A0A061ACU2-4; Sequence=VSP_060812; Name=d {ECO:0000312|WormBase:C10C5.1d}; IsoId=A0A061ACU2-5; Sequence=VSP_060810, VSP_060812; Name=e {ECO:0000312|WormBase:C10C5.1e}; IsoId=A0A061ACU2-6; Sequence=VSP_060811; Name=f {ECO:0000312|WormBase:C10C5.1f}; IsoId=A0A061ACU2-7; Sequence=VSP_060810, VSP_060811; Name=h {ECO:0000312|WormBase:C10C5.1h}; IsoId=A0A061ACU2-8; Sequence=VSP_060810; Name=i {ECO:0000312|WormBase:C10C5.1i}; IsoId=A0A061ACU2-9; Sequence=VSP_060808; Name=j {ECO:0000312|WormBase:C10C5.1j}; IsoId=A0A061ACU2-10; Sequence=VSP_060808, VSP_060813; Name=k {ECO:0000312|WormBase:C10C5.1k}; IsoId=A0A061ACU2-11; Sequence=VSP_060809; Name=l {ECO:0000312|WormBase:C10C5.1l}; IsoId=A0A061ACU2-12; Sequence=VSP_060807;</td><td style = \"text-align: left;\">VAR_SEQ 1..1404; /note=&quot;Missing (in isoform l)&quot;; /evidence=&quot;ECO:0000305&quot;; /id=&quot;VSP_060807&quot;; VAR_SEQ 1..804; /note=&quot;MTVPPLLKSCVVKLLLPAALLAAAIIRPSFLSIGYVLLALVSAVLPPIRKSLALPKLVGTFVIITFLFCLAVALGVGSYQISEQVVHKNDRTYICNRSDTTLFRSIGLVRFHPTGTFESTRAFLPEIIATSAALLTIIIVMFLSHRDEQLDVVGDVVTVRSESGREQRRQRKLAAIMWSAIGNSLRRLTNFVLFLFTAYVGIVKPSLSNSIYFLAFLFISTWWSTYTPLRHGVYNQIKKFLIFYSALHFLVLYTYQIPIVHHSWLPTGSFLPRLFGLTVLMDSSCPEWWKFPFVAPDFNDDDLIMKWPLYANPIVVLVFFYLTVAQYKFTRNGSREYIDDNEYGSSVHEERFVSAGTVETNVDDVGQLISISESTASAPSGRGRGNTLLLSNASSSANDDEQGRARSRSPLRNGEEQGSIPLRKVTSQVVDRNKLSNIFNTTAPGDKESAASKGMIAVMTFVIFHSYSIALTAMMTWALLYHSIFGLILLILTCILWIFRDTRKSSFAMAPIILMYIEFLLILQYFLSMDIHAEIGDPAWMNFVGIEWTTLPVHAVIILCVQTLLTLPVFLLLRLARREKFYESLSDYERQRRINSYGTFGASKTGAGGVAVAKFQDPKSRKFAAFVEYLSNKVSVYFIFVVSVVLLVVSTCFAPNFYNILFFALWALNLIYLKFSFRLYRGLAYAFWLTLTFYTSIVIIALYIYQFPGVSQWIIRNTSLSQEWLNAIGLVDFRAIGESGALFLQLLAPIALFVVTMLQLKFFHGPWSRATSPRRAENDPPTSTTEAAAVASTSGTQGRAHAAG -&gt; MLTKSIVSSSGDSGIYEDGCGIMPYIDDDDDLTPIIMARPAIGSGYTGGRNGWHRSPRQYLTNWWS (in isoform i and isoform j)&quot;; /evidence=&quot;ECO:0000305&quot;; /id=&quot;VSP_060808&quot;; VAR_SEQ 1..756; /note=&quot;Missing (in isoform k)&quot;; /evidence=&quot;ECO:0000305&quot;; /id=&quot;VSP_060809&quot;; VAR_SEQ 351..388; /note=&quot;Missing (in isoform b, isoform d, isoform f and isoform h)&quot;; /evidence=&quot;ECO:0000305&quot;; /id=&quot;VSP_060810&quot;; VAR_SEQ 442..443; /note=&quot;Missing (in isoform a, isoform b, isoform e and isoform f)&quot;; /evidence=&quot;ECO:0000305&quot;; /id=&quot;VSP_060811&quot;; VAR_SEQ 615..616; /note=&quot;Missing (in isoform a, isoform b, isoform c and isoform d)&quot;; /evidence=&quot;ECO:0000305&quot;; /id=&quot;VSP_060812&quot;; VAR_SEQ 1396; /note=&quot;H -&gt; HECMRIDEDDPFPYYDLRISSQDTENE (in isoform j)&quot;; /evidence=&quot;ECO:0000305&quot;; /id=&quot;VSP_060813&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">276793</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MTVPPLLKSCVVKLLLPAALLAAAIIRPSFLSIGYVLLALVSAVLPPIRKSLALPKLVGTFVIITFLFCLAVALGVGSYQISEQVVHKNDRTYICNRSDTTLFRSIGLVRFHPTGTFESTRAFLPEIIATSAALLTIIIVMFLSHRDEQLDVVGDVVTVRSESGREQRRQRKLAAIMWSAIGNSLRRLTNFVLFLFTAYVGIVKPSLSNSIYFLAFLFISTWWSTYTPLRHGVYNQIKKFLIFYSALHFLVLYTYQIPIVHHSWLPTGSFLPRLFGLTVLMDSSCPEWWKFPFVAPDFNDDDLIMKWPLYANPIVVLVFFYLTVAQYKFTRNGSREYIDDNEYGSSVHEERFVSAGTVETNVDDVGQLISISESTASAPSGRGRGNTLLLSNASSSANDDEQGRARSRSPLRNGEEQGSIPLRKVTSQVVDRNKLSNIFNTTAPGDKESAASKGMIAVMTFVIFHSYSIALTAMMTWALLYHSIFGLILLILTCILWIFRDTRKSSFAMAPIILMYIEFLLILQYFLSMDIHAEIGDPAWMNFVGIEWTTLPVHAVIILCVQTLLTLPVFLLLRLARREKFYESLSDYERQRRINSYGTFGASKTGAGGVAVAKFQDPKSRKFAAFVEYLSNKVSVYFIFVVSVVLLVVSTCFAPNFYNILFFALWALNLIYLKFSFRLYRGLAYAFWLTLTFYTSIVIIALYIYQFPGVSQWIIRNTSLSQEWLNAIGLVDFRAIGESGALFLQLLAPIALFVVTMLQLKFFHGPWSRATSPRRAENDPPTSTTEAAAVASTSGTQGRAHAAGDTLVKKLHKLANQTIELLWRFFEVHISKIVFVIIAIFIANNINALYIPLVILLSLAICLPSAADGIFSLFMCAYLFLVALSKMIYQLDIVPELSQIDRGVGADNCSHGNISMPEWFGLKKEVEGTEPIYMLFGVIVSIIALAFQSIVIYRQRHYRASLGLPESMRAKVFPDFHHSHFDRSLKNAIQFLIDYGFYKFGLEITMIAIGIDIFNRMDALAAIQCFWLVLFALNKRVFVRRIWVFYVIYMAILYPLQFFSYVGLPPDSCIEYPWSYWIPSYSDDARFNLSYLLNLSIYGVNWPSAYLIGDFFVLLLASCQLAVFRREGEDNDSIYNDGNFVIKPENPQYDFIDTKKSYVDYFKSFVFHYGHWITLMSTLAAGIAGTSLFALGYIIFTLTMLWSGNNLYVMNSTLRSFEHTLKRWNALLGYTLFTITMKVCLQIFGCVFLSWFDQSGGWGKTLCIVRQLFSITCVNNECHVLKELEDFSKACAVETKEGNIGFDVIALSFLVFQIRIFHSWYFQHCMVEYRSEVILANRGAVLKNQLIEKEMKEQNEQQKAKFNDIRRRTEAIRERYQKQIERGAAERDFEPVTYGHAKRAGDYYMFKYDPENDDLVEPVDSFVPEVDPKATAYDRLDPGQIMYAATAHDLDLAKTVQQVKKGDTIKDPDSRALIAVSEPEARKPGGTEETDGDEDEDNKDSKVESTAKFIQKMIASALDLCSVTLNKLCREHRYVGFVLSKEKQKLKSGHSESLSNTSRKLTDIRSAVDLPSLQLVQSANDVEKMETAVSVDWQQKSSATRLLNAVVNCIGAHTDILCYFFAIMTQVMTGGLITLPLPLMSLFWGNLSNPRPSKFFWVTMITYTECVIVIKFVCQFAFMPYNSITWRTEHQMDPMSLDKLFGVSQRDSFALWDIVLLFSLFFHRYMLRKLGLWKDANLTDTFTLKEEPRSASGSDTGSPKKIAQEPKVVVTQSDTLEGTSGGEIVIPSDPNAVSNMEELDCEPPIPEKQSGPIGRFIHQLFHPKFRYIRDLYPIMFGIDVICFLIMTFGYSAFGEGGSGNVLDDVKASRIPVTLVVMLVGMTLAIIIDRALYLRKSVVGKLIYQVLMIAFLHIWVFLVLPNMTRRSAISNHVAQALYVIKSCYFLVSAWQIRNGYPELCIGNLLTHSYGMTNMIAFKVFMNIPFLFELRTAIDWTWTDTSMPLFDFFNMENFYAHIFNIKCARQFEAAYPAPRGIPKGKLVKYMMGFPIIIGVVIFIFSPLLLWSLLNQIGTISMPEKVTLRISIEGYPPLYEMEAQGSNHDNAELGMIKPDQLASLNQALTDSYTTRDTNSILRSRMSVSYLKGYTYEDILIVRFRPESEIYWPISQDSRNAMIDKLSRNTSVNFEVSLEFTRPYDPNENAALKHSKSWLVPISLDMTIRAKIQSALRGDPGHPILIPQSIPAFIQVPNQGELTLPTSIGNTIINDGNPRINTTGMEKSDEARAWFDSLTLNLEQGKSQNEKMWIATSEHPGDQNAKLWIKTANTTYSGRPYLQVVGFIDRAFPSFLAKVFKGGVIAVYLSVILVVGRGLVRGIFTTSPSTVMFTELPNADHLLKICLDIYLVREAKDFMLEQDLFAKLIFLFRSPATLIEWTRMSKKKQE</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">FUNCTION: Pore-forming subunit of a mechanosensitive non-specific cation channel (By similarity). Generates currents characterized by a linear current-voltage relationship (By similarity). Plays a role in reproduction by positively regulating inter-tissue signaling to promote oocyte maturation, ovulation and fertilization, and sperm navigation from and to the spermatheca (PubMed:32490809). May play a role in regulating cytosolic and endoplasmic reticulum calcium ion release (PubMed:32490809). {ECO:0000250|UniProtKB:Q92508, ECO:0000269|PubMed:32490809}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;Alternative initiation;Alternative splicing;Cell membrane;Glycoprotein;Ion channel;Ion transport;Membrane;Reference proteome;Transmembrane;Transmembrane helix;Transport</td><td style = \"text-align: left;\">KW-0002; KW-0024; KW-0025; KW-1003; KW-0325; KW-0407; KW-0406; KW-0472; KW-1185; KW-0812; KW-1133; KW-0813</td><td style = \"text-align: left;\">MISCELLANEOUS: Piezo comes from the Greek &apos;piesi&apos; meaning pressure. {ECO:0000305}.; MISCELLANEOUS: [Isoform a]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform b]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform c]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform d]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform e]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform f]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform g]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform h]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform i]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform j]: Produced by alternative splicing. {ECO:0000305}.; MISCELLANEOUS: [Isoform k]: Produced by alternative initiation at Met-755 of isoform g. {ECO:0000305}.; MISCELLANEOUS: [Isoform l]: Produced by alternative initiation at Met-1405 of isoform g. {ECO:0000305}.</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000499D6FD</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS (12); DEVELOPMENTAL STAGE (1); DISRUPTION PHENOTYPE (1); FUNCTION (1); MISCELLANEOUS (13); SIMILARITY (1); SUBCELLULAR LOCATION (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Alternative sequence (7); Beta strand (12); Chain (1); Compositional bias (1); Glycosylation (10); Helix (6); Mutagenesis (2); Region (3); Topological domain (37); Transmembrane (36); Turn (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DEVELOPMENTAL STAGE: Expressed from embryogenesis to adulthood (PubMed:32490809). Expressed in one-cell embryos, 4-cell embryos and multi-cell embryos (PubMed:32490809). {ECO:0000269|PubMed:32490809}.</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Expressed in the pharyngeal-intestinal and spermathecal-uterine valves and in multiple reproductive tissues including the germline, somatic oviduct, and spermatheca (PubMed:32490809). During reproduction, it is expressed in sheath cells, sperm, both spermathecal valves and the spermathecal bag cells (PubMed:32490809). {ECO:0000269|PubMed:32490809}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">cation transmembrane transport [GO:0098655]; cellular response to mechanical stimulus [GO:0071260]; detection of mechanical stimulus [GO:0050982]; flagellated sperm motility [GO:0030317]; positive regulation of brood size [GO:0090727]; positive regulation of ovulation [GO:0060279]; regulation of membrane potential [GO:0042391]; response to mechanical stimulus [GO:0009612]</td><td style = \"text-align: left;\">integral component of membrane [GO:0016021]; membrane [GO:0016020]; plasma membrane [GO:0005886]</td><td style = \"text-align: left;\">cation channel activity [GO:0005261]; mechanosensitive ion channel activity [GO:0008381]</td><td style = \"text-align: left;\">GO:0005261; GO:0005886; GO:0008381; GO:0009612; GO:0016020; GO:0016021; GO:0030317; GO:0042391; GO:0050982; GO:0060279; GO:0071260; GO:0090727; GO:0098655</td><td style = \"text-align: left;\">integral component of membrane [GO:0016021]; membrane [GO:0016020]; plasma membrane [GO:0005886]; cation channel activity [GO:0005261]; mechanosensitive ion channel activity [GO:0008381]; cation transmembrane transport [GO:0098655]; cellular response to mechanical stimulus [GO:0071260]; detection of mechanical stimulus [GO:0050982]; flagellated sperm motility [GO:0030317]; positive regulation of brood size [GO:0090727]; positive regulation of ovulation [GO:0060279]; regulation of membrane potential [GO:0042391]; response to mechanical stimulus [GO:0009612]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISRUPTION PHENOTYPE: Reduced brood size due to defects in ovulation with incomplete constriction of the sheath cells impairing oocyte transit, an accumulation of ooplasm in the uterus as a result of &apos;crushed&apos; oocytes, caused by incomplete opening of the spermathecal-uterine valve, which in wild-type allows the fertilized oocyte to be expelled into the uterus, and sperm navigation defects. {ECO:0000269|PubMed:32490809}.</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MUTAGEN 2094; /note=&quot;M-&gt;R: Does not cause significant conformational changes the last cytoplasmic domain.&quot;; /evidence=&quot;ECO:0000269|PubMed:25242456&quot;; MUTAGEN 2405; /note=&quot;R-&gt;P: Reduces brood size and ovulation rates, and there is an accumulation of ooplasmic uterine masses.&quot;; /evidence=&quot;ECO:0000269|PubMed:32490809&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Cell membrane {ECO:0000269|PubMed:32490809}; Multi-pass membrane protein {ECO:0000255}.</td><td style = \"text-align: left;\">TOPO_DOM 1..5; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 27; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 49..56; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 78..122; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 144..173; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 197..198; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 220..239; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 261..303; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 325..454; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 476..478; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 500..506; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 528..552; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 574..633; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 655..656; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 678..683; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 705..739; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 761..832; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 854..874; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 896..931; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 953..990; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1012; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1034..1041; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1063..1096; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1118..1160; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1182..1187; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1211..1231; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1253..1299; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1321..1615; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1637..1654; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1676..1706; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1728..1833; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1855..1866; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1888..1900; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1922..1930; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 1952..2046; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2068..2346; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305&quot;; TOPO_DOM 2368..2442; /note=&quot;Extracellular&quot;; /evidence=&quot;ECO:0000305&quot;</td><td style = \"text-align: left;\">TRANSMEM 6..26; /note=&quot;Helical; Name=1&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 28..48; /note=&quot;Helical; Name=2&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 57..77; /note=&quot;Helical; Name=3&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 123..143; /note=&quot;Helical; Name=4&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 174..196; /note=&quot;Helical; Name=5&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 199..219; /note=&quot;Helical; Name=6&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 240..260; /note=&quot;Helical; Name=7&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 304..324; /note=&quot;Helical; Name=8&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 455..475; /note=&quot;Helical; Name=9&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 479..499; /note=&quot;Helical; Name=10&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 507..527; /note=&quot;Helical; Name=11&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 553..573; /note=&quot;Helical; Name=12&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 634..654; /note=&quot;Helical; Name=13&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 657..677; /note=&quot;Helical; Name=14&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 684..704; /note=&quot;Helical; Name=15&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 740..760; /note=&quot;Helical; Name=16&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 833..853; /note=&quot;Helical; Name=17&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 875..895; /note=&quot;Helical; Name=18&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 932..952; /note=&quot;Helical; Name=19&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 991..1011; /note=&quot;Helical; Name=20&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1013..1033; /note=&quot;Helical; Name=21&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1042..1062; /note=&quot;Helical; Name=22&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1097..1117; /note=&quot;Helical; Name=23&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1161..1181; /note=&quot;Helical; Name=24&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1188..1210; /note=&quot;Helical; Name=25&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1232..1252; /note=&quot;Helical; Name=26&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1300..1320; /note=&quot;Helical; Name=27&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1616..1636; /note=&quot;Helical; Name=28&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1655..1675; /note=&quot;Helical; Name=29&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1707..1727; /note=&quot;Helical; Name=30&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1834..1854; /note=&quot;Helical; Name=31&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1867..1887; /note=&quot;Helical; Name=32&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1901..1921; /note=&quot;Helical; Name=33&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 1931..1951; /note=&quot;Helical; Name=34&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 2047..2067; /note=&quot;Helical; Name=35&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 2347..2367; /note=&quot;Helical; Name=36&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"text-align: left;\">CHAIN 1..2442; /note=&quot;Piezo-type mechanosensitive ion channel component 1&quot;; /id=&quot;PRO_0000451574&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CARBOHYD 332; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 392; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 440; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 816; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 908; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 913; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 1088; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 1094; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 1646; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 1737; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">X-ray crystallography (2)</td><td style = \"text-align: left;\">STRAND 2079..2085; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2091..2096; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2106..2109; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2151..2156; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2184..2191; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2206..2213; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2236..2242; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2244..2248; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2250..2253; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2288..2294; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2305..2310; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; STRAND 2334..2340; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;</td><td style = \"text-align: left;\">HELIX 2113..2121; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; HELIX 2130..2143; /evidence=&quot;ECO:0007829|PDB:4PKX&quot;; HELIX 2148..2150; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; HELIX 2168..2179; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; HELIX 2218..2229; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;; HELIX 2258..2264; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;</td><td style = \"text-align: left;\">TURN 2327..2329; /evidence=&quot;ECO:0007829|PDB:4PKE&quot;</td><td style = \"text-align: left;\">9851916; 32490809; 25242456</td><td style = \"text-align: left;\">2020-12-02</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">9</td><td style = \"text-align: right;\">8</td><td style = \"text-align: left;\">A0A061I403</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">FICD_CRIGR</td><td style = \"text-align: left;\">Protein adenylyltransferase FICD, EC 2.7.7.n1 (AMPylator FICD) (De-AMPylase FICD, EC 3.1.4.-) (FIC domain-containing protein) (Huntingtin-interacting protein E)</td><td style = \"text-align: left;\">FICD HYPE H671_4g11989 I79_014982</td><td style = \"text-align: left;\">Cricetulus griseus (Chinese hamster) (Cricetulus barabensis griseus)</td><td style = \"text-align: right;\">455</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">H671_4g11989 I79_014982</td><td style = \"text-align: left;\">FICD</td><td style = \"text-align: left;\">HYPE</td><td style = \"text-align: right;\">10029</td><td style = \"text-align: left;\">UP000001075: Unassembled WGS sequence; UP000030759: Unassembled WGS sequence</td><td style = \"text-align: left;\">Cricetulus (genus), Cricetinae (subfamily), Cricetidae (family), Muroidea (no rank), Myomorpha (suborder), Rodentia (order), Glires (no rank), Euarchontoglires (superorder), Boreoeutheria (no rank), Eutheria (no rank), Theria (no rank), Mammalia (class), Amniota (no rank), Tetrapoda (no rank), Dipnotetrapodomorpha (no rank), Sarcopterygii (superclass), Euteleostomi (no rank), Teleostomi (no rank), Gnathostomata (no rank), Vertebrata (no rank), Craniata (subphylum), Chordata (phylum), Deuterostomia (no rank), Bilateria (no rank), Eumetazoa (no rank), Metazoa (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">10028 (genus), 10026 (subfamily), 337677 (family), 337687 (no rank), 1963758 (suborder), 9989 (order), 314147 (no rank), 314146 (superorder), 1437010 (no rank), 9347 (no rank), 32525 (no rank), 40674 (class), 32524 (no rank), 32523 (no rank), 1338369 (no rank), 8287 (superclass), 117571 (no rank), 117570 (no rank), 7776 (no rank), 7742 (no rank), 89593 (subphylum), 7711 (phylum), 33511 (no rank), 33213 (no rank), 6072 (no rank), 33208 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Sequence=EGW11974.1; Type=Erroneous gene model prediction; Evidence={ECO:0000305};</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">51477</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MPMASVIAVAEPKWISVWGRFLWLTLLSMALGSLLALLLPLGAVEEQCLAVLRSFHLLRSKLDRTQHVVTKCTSPSTELSVTSGDVGLLTVKTKTSPAGKLEAKAALNQALEMKRQGKREKAHKLFLHALKMDPGFVDALNEFGIFSEEEKDIIQADYLYTRALTISPFHEKALVNRDRTLPLVEEIDQRYFSIIDSKVKKVMSIPKGSSALRRVMEETYYHHIYHTVAIEGNTLTLSEIRHILETRYAVPGKSLEEQNEVIGMHAAMKYINTTLVSRIGSVTIDDMLEIHRRVLGYVDPVEAGRFRRTQVLVGHHIPPHPRDVEKQMQEFTQWLNSEDAMNLHPVEFAALAHYKLVYIHPFIDGNGRTSRLLMNLILMQAGYPPITILKEQRSEYYHVLEVANEGDVRPFIRFIAKCTEVTLDTLLLATTEYSVALPEAQPNHSGLKETLPVRP</td><td style = \"text-align: left;\">SEQUENCE CAUTION:  Sequence=EGW11974.1; Type=Erroneous gene model prediction; Evidence={ECO:0000305};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 360; /evidence=&quot;ECO:0000269|PubMed:27918543&quot;</td><td style = \"text-align: left;\">BINDING 231; /ligand=&quot;ATP&quot;; /ligand_id=&quot;ChEBI:CHEBI:30616&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;; BINDING 313..316; /ligand=&quot;ATP&quot;; /ligand_id=&quot;ChEBI:CHEBI:30616&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;; BINDING 364..371; /ligand=&quot;ATP&quot;; /ligand_id=&quot;ChEBI:CHEBI:30616&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;; BINDING 396..397; /ligand=&quot;ATP&quot;; /ligand_id=&quot;ChEBI:CHEBI:30616&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;; BINDING 404; /ligand=&quot;ATP&quot;; /ligand_id=&quot;ChEBI:CHEBI:30616&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=ATP + L-tyrosyl-[protein] = diphosphate + O-(5&apos;-adenylyl)-L-tyrosyl-[protein]; Xref=Rhea:RHEA:54288, Rhea:RHEA-COMP:10136, Rhea:RHEA-COMP:13846, ChEBI:CHEBI:30616, ChEBI:CHEBI:33019, ChEBI:CHEBI:46858, ChEBI:CHEBI:83624; EC=2.7.7.n1; Evidence={ECO:0000250|UniProtKB:Q9BVA6}; CATALYTIC ACTIVITY: Reaction=3-O-(5&apos;-adenylyl)-L-threonyl-[protein] + H2O = AMP + H(+) + L-threonyl-[protein]; Xref=Rhea:RHEA:55932, Rhea:RHEA-COMP:11060, Rhea:RHEA-COMP:13847, ChEBI:CHEBI:15377, ChEBI:CHEBI:15378, ChEBI:CHEBI:30013, ChEBI:CHEBI:138113, ChEBI:CHEBI:456215; Evidence={ECO:0000269|PubMed:27918543}; CATALYTIC ACTIVITY: Reaction=ATP + L-threonyl-[protein] = 3-O-(5&apos;-adenylyl)-L-threonyl-[protein] + diphosphate; Xref=Rhea:RHEA:54292, Rhea:RHEA-COMP:11060, Rhea:RHEA-COMP:13847, ChEBI:CHEBI:30013, ChEBI:CHEBI:30616, ChEBI:CHEBI:33019, ChEBI:CHEBI:138113; EC=2.7.7.n1; Evidence={ECO:0000269|PubMed:26673894, ECO:0000269|PubMed:27918543, ECO:0000269|PubMed:29064368};</td><td style = \"text-align: left;\">COFACTOR: Name=Mg(2+); Xref=ChEBI:CHEBI:18420; Evidence={ECO:0000250|UniProtKB:Q9BVA6}; Name=Mn(2+); Xref=ChEBI:CHEBI:29035; Evidence={ECO:0000250|UniProtKB:Q9BVA6}; Note=Divalent metal cation. Prefers Mn(2+) over Mg(2+). {ECO:0000250|UniProtKB:Q9BVA6};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.7.7.n1; 3.1.4.-</td><td style = \"text-align: left;\">FUNCTION: Protein that can both mediate the addition of adenosine 5&apos;-monophosphate (AMP) to specific residues of target proteins (AMPylation), and the removal of the same modification from target proteins (de-AMPylation), depending on the context (PubMed:27918543). The side chain of Glu-231 determines which of the two opposing activities (AMPylase or de-AMPylase) will take place (PubMed:27918543). Acts as a key regulator of the ERN1/IRE1-mediated unfolded protein response (UPR) by mediating AMPylation or de-AMPylation of HSPA5/BiP (PubMed:27918543). In unstressed cells, acts as an adenylyltransferase by mediating AMPylation of HSPA5/BiP at &apos;Thr-518&apos;, thereby inactivating it (PubMed:26673894, PubMed:29064368, PubMed:27918543). In response to endoplasmic reticulum stress, acts as a phosphodiesterase by mediating removal of ATP (de-AMPylation) from HSPA5/BiP at &apos;Thr-518&apos;, leading to restore HSPA5/BiP activity (PubMed:27918543). Although it is able to AMPylate RhoA, Rac and Cdc42 Rho GTPases in vitro, Rho GTPases do not constitute physiological substrates (By similarity). {ECO:0000250|UniProtKB:Q9BVA6, ECO:0000269|PubMed:26673894, ECO:0000269|PubMed:27918543, ECO:0000269|PubMed:29064368}.</td><td style = \"text-align: left;\">ACTIVITY REGULATION: The side chain of Glu-231 determines which of the two opposing activities (AMPylase or de-AMPylase) will take place (PubMed:27918543). In response to endoplasmic reticulum stress, mediates de-AMPylase activity (PubMed:27918543). Adenylyltransferase activity is inhibited by the inhibitory helix present at the N-terminus: Glu-231 binds ATP and competes with ATP-binding at Arg-371, thereby preventing adenylyltransferase activity (By similarity). In unstressed cells, disengagement of Glu-231 promotes adenylyltransferase activity (PubMed:27918543). Activation dissociates ATP-binding from Glu-231, allowing ordered binding of the entire ATP moiety with the alpha-phosphate in an orientation that is productive for accepting an incoming target hydroxyl side chain (By similarity). {ECO:0000250|UniProtKB:Q9BVA6, ECO:0000269|PubMed:27918543}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:54288 RHEA-COMP:10136 RHEA-COMP:13846 RHEA:55932 RHEA-COMP:11060 RHEA-COMP:13847 RHEA:54292 RHEA-COMP:11060 RHEA-COMP:13847</td><td style = \"text-align: left;\">SITE 231; /note=&quot;Important for autoinhibition of adenylyltransferase activity&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ATP-binding;Endoplasmic reticulum;Glycoprotein;Hydrolase;Magnesium;Manganese;Membrane;Nucleotide-binding;Nucleotidyltransferase;Phosphoprotein;Reference proteome;Repeat;Signal-anchor;TPR repeat;Transferase;Transmembrane;Transmembrane helix;Unfolded protein response</td><td style = \"text-align: left;\">KW-0067; KW-0256; KW-0325; KW-0378; KW-0460; KW-0464; KW-0472; KW-0547; KW-0548; KW-0597; KW-1185; KW-0677; KW-0735; KW-0802; KW-0808; KW-0812; KW-1133; KW-0834</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00049C7B5B</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); CATALYTIC ACTIVITY (3); COFACTOR (1); DOMAIN (1); FUNCTION (1); PTM (1); SEQUENCE CAUTION (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); SUBUNIT (1)</td><td style = \"text-align: left;\">Active site (1); Binding site (5); Chain (1); Domain (1); Glycosylation (1); Modified residue (3); Motif (1); Mutagenesis (2); Repeat (2); Site (1); Topological domain (2); Transmembrane (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBUNIT: Homodimer. Interacts with HD. {ECO:0000250|UniProtKB:Q9BVA6}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">protein adenylylation [GO:0018117]; protein deadenylylation [GO:0044602]; regulation of IRE1-mediated unfolded protein response [GO:1903894]; response to endoplasmic reticulum stress [GO:0034976]; response to unfolded protein [GO:0006986]</td><td style = \"text-align: left;\">integral component of endoplasmic reticulum membrane [GO:0030176]</td><td style = \"text-align: left;\">ATP binding [GO:0005524]; chaperone binding [GO:0051087]; Hsp70 protein binding [GO:0030544]; protein adenylylhydrolase activity [GO:0044603]; protein adenylyltransferase activity [GO:0070733]; protein homodimerization activity [GO:0042803]</td><td style = \"text-align: left;\">GO:0005524; GO:0006986; GO:0018117; GO:0030176; GO:0030544; GO:0034976; GO:0042803; GO:0044602; GO:0044603; GO:0051087; GO:0070733; GO:1903894</td><td style = \"text-align: left;\">integral component of endoplasmic reticulum membrane [GO:0030176]; ATP binding [GO:0005524]; chaperone binding [GO:0051087]; Hsp70 protein binding [GO:0030544]; protein adenylylhydrolase activity [GO:0044603]; protein adenylyltransferase activity [GO:0070733]; protein homodimerization activity [GO:0042803]; protein adenylylation [GO:0018117]; protein deadenylylation [GO:0044602]; regulation of IRE1-mediated unfolded protein response [GO:1903894]; response to endoplasmic reticulum stress [GO:0034976]; response to unfolded protein [GO:0006986]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MUTAGEN 231; /note=&quot;E-&gt;G: Impaired phosphodiesterase activity. Promotes adenylyltransferase activity.&quot;; /evidence=&quot;ECO:0000269|PubMed:26673894, ECO:0000269|PubMed:27918543&quot;; MUTAGEN 360; /note=&quot;H-&gt;A: Abolishes adenylyltransferase and phosphodiesterase activities.&quot;; /evidence=&quot;ECO:0000269|PubMed:27918543&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Endoplasmic reticulum membrane {ECO:0000250|UniProtKB:Q9BVA6}; Single-pass type II membrane protein {ECO:0000250|UniProtKB:Q9BVA6}.</td><td style = \"text-align: left;\">TOPO_DOM 1..20; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;; TOPO_DOM 42..455; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;</td><td style = \"text-align: left;\">TRANSMEM 21..41; /note=&quot;Helical; Signal-anchor for type II membrane protein&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"text-align: left;\">CHAIN 1..455; /note=&quot;Protein adenylyltransferase FICD&quot;; /id=&quot;PRO_0000443449&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CARBOHYD 272; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">MOD_RES 76; /note=&quot;O-AMP-serine; by autocatalysis&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;; MOD_RES 77; /note=&quot;O-AMP-threonine; by autocatalysis&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;; MOD_RES 180; /note=&quot;O-AMP-threonine; by autocatalysis&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q9BVA6&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PTM: Auto-AMPylated in vitro. {ECO:0000250|UniProtKB:Q9BVA6}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">21804562; 23929341; 26673894; 29064368; 27918543</td><td style = \"text-align: left;\">2018-02-28</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">10</td><td style = \"text-align: right;\">9</td><td style = \"text-align: left;\">A0A067CMC7</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">HTP3_SAPPC</td><td style = \"text-align: left;\">Endonuclease Htp3, EC 3.1.31.- (Host targeting protein 3) (RxLR effector protein Htp3)</td><td style = \"text-align: left;\">HTP3 SPRG_03573</td><td style = \"text-align: left;\">Saprolegnia parasitica (strain CBS 223.65)</td><td style = \"text-align: right;\">211</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SPRG_03573</td><td style = \"text-align: left;\">HTP3</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">695850</td><td style = \"text-align: left;\">UP000030745: Unassembled WGS sequence</td><td style = \"text-align: left;\">Saprolegnia parasitica (species), Saprolegnia (genus), Saprolegniaceae (family), Saprolegniales (order), Oomycota (phylum), Stramenopiles (no rank), Sar (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">101203 (species), 4769 (genus), 4764 (family), 4763 (order), 4762 (phylum), 33634 (no rank), 2698737 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">23841</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MLEVPVWIPILAFAVGLGLGLLIPHLQKPFQRFSTVNDIPKEFFEHERTLRGKVVSVTDGDTIRVRHVPWLANGDGDFKGKLTETTLQLRVAGVDCPETAKFGRTGQPFGEEAKAWLKGELQDQVVSFKLLMKDQYSRAVCLVYYGSWAAPMNVSEELLRHGYANIYRQSGAVYGGLLETFEALEAEAREKRVNIWSLDKRETPAQYKARK</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 90; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00272&quot;; ACT_SITE 98; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00272&quot;; ACT_SITE 138; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00272&quot;</td><td style = \"text-align: left;\">BINDING 77; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P00644&quot;; BINDING 95; /ligand=&quot;Ca(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29108&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P00644&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3.1.31.-</td><td style = \"text-align: left;\">FUNCTION: Effector involved in the disease saprolegniosis in salmonids and other freshwater fish, resulting in considerable economic losses in aquaculture (PubMed:29904064). Within the host fish cells, Htp3 is released from vesicles into host cytosol where it degrades nucleic acids (PubMed:29904064). {ECO:0000269|PubMed:29904064}.</td><td style = \"text-align: left;\">ACTIVITY REGULATION: The nuclease activity shows a general salt dependency with a clear reduction by magnesium and sulfate ions. {ECO:0000269|PubMed:29904064}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Calcium;Endonuclease;Glycoprotein;Host cytoplasm;Hydrolase;Metal-binding;Nuclease;Reference proteome;Secreted;Signal;Virulence</td><td style = \"text-align: left;\">KW-0106; KW-0255; KW-0325; KW-1035; KW-0378; KW-0479; KW-0540; KW-1185; KW-0964; KW-0732; KW-0843</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00049C82A7</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); DOMAIN (2); FUNCTION (1); SIMILARITY (2); SUBCELLULAR LOCATION (1); SUBUNIT (1)</td><td style = \"text-align: left;\">Active site (3); Binding site (2); Chain (1); Domain (1); Glycosylation (1); Motif (1); Mutagenesis (2); Region (1); Signal (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBUNIT: Interacts with the host cell surface endoplasmin gp96, in order to get translocated into to host cell (PubMed:29904064). Interacts with the effector Htp1, in order to get released from vesicles into the host cytosol (PubMed:29904064). {ECO:0000269|PubMed:29904064}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; host cell cytosol [GO:0044164]</td><td style = \"text-align: left;\">endonuclease activity [GO:0004519]; metal ion binding [GO:0046872]</td><td style = \"text-align: left;\">GO:0004519; GO:0005576; GO:0044164; GO:0046872</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; host cell cytosol [GO:0044164]; endonuclease activity [GO:0004519]; metal ion binding [GO:0046872]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MUTAGEN 208; /note=&quot;K-&gt;A: Impairs host cell surface association and translocation into the host cell; when associated with A-210.&quot;; /evidence=&quot;ECO:0000269|PubMed:29904064&quot;; MUTAGEN 210; /note=&quot;R-&gt;A: Impairs host cell surface association and translocation into the host cell; when associated with A-208.&quot;; /evidence=&quot;ECO:0000269|PubMed:29904064&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Secreted {ECO:0000269|PubMed:29904064}. Host cytoplasm, host cytosol {ECO:0000269|PubMed:29904064}. Note=Uptake into host cells is more efficient at a lower pH of 5.5. S.parasitica acidifies the pH of its environment, which likely leads to the exposure of a gp96 protein to the host cell surface. The gp96 protein is working as a receptor and mediates the translocation of Htp3 via lipid rafts into the cell. Finally, Htp3 is released from vesicles with the help of other effector proteins, such as Htp1, into the cytosol where it is functionally active as a nuclease. {ECO:0000269|PubMed:29904064}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 21..211; /note=&quot;Endonuclease Htp3&quot;; /id=&quot;PRO_0000446903&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CARBOHYD 153; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SIGNAL 1..20; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">23785293; 29904064</td><td style = \"text-align: left;\">2019-05-08</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11</td><td style = \"text-align: right;\">10</td><td style = \"text-align: left;\">A0A067XGX8</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">AROG2_PETHY</td><td style = \"text-align: left;\">Phospho-2-dehydro-3-deoxyheptonate aldolase 2, chloroplastic, EC 2.5.1.54 (3-deoxy-D-arabino-heptulosonate 7-phosphate synthase 2, DAHP synthase 2, PhDAHP2) (Phospho-2-keto-3-deoxyheptonate aldolase 2)</td><td style = \"text-align: left;\">DAHP2 DHS2</td><td style = \"text-align: left;\">Petunia hybrida (Petunia)</td><td style = \"text-align: right;\">512</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DAHP2</td><td style = \"text-align: left;\">DHS2</td><td style = \"text-align: right;\">4102</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Petunia (genus), Petunioideae (subfamily), Solanaceae (family), Solanales (order), lamiids (no rank), asterids (no rank), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">4101 (genus), 424555 (subfamily), 4070 (family), 4069 (order), 91888 (no rank), 71274 (no rank), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">57062</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MALTATATTRGGSALPNSCLQTPKFQSLQKPTFISSFPTNKKTKPRTKHISAVQSPPSTTKWNLESWKTKPAFQLPDYPDKVELESVLKTLSTYPPIVFAGEARNLEEKLGEAALGNAFLLQGGDCAESFKEFSANNIRDTFRVMLQMGVVLMFGGQMPVIKVGRMAGQFAKPRSDPFEEKDGVKLPSYRGDNVNGDAFDEKSRIPDPHRMVRAYTQSVATLNLLRAFASGGYAAMQRVNQWNLDFTDQSEQGDRYRELAHRVDEAMGFMTAAGLTVDHTIMTTTDFWTSHECLLLPYEQALTREDSTSGLYYDCSAHMIWVGERTRQLDGAHVEFLRGIANPLGIKVSHKMDPDELVKLIDILNPQNKPGRITVITRMGADNMRVKLPHLIRAVRGAGQIVTWVSDPMHGNTTKAPCGLKTRSFDSIRAELRAFFDVHEQEGSYPGGVHLEMTGQNVTECVGGSRTITYNDLSSRYHTHCDPRLNASQALELAFAIAERLRRRRLGPKFSL</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">BINDING 126; /ligand=&quot;Mn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29035&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 165; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 324..325; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 347; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 378; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 410; /ligand=&quot;Mn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29035&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 452; /ligand=&quot;Mn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29035&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 482; /ligand=&quot;Mn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29035&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=D-erythrose 4-phosphate + H2O + phosphoenolpyruvate = 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate + phosphate; Xref=Rhea:RHEA:14717, ChEBI:CHEBI:15377, ChEBI:CHEBI:16897, ChEBI:CHEBI:43474, ChEBI:CHEBI:58394, ChEBI:CHEBI:58702; EC=2.5.1.54; Evidence={ECO:0000250|UniProtKB:O53512};</td><td style = \"text-align: left;\">COFACTOR: Name=Mn(2+); Xref=ChEBI:CHEBI:29035; Evidence={ECO:0000250|UniProtKB:O53512}; Note=Binds 1 divalent metal cation per subunit that could be manganese. {ECO:0000250|UniProtKB:O53512};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.5.1.54</td><td style = \"text-align: left;\">FUNCTION: Involved in the production of volatile organic compounds (VOCs) (PubMed:24815009). Catalyzes an aldol-like condensation reaction between phosphoenolpyruvate (PEP) and D-erythrose 4-phosphate (E4P) to generate 3-deoxy-D-arabino-heptulosonate 7-phosphate (DAH7P) and inorganic phosphate (By similarity). {ECO:0000250|UniProtKB:O53512, ECO:0000269|PubMed:24815009}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PATHWAY: Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 1/7. {ECO:0000250|UniProtKB:O53512}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:14717</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Amino-acid biosynthesis;Aromatic amino acid biosynthesis;Chloroplast;Manganese;Metal-binding;Plastid;Transferase;Transit peptide</td><td style = \"text-align: left;\">KW-0028; KW-0057; KW-0150; KW-0464; KW-0479; KW-0934; KW-0808; KW-0809</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at transcript level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00049B270B</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY (1); COFACTOR (1); DEVELOPMENTAL STAGE (1); DISRUPTION PHENOTYPE (1); FUNCTION (1); PATHWAY (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); SUBUNIT (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Binding site (8); Chain (1); Region (1); Transit peptide (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBUNIT: Homodimer. {ECO:0000250|UniProtKB:O53512}.</td><td style = \"text-align: left;\">DEVELOPMENTAL STAGE: During floral development, expressed constitutively. {ECO:0000269|PubMed:24815009}.</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Mostly expressed in leaves and stems, and, to a lower extent, in roots, stigmas, anthers, petal tubes, petal limbs and sepals. {ECO:0000269|PubMed:24815009}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">aromatic amino acid family biosynthetic process [GO:0009073]; cellular amino acid biosynthetic process [GO:0008652]; chorismate biosynthetic process [GO:0009423]</td><td style = \"text-align: left;\">chloroplast [GO:0009507]</td><td style = \"text-align: left;\">3-deoxy-7-phosphoheptulonate synthase activity [GO:0003849]; metal ion binding [GO:0046872]</td><td style = \"text-align: left;\">GO:0003849; GO:0008652; GO:0009073; GO:0009423; GO:0009507; GO:0046872</td><td style = \"text-align: left;\">chloroplast [GO:0009507]; 3-deoxy-7-phosphoheptulonate synthase activity [GO:0003849]; metal ion binding [GO:0046872]; aromatic amino acid family biosynthetic process [GO:0009073]; cellular amino acid biosynthetic process [GO:0008652]; chorismate biosynthetic process [GO:0009423]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISRUPTION PHENOTYPE: Normal floral volatile benzenoids and phenylpropanoids (FVBP) emission. {ECO:0000269|PubMed:24815009}.</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Plastid, chloroplast {ECO:0000269|PubMed:24815009}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 58..512; /note=&quot;Phospho-2-dehydro-3-deoxyheptonate aldolase 2, chloroplastic&quot;; /id=&quot;PRO_0000451510&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">TRANSIT 1..57; /note=&quot;Chloroplast&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">24815009</td><td style = \"text-align: left;\">2020-12-02</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">12</td><td style = \"text-align: right;\">11</td><td style = \"text-align: left;\">A0A067XH53</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">AROG1_PETHY</td><td style = \"text-align: left;\">Phospho-2-dehydro-3-deoxyheptonate aldolase 1, chloroplastic, EC 2.5.1.54 (3-deoxy-D-arabino-heptulosonate 7-phosphate synthase 1, DAHP synthase 1, PhDAHP1) (Phospho-2-keto-3-deoxyheptonate aldolase 1)</td><td style = \"text-align: left;\">DAHP1 DAHPS DHS1</td><td style = \"text-align: left;\">Petunia hybrida (Petunia)</td><td style = \"text-align: right;\">533</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DAHP1</td><td style = \"text-align: left;\">DAHPS DHS1</td><td style = \"text-align: right;\">4102</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Petunia (genus), Petunioideae (subfamily), Solanaceae (family), Solanales (order), lamiids (no rank), asterids (no rank), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">4101 (genus), 424555 (subfamily), 4070 (family), 4069 (order), 91888 (no rank), 71274 (no rank), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">58894</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MALSTNSTTSSLLPKTPLVQQPLLKNASLPTTTKAIRFIQPISAIHSSDSSKNTPIVSAKPSSPPAATSTAAATAVTKQEWSIDSWKTKKALQLPEYPNQEELKNVLKTIEDFPPIVFAGEARHLEEKLGEAAMGRAFLLQGGDCAESFKEFNANNIRDTFRILLQMGAVLMFGGQMPVIKVGRMAGQFAKPRSDNFEEKNGVKLPSYRGDNVNGDAFDLKSRTPDPQRLIRAYCQSAATLNLLRAFATGGYAAMQRVTQWNLDFTEHSEQGDRYRELANRVDEALGFMNAAGLTTDHPIMTTTEFWTSHECLLLPYEQSLTRLDSTSGLYYDCSAHFLWVGERTRQLDGAHVEFLRGIANPLGIKVSDKMDPSALVKLIEILNPQNKAGRITIITRMGAENMRVKLPHLIRAVRGAGQIVTWVSDPMHGNTIKAPCGLKTRPFDSIRAEVRAFFDVHEQEGSHPGGVHLEMTGQNVTECIGGSRTVTFDDLSSRYHTHCDPRLNASQSLELAFIIAERLRKRRLGSQSVLGQ</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">BINDING 145; /ligand=&quot;Mn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29035&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 184; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 343..344; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 366; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 397; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 429; /ligand=&quot;Mn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29035&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 471; /ligand=&quot;Mn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29035&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;; BINDING 501; /ligand=&quot;Mn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29035&quot;; /evidence=&quot;ECO:0000250|UniProtKB:O53512&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=D-erythrose 4-phosphate + H2O + phosphoenolpyruvate = 7-phospho-2-dehydro-3-deoxy-D-arabino-heptonate + phosphate; Xref=Rhea:RHEA:14717, ChEBI:CHEBI:15377, ChEBI:CHEBI:16897, ChEBI:CHEBI:43474, ChEBI:CHEBI:58394, ChEBI:CHEBI:58702; EC=2.5.1.54; Evidence={ECO:0000250|UniProtKB:O53512};</td><td style = \"text-align: left;\">COFACTOR: Name=Mn(2+); Xref=ChEBI:CHEBI:29035; Evidence={ECO:0000250|UniProtKB:O53512}; Note=Binds 1 divalent metal cation per subunit that could be manganese. {ECO:0000250|UniProtKB:O53512};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.5.1.54</td><td style = \"text-align: left;\">FUNCTION: Involved in the production of volatile organic compounds (VOCs), including floral volatile benzenoids and phenylpropanoids (FVBP), in flowers of fragrant cultivars (e.g. cv. Mitchell and cv. V26), scent attracting pollinators (e.g. the night-active hawkmoth pollinator Manduca sexta) (PubMed:24815009). Catalyzes an aldol-like condensation reaction between phosphoenolpyruvate (PEP) and D-erythrose 4-phosphate (E4P) to generate 3-deoxy-D-arabino-heptulosonate 7-phosphate (DAH7P) and inorganic phosphate (By similarity). {ECO:0000250|UniProtKB:O53512, ECO:0000269|PubMed:24815009}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PATHWAY: Metabolic intermediate biosynthesis; chorismate biosynthesis; chorismate from D-erythrose 4-phosphate and phosphoenolpyruvate: step 1/7. {ECO:0000250|UniProtKB:O53512}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:14717</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Amino-acid biosynthesis;Aromatic amino acid biosynthesis;Chloroplast;Manganese;Metal-binding;Plastid;Transferase;Transit peptide</td><td style = \"text-align: left;\">KW-0028; KW-0057; KW-0150; KW-0464; KW-0479; KW-0934; KW-0808; KW-0809</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at transcript level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000499BE38</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY (1); COFACTOR (1); DEVELOPMENTAL STAGE (1); DISRUPTION PHENOTYPE (1); FUNCTION (1); INDUCTION (1); PATHWAY (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); SUBUNIT (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Binding site (8); Chain (1); Region (1); Transit peptide (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBUNIT: Homodimer. {ECO:0000250|UniProtKB:O53512}.</td><td style = \"text-align: left;\">DEVELOPMENTAL STAGE: During floral development, accumulates to highest levels during open flower stages. {ECO:0000269|PubMed:24815009}.</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Mostly expressed in flowers, especially in petal limbs and tubes, and, to a lower extent, in roots, stems, stigmas, anthers, leaves and sepals. {ECO:0000269|PubMed:24815009}.</td><td style = \"text-align: left;\">INDUCTION: Triggered by EOBI in flowers. {ECO:0000269|PubMed:23275577}.</td><td style = \"text-align: left;\">aromatic amino acid family biosynthetic process [GO:0009073]; cellular amino acid biosynthetic process [GO:0008652]; chorismate biosynthetic process [GO:0009423]; green leaf volatile biosynthetic process [GO:0010597]</td><td style = \"text-align: left;\">chloroplast [GO:0009507]</td><td style = \"text-align: left;\">3-deoxy-7-phosphoheptulonate synthase activity [GO:0003849]; metal ion binding [GO:0046872]</td><td style = \"text-align: left;\">GO:0003849; GO:0008652; GO:0009073; GO:0009423; GO:0009507; GO:0010597; GO:0046872</td><td style = \"text-align: left;\">chloroplast [GO:0009507]; 3-deoxy-7-phosphoheptulonate synthase activity [GO:0003849]; metal ion binding [GO:0046872]; aromatic amino acid family biosynthetic process [GO:0009073]; cellular amino acid biosynthetic process [GO:0008652]; chorismate biosynthetic process [GO:0009423]; green leaf volatile biosynthetic process [GO:0010597]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISRUPTION PHENOTYPE: Reduced floral volatile benzenoids and phenylpropanoids (FVBP) emission. {ECO:0000269|PubMed:24815009}.</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Plastid, chloroplast {ECO:0000269|PubMed:24815009}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 58..533; /note=&quot;Phospho-2-dehydro-3-deoxyheptonate aldolase 1, chloroplastic&quot;; /id=&quot;PRO_0000451509&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">TRANSIT 1..57; /note=&quot;Chloroplast&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">24815009; 23275577</td><td style = \"text-align: left;\">2020-12-02</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">13</td><td style = \"text-align: right;\">12</td><td style = \"text-align: left;\">A0A067XR63</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">XTH7_DIOKA</td><td style = \"text-align: left;\">Xyloglucan endotransglucosylase protein 7, XET protein 7, EC 2.4.1.207 (DkXTH7) (Xyloglucan endotransglucosylase/hydrolase protein 7, XTH protein 7)</td><td style = \"text-align: left;\">XTH7</td><td style = \"text-align: left;\">Diospyros kaki (Kaki persimmon) (Diospyros chinensis)</td><td style = \"text-align: right;\">268</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">XTH7</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">35925</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Diospyros (genus), Ebenaceae (family), Ericales (order), asterids (no rank), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">13492 (genus), 19955 (family), 41945 (order), 71274 (no rank), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">30834</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MNAEGGNLHREFEITWGDGRARIHNNGGLLTLSLDRASGSGFRSKNEYLFGRIEIQIKLVAGNSAGTVATYYLSSEGPTHDEIDFEFLGNSSGEPYTLHTNVFSQGKGNREQQFFLWFDPTMDFHTYTILWNPQRIIFYVDETPIREFKNLERHGIPFPRSQAMRVYSSMWNADDWATRGGLVKTDWTKAPFTASYRSYKADACVWSGEASSCGSQDSNPSDKWWMTEELNATRMKRLRWVQKKYMVYNYCVDKMRFPEGLAPECNIS</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 82; /note=&quot;Nucleophile&quot;; /evidence=&quot;ECO:0000255|PIRSR:PIRSR005604-1, ECO:0000255|PROSITE-ProRule:PRU10064&quot;; ACT_SITE 86; /note=&quot;Proton donor&quot;; /evidence=&quot;ECO:0000255|PIRSR:PIRSR005604-1, ECO:0000255|PROSITE-ProRule:PRU10064&quot;</td><td style = \"text-align: left;\">BINDING 86; /ligand=&quot;xyloglucan&quot;; /ligand_id=&quot;ChEBI:CHEBI:18233&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;; BINDING 99..101; /ligand=&quot;xyloglucan&quot;; /ligand_id=&quot;ChEBI:CHEBI:18233&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;; BINDING 109..111; /ligand=&quot;xyloglucan&quot;; /ligand_id=&quot;ChEBI:CHEBI:18233&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;; BINDING 175..176; /ligand=&quot;xyloglucan&quot;; /ligand_id=&quot;ChEBI:CHEBI:18233&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;; BINDING 180; /ligand=&quot;xyloglucan&quot;; /ligand_id=&quot;ChEBI:CHEBI:18233&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;; BINDING 256; /ligand=&quot;xyloglucan&quot;; /ligand_id=&quot;ChEBI:CHEBI:18233&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=breaks a beta-(1-&gt;4) bond in the backbone of a xyloglucan and transfers the xyloglucanyl segment on to O-4 of the non-reducing terminal glucose residue of an acceptor, which can be a xyloglucan or an oligosaccharide of xyloglucan.; EC=2.4.1.207; Evidence={ECO:0000255|RuleBase:RU361120, ECO:0000269|PubMed:27242828};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.4.1.207</td><td style = \"text-align: left;\">FUNCTION: Catalyzes xyloglucan endotransglycosylation (XET). Cleaves and religates xyloglucan polymers. Does not catalyze xyloglucan endohydrolysis (XEH). Probably involved in cell wall assembly and synthesis in fast growing tissues and in the maintenance of firmness in mature fruits. {ECO:0000269|PubMed:27242828}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  pH dependence: Optimum pH is between 5-6. Activity decreases sharply when the pH is lowered from 5 to 4. {ECO:0000269|PubMed:27242828};</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SITE 84; /note=&quot;Important for catalytic activity&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Cell wall biogenesis/degradation;Cytoplasm;Disulfide bond;Fruit ripening;Glycoprotein;Glycosidase;Glycosyltransferase;Hydrolase;Transferase</td><td style = \"text-align: left;\">KW-0961; KW-0963; KW-1015; KW-0292; KW-0325; KW-0326; KW-0328; KW-0378; KW-0808</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI0004A9B658</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES (1); CATALYTIC ACTIVITY (1); DEVELOPMENTAL STAGE (1); FUNCTION (1); INDUCTION (1); PTM (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Active site (2); Binding site (6); Chain (1); Disulfide bond (2); Domain (1); Glycosylation (1); Site (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DEVELOPMENTAL STAGE: Expressed during fruit ripening. Expressed at an extremely high level in fast growing tissues, such as young leaves and fruits. {ECO:0000269|PubMed:27242828}.</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Expressed at a very high level in flowers and stems (picked at anthesis), and at a lower level in ripe leaves and fruits. {ECO:0000269|PubMed:27242828}.</td><td style = \"text-align: left;\">INDUCTION: In fruits, up-regulated by gibberellic acid (GA3) or by cold treatment during storage. {ECO:0000269|PubMed:27242828}.</td><td style = \"text-align: left;\">cell wall assembly [GO:0070726]; fruit ripening [GO:0009835]; xyloglucan metabolic process [GO:0010411]</td><td style = \"text-align: left;\">apoplast [GO:0048046]; cytoplasm [GO:0005737]</td><td style = \"text-align: left;\">hydrolase activity, hydrolyzing O-glycosyl compounds [GO:0004553]; xyloglucan:xyloglucosyl transferase activity [GO:0016762]</td><td style = \"text-align: left;\">GO:0004553; GO:0005737; GO:0009835; GO:0010411; GO:0016762; GO:0048046; GO:0070726</td><td style = \"text-align: left;\">apoplast [GO:0048046]; cytoplasm [GO:0005737]; hydrolase activity, hydrolyzing O-glycosyl compounds [GO:0004553]; xyloglucan:xyloglucosyl transferase activity [GO:0016762]; cell wall assembly [GO:0070726]; fruit ripening [GO:0009835]; xyloglucan metabolic process [GO:0010411]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Cytoplasm {ECO:0000269|PubMed:27242828}. Note=Dispersed throughout the cell. {ECO:0000269|PubMed:27242828}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 1..268; /note=&quot;Xyloglucan endotransglucosylase protein 7&quot;; /id=&quot;PRO_0000454652&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISULFID 204..213; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;; DISULFID 251..265; /evidence=&quot;ECO:0000250|UniProtKB:Q8GZD5&quot;</td><td style = \"text-align: left;\">CARBOHYD 90; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PIRSR:PIRSR005604-2, ECO:0000255|PROSITE-ProRule:PRU00498&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PTM: Contains at least one intrachain disulfide bond essential for its enzymatic activity. {ECO:0000255|RuleBase:RU361120}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">27242828</td><td style = \"text-align: left;\">2022-02-23</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&dtdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">38</td><td style = \"text-align: right;\">37</td><td style = \"text-align: left;\">A0A0A0LLY1</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">SRBP1_CUCSA</td><td style = \"text-align: left;\">Small RNA binding protein 1, CsSRBP1</td><td style = \"text-align: left;\">SRBP1 Csa_2G247040</td><td style = \"text-align: left;\">Cucumis sativus (Cucumber)</td><td style = \"text-align: right;\">160</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">Csa_2G247040</td><td style = \"text-align: left;\">SRBP1</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">3659</td><td style = \"font-style: italic; text-align: left;\">UP000029981: Chromosome 2</td><td style = \"text-align: left;\">Cucumis (genus), Benincaseae (tribe), Cucurbitaceae (family), Cucurbitales (order), fabids (no rank), rosids (no rank), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">3655 (genus), 1003877 (tribe), 3650 (family), 71239 (order), 91835 (no rank), 71275 (no rank), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">15973</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MASSSVEFRCFVGGLAWATDSNSLEKAFSVYGEIVEAKIVSDRETGRSRGFGFVTFLEEEAMRSAIEAMNGHILDGRNITVNEAQQRGGGGGGGYNRGGGYGGRRDGGGFSRGGGGGYGGGGGGGYGGGRDRGYGGGGGYGGGRDSRGSGGGGSEGGWRN</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">FUNCTION: Possibly has a role in RNA transcription or processing during stress (By similarity). Binds sequence non-specifically to RNAs and DNAs (By similarity). Mediates cell-to-cell trafficking of RNA interference (RNAi) signals (small RNAs (sRNA), e.g. small interfering RNA (siRNA) and microRNA (miRNA)) which regulate growth and development, as well as responses to environmental inputs, including pathogen attack; can compromise zucchini yellow mosaic virus (ZYMV) and tobacco rattle virus (TRV) infections at the early stage (PubMed:31812689). {ECO:0000250|UniProtKB:Q9LIS2, ECO:0000269|PubMed:31812689}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Plant defense;Reference proteome;RNA-binding;Secreted</td><td style = \"text-align: left;\">KW-0611; KW-1185; KW-0694; KW-0964</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00052B1220</td><td style = \"text-align: left;\">DOMAIN (1); FUNCTION (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); SUBUNIT (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Chain (1); Domain (1); Region (2)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">SUBUNIT: Binds to small phloem-mobile single-stranded RNAs (ss-sRNA, e.g. small interfering RNA (siRNA) and microRNA (miRNA)) in the phloeme exudate, including viral-derived sRNA (vsiRNA). {ECO:0000269|PubMed:31812689}.</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Accumulates in phloem exudates. {ECO:0000269|PubMed:31812689}.</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">alternative mRNA splicing, via spliceosome [GO:0000380]; defense response [GO:0006952]; extracellular transport [GO:0006858]; miRNA transport [GO:1990428]; regulation of defense response to virus [GO:0050688]; RNA transport [GO:0050658]</td><td style = \"text-align: left;\">extracellular space [GO:0005615]; nucleus [GO:0005634]</td><td style = \"text-align: left;\">miRNA binding [GO:0035198]; mRNA 3&apos;-UTR binding [GO:0003730]; single-stranded RNA binding [GO:0003727]; siRNA binding [GO:0035197]</td><td style = \"text-align: left;\">GO:0000380; GO:0003727; GO:0003730; GO:0005615; GO:0005634; GO:0006858; GO:0006952; GO:0035197; GO:0035198; GO:0050658; GO:0050688; GO:1990428</td><td style = \"text-align: left;\">extracellular space [GO:0005615]; nucleus [GO:0005634]; miRNA binding [GO:0035198]; mRNA 3&apos;-UTR binding [GO:0003730]; single-stranded RNA binding [GO:0003727]; siRNA binding [GO:0035197]; alternative mRNA splicing, via spliceosome [GO:0000380]; defense response [GO:0006952]; extracellular transport [GO:0006858]; miRNA transport [GO:1990428]; regulation of defense response to virus [GO:0050688]; RNA transport [GO:0050658]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Secreted {ECO:0000269|PubMed:31812689}. Note=Observed in the phloem translocation stream. {ECO:0000269|PubMed:31812689}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 1..160; /note=&quot;Small RNA binding protein 1&quot;; /id=&quot;PRO_0000454536&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">31812689; 19881527</td><td style = \"text-align: left;\">2022-02-23</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">39</td><td style = \"text-align: right;\">38</td><td style = \"text-align: left;\">A0A0A1C3I2</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">HMGR1_PANGI</td><td style = \"text-align: left;\">3-hydroxy-3-methylglutaryl coenzyme A reductase 1, HMG-CoA reductase 1, Hydroxymethylglutaryl-CoA reductase, PgHMGR1, EC 1.1.1.34</td><td style = \"text-align: left;\">HMGR1 HMGR</td><td style = \"text-align: left;\">Panax ginseng (Korean ginseng)</td><td style = \"text-align: right;\">573</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">HMGR1</td><td style = \"font-style: italic; text-align: left;\">HMGR</td><td style = \"text-align: right;\">4054</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Panax (genus), Araliaceae (family), Apiineae (suborder), Apiales (order), campanulids (no rank), asterids (no rank), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">4053 (genus), 4050 (family), 364270 (suborder), 4036 (order), 91882 (no rank), 71274 (no rank), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">61680</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MDVRRRLPPKLRRPLPITESSHHHRKTPFPADVDRSPSPTPKASDALPLPLYLTNGIFFTLFFSVAYYLLHRWRDKIRSSTPLHIVTLSELAAIVSLIASFIYLLGFFGIDFVQSFVSRADVDVDIDVEPDILEADRRPCSKLMDQPPPPPVVMSSEEDEEIVKSVVSGKTPSYSLESKLGDCYRAASIRREAVQRTTGRSLLGLPLDGFDYESILGQCCEMPIGYVQIPVGIAGPLLLNGCEYVVPMATTEGCLVASTNRGCKAIYACGGATGILLKDGMTRAPVVRFSTAKRASDLKFFLEDPLNFDTLAVVFNKSSRFARLQSIQCSMAGKNLYIRFCCSTGDAMGMNMVSKGVQNVLEFLQSDFPDMDVIGISGNFCSDKKPAAVNWIEGRGKSVVCEAIITDDVVKKVLKTTVPALVELNMLKNLAGSAVAGALGGFNAHAANIVSAVFIATGQDPAQNIESSHCITMMEAINNGKDLHISVTMPSIEVGTVGGGTQLASQSACLNLLGVKGANKESHGSNSRLLATIVAGSVLAGELSLMSAIAAGQLVRSHMKYNRSSRDMSKIGS</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">CONFLICT 2..4; /note=&quot;DVR -&gt; ETS (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 10..11; /note=&quot;KL -&gt; IP (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 30; /note=&quot;P -&gt; L (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 57; /note=&quot;I -&gt; V (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 135; /note=&quot;A -&gt; T (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 148; /note=&quot;P -&gt; L (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 168; /note=&quot;S -&gt; T (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 306; /note=&quot;L -&gt; F (in Ref. 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 315..316; /note=&quot;FN -&gt; SK (in Ref. 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 316; /note=&quot;N -&gt; K (in Ref. 2; AJV26444)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 340; /note=&quot;F -&gt; L (in Ref. 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 345..347; /note=&quot;GDA -&gt; VDV (in Ref. 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 374; /note=&quot;I -&gt; T (in Ref. 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 407; /note=&quot;D -&gt; E (in Ref. 3; ACW83617)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 413; /note=&quot;V -&gt; E (in Ref. 3; ACW83617)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 425; /note=&quot;N -&gt; NH (in Ref. 3; ACW83617)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 427; /note=&quot;L -&gt; V (in Ref. 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 438; /note=&quot;A -&gt; S (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 454..456; /note=&quot;FIA -&gt; YIP (in Ref. 3; ACW83617)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 469; /note=&quot;H -&gt; Q (in Ref. 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;; CONFLICT 479; /note=&quot;N -&gt; D (in Ref. 2; AJV26444 and 3; ADI80346)&quot;; /evidence=&quot;ECO:0000305&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">ACT_SITE 252; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P04035&quot;; ACT_SITE 384; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P04035&quot;; ACT_SITE 460; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P04035&quot;; ACT_SITE 558; /note=&quot;Proton donor&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU10003&quot;</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=(R)-mevalonate + CoA + 2 NADP(+) = (3S)-hydroxy-3-methylglutaryl-CoA + 2 H(+) + 2 NADPH; Xref=Rhea:RHEA:15989, ChEBI:CHEBI:15378, ChEBI:CHEBI:36464, ChEBI:CHEBI:43074, ChEBI:CHEBI:57287, ChEBI:CHEBI:57783, ChEBI:CHEBI:58349; EC=1.1.1.34; Evidence={ECO:0000255|PROSITE-ProRule:PRU10003};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">1.1.1.34</td><td style = \"text-align: left;\">FUNCTION: Catalyzes the synthesis of mevalonate, the specific precursor of all isoprenoid compounds present in plants (By similarity). Component of the triterpene saponins (e.g. ginsenosides or panaxosides) and phytosterols biosynthetic pathways (PubMed:24569845, PubMed:29378087). Promotes triterpenes accumulation in roots (PubMed:24569845). {ECO:0000250|UniProtKB:P14891, ECO:0000269|PubMed:24569845, ECO:0000303|PubMed:29378087}.</td><td style = \"font-style: italic; text-align: left;\">ACTIVITY REGULATION: Competitive inhibition by mevinolin (Mev) is leading to a significant reduction of total ginsenoside in adventitious roots. Triggered by darkness. {ECO:0000269|PubMed:24569845}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">PATHWAY: Metabolic intermediate biosynthesis; (R)-mevalonate biosynthesis; (R)-mevalonate from acetyl-CoA: step 3/3. {ECO:0000305}.</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">RHEA:15989</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Chloroplast;Endoplasmic reticulum;Glycoprotein;Isoprene biosynthesis;Membrane;NADP;Oxidoreductase;Peroxisome;Plastid;Transmembrane;Transmembrane helix</td><td style = \"text-align: left;\">KW-0150; KW-0256; KW-0325; KW-0414; KW-0472; KW-0521; KW-0560; KW-0576; KW-0934; KW-0812; KW-1133</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at transcript level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000535EB1D</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); CATALYTIC ACTIVITY (1); DEVELOPMENTAL STAGE (1); FUNCTION (1); INDUCTION (1); PATHWAY (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Active site (4); Chain (1); Compositional bias (1); Glycosylation (1); Region (1); Sequence conflict (21); Topological domain (4); Transmembrane (3)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DEVELOPMENTAL STAGE: Relatively constant levels in mature roots (PubMed:24569845). Rapid decrease in roots and leaves from the leaf opened to the green fruit stage (PubMed:30577538). At the leaf opened stage, accumulates mostly in roots (PubMed:30577538). {ECO:0000269|PubMed:24569845, ECO:0000269|PubMed:30577538}.</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Mostly expressed in the petioles of seedlings and roots (both in main roots and lateral roots), and, to a lower extent, in seeds, seedlings, leaves, stems and flowers. {ECO:0000269|PubMed:24569845, ECO:0000269|PubMed:30577538}.</td><td style = \"text-align: left;\">INDUCTION: Induced in the hypocotyl of a 4 days-etiolated seedling and reversed by light exposure within 2 hours. Inhibited by dark treatment in 7-days-old seedling, where true leaves are emerged. Repressed by darkness in roots and leaves, but increased ginsenosides accumulation. Triggered by methyl jasmonate (MJ). Repressed transiently by mevinolin (Mev) (PubMed:24569845). Influenced in roots by relative humidity and photosynthetically active radiation (PAR), and in leaves by rain and soil water potential (PubMed:30577538). {ECO:0000269|PubMed:24569845, ECO:0000269|PubMed:30577538}.</td><td style = \"text-align: left;\">coenzyme A metabolic process [GO:0015936]; response to absence of light [GO:0009646]; response to jasmonic acid [GO:0009753]; saponin biosynthetic process [GO:0016135]; triterpenoid biosynthetic process [GO:0016104]</td><td style = \"text-align: left;\">chloroplast membrane [GO:0031969]; endoplasmic reticulum membrane [GO:0005789]; integral component of membrane [GO:0016021]; peroxisomal membrane [GO:0005778]</td><td style = \"text-align: left;\">hydroxymethylglutaryl-CoA reductase (NADPH) activity [GO:0004420]</td><td style = \"text-align: left;\">GO:0004420; GO:0005778; GO:0005789; GO:0009646; GO:0009753; GO:0015936; GO:0016021; GO:0016104; GO:0016135; GO:0031969</td><td style = \"text-align: left;\">chloroplast membrane [GO:0031969]; endoplasmic reticulum membrane [GO:0005789]; integral component of membrane [GO:0016021]; peroxisomal membrane [GO:0005778]; hydroxymethylglutaryl-CoA reductase (NADPH) activity [GO:0004420]; coenzyme A metabolic process [GO:0015936]; response to absence of light [GO:0009646]; response to jasmonic acid [GO:0009753]; saponin biosynthetic process [GO:0016135]; triterpenoid biosynthetic process [GO:0016104]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Endoplasmic reticulum membrane {ECO:0000269|PubMed:24569845}; Multi-pass membrane protein {ECO:0000255}. Plastid, chloroplast membrane {ECO:0000269|PubMed:24569845}; Multi-pass membrane protein {ECO:0000255}. Peroxisome membrane {ECO:0000269|PubMed:24569845}; Multi-pass membrane protein {ECO:0000255}. Note=Localized in intracellular vesicles. {ECO:0000269|PubMed:24569845}.</td><td style = \"font-style: italic; text-align: left;\">TOPO_DOM 1..45; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305|PubMed:24569845&quot;; TOPO_DOM 67..92; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305|PubMed:24569845&quot;; TOPO_DOM 114..528; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305|PubMed:24569845&quot;; TOPO_DOM 550..573; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305|PubMed:24569845&quot;</td><td style = \"font-style: italic; text-align: left;\">TRANSMEM 46..66; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 93..113; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 529..549; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"text-align: left;\">CHAIN 1..573; /note=&quot;3-hydroxy-3-methylglutaryl coenzyme A reductase 1&quot;; /id=&quot;PRO_0000446946&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">CARBOHYD 316; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">24569845; 29378087; 29509695; 30577538</td><td style = \"text-align: left;\">2019-05-08</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">40</td><td style = \"text-align: right;\">39</td><td style = \"text-align: left;\">A0A0A1C930</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">HMR2A_PANGI</td><td style = \"text-align: left;\">3-hydroxy-3-methylglutaryl coenzyme A reductase 2-A, HMG-CoA reductase 2, Hydroxymethylglutaryl-CoA reductase, PgHMGR2, EC 1.1.1.34</td><td style = \"text-align: left;\">HMGR2</td><td style = \"text-align: left;\">Panax ginseng (Korean ginseng)</td><td style = \"text-align: right;\">594</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">HMGR2</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">4054</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Panax (genus), Araliaceae (family), Apiineae (suborder), Apiales (order), campanulids (no rank), asterids (no rank), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">4053 (genus), 4050 (family), 364270 (suborder), 4036 (order), 91882 (no rank), 71274 (no rank), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">63621</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MDVRRRPVKSLSSAKTATAGEPPKSQQQHPKASDALPLPLYLTNGLFFTMFFSVMYFLLHRWREKIRNSTPLHVVTLSELAALVLLMASVIYLLGFFGIGFVRSVIRPSPDAWDILEDDNAINEEDSRREPCAEAIDCSLPPKPKIVHMVPQKALNPKSAFADMMVEQPALAIAPLTEEDEEIVKSVVTGKIPSYSLESKLGDCKKAASIRREALQRITGKSLAGLPLDGFDYKSILGQCCEMPVGYVQIPVGIAGPLLLNETEYSVPMATTEGCLVASTNRGCKAIYASGGATSVLLRDGMTRAPVVRFSTVKRAAELKFFLEEPLNYDTPAHVFNKSSRFGRLQGIKCAVAGKNLYIRFTCSTGDAMGMNMVSKGVQNVLDFLQSDFPDMDVMGISGNYCSDKKPAAVNWIEGCGKSVVCEAIIKEEVVKKVLKTNVAALVELNMLKNLAGSAVAGALGGFNAHASNIVSAVYISTGQDPAQNVESSHCITMMEAVNNGKDLHISVTMPSIEVGTVGGGTQLASQSACLNLLGVKGASKESPGSNSRLLASIVAGSVLAGELSLMSALAAGQLVKSHMKYNRSSKDITKLSS</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">ACT_SITE 273; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P04035&quot;; ACT_SITE 405; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P04035&quot;; ACT_SITE 481; /note=&quot;Charge relay system&quot;; /evidence=&quot;ECO:0000250|UniProtKB:P04035&quot;; ACT_SITE 579; /note=&quot;Proton donor&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU10003&quot;</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=(R)-mevalonate + CoA + 2 NADP(+) = (3S)-hydroxy-3-methylglutaryl-CoA + 2 H(+) + 2 NADPH; Xref=Rhea:RHEA:15989, ChEBI:CHEBI:15378, ChEBI:CHEBI:36464, ChEBI:CHEBI:43074, ChEBI:CHEBI:57287, ChEBI:CHEBI:57783, ChEBI:CHEBI:58349; EC=1.1.1.34; Evidence={ECO:0000255|PROSITE-ProRule:PRU10003};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">1.1.1.34</td><td style = \"text-align: left;\">FUNCTION: Catalyzes the synthesis of mevalonate, the specific precursor of all isoprenoid compounds present in plants (By similarity). Component of the triterpene saponins (e.g. ginsenosides or panaxosides) and phytosterols biosynthetic pathways (PubMed:24569845, PubMed:29378087). {ECO:0000250|UniProtKB:P14891, ECO:0000269|PubMed:24569845, ECO:0000303|PubMed:29378087}.</td><td style = \"font-style: italic; text-align: left;\">ACTIVITY REGULATION: Competitive inhibition by mevinolin (Mev) is leading to a significant reduction of total ginsenoside in adventitious roots. Triggered by darkness. {ECO:0000269|PubMed:24569845}.</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">PATHWAY: Metabolic intermediate biosynthesis; (R)-mevalonate biosynthesis; (R)-mevalonate from acetyl-CoA: step 3/3. {ECO:0000305}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:15989</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Chloroplast;Endoplasmic reticulum;Glycoprotein;Isoprene biosynthesis;Membrane;NADP;Oxidoreductase;Peroxisome;Plastid;Transmembrane;Transmembrane helix</td><td style = \"text-align: left;\">KW-0150; KW-0256; KW-0325; KW-0414; KW-0472; KW-0521; KW-0560; KW-0576; KW-0934; KW-0812; KW-1133</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at transcript level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000535B7AC</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); CATALYTIC ACTIVITY (1); DEVELOPMENTAL STAGE (1); FUNCTION (1); INDUCTION (1); PATHWAY (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Active site (4); Chain (1); Compositional bias (1); Glycosylation (2); Region (1); Topological domain (4); Transmembrane (3)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">DEVELOPMENTAL STAGE: Accumulates progressively in mature roots. {ECO:0000269|PubMed:24569845}.</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Mostly expressed in the petioles of seedlings, seedlings and roots, and, to a lower extent, in seeds, leaves, stems and flowers. {ECO:0000269|PubMed:24569845}.</td><td style = \"font-style: italic; text-align: left;\">INDUCTION: Repressed by darkness in roots and leaves, but increased ginsenosides accumulation. Repressed transiently by mevinolin (Mev). {ECO:0000269|PubMed:24569845}.</td><td style = \"text-align: left;\">coenzyme A metabolic process [GO:0015936]; response to jasmonic acid [GO:0009753]; saponin biosynthetic process [GO:0016135]; triterpenoid biosynthetic process [GO:0016104]</td><td style = \"font-style: italic; text-align: left;\">chloroplast membrane [GO:0031969]; endoplasmic reticulum membrane [GO:0005789]; integral component of membrane [GO:0016021]; peroxisomal membrane [GO:0005778]</td><td style = \"text-align: left;\">hydroxymethylglutaryl-CoA reductase (NADPH) activity [GO:0004420]</td><td style = \"text-align: left;\">GO:0004420; GO:0005778; GO:0005789; GO:0009753; GO:0015936; GO:0016021; GO:0016104; GO:0016135; GO:0031969</td><td style = \"text-align: left;\">chloroplast membrane [GO:0031969]; endoplasmic reticulum membrane [GO:0005789]; integral component of membrane [GO:0016021]; peroxisomal membrane [GO:0005778]; hydroxymethylglutaryl-CoA reductase (NADPH) activity [GO:0004420]; coenzyme A metabolic process [GO:0015936]; response to jasmonic acid [GO:0009753]; saponin biosynthetic process [GO:0016135]; triterpenoid biosynthetic process [GO:0016104]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">SUBCELLULAR LOCATION: Endoplasmic reticulum membrane {ECO:0000250|UniProtKB:A0A0A1C3I2}; Multi-pass membrane protein {ECO:0000255}. Plastid, chloroplast membrane {ECO:0000250|UniProtKB:A0A0A1C3I2}; Multi-pass membrane protein {ECO:0000255}. Peroxisome membrane {ECO:0000250|UniProtKB:A0A0A1C3I2}; Multi-pass membrane protein {ECO:0000255}. Note=Localized in intracellular vesicles. {ECO:0000269|PubMed:24569845}.</td><td style = \"font-style: italic; text-align: left;\">TOPO_DOM 1..37; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305|PubMed:24569845&quot;; TOPO_DOM 59..81; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305|PubMed:24569845&quot;; TOPO_DOM 103..549; /note=&quot;Lumenal&quot;; /evidence=&quot;ECO:0000305|PubMed:24569845&quot;; TOPO_DOM 571..594; /note=&quot;Cytoplasmic&quot;; /evidence=&quot;ECO:0000305|PubMed:24569845&quot;</td><td style = \"font-style: italic; text-align: left;\">TRANSMEM 38..58; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 82..102; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;; TRANSMEM 550..570; /note=&quot;Helical&quot;; /evidence=&quot;ECO:0000255&quot;</td><td style = \"text-align: left;\">CHAIN 1..594; /note=&quot;3-hydroxy-3-methylglutaryl coenzyme A reductase 2-A&quot;; /id=&quot;PRO_0000446947&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">CARBOHYD 261; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;; CARBOHYD 337; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">24569845; 29378087; 29509695</td><td style = \"text-align: left;\">2019-05-08</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">41</td><td style = \"text-align: right;\">40</td><td style = \"text-align: left;\">A0A0A1H8I4</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">AIS_PSESP</td><td style = \"text-align: left;\">Aconitate isomerase, AI, EC 5.3.3.7</td><td style = \"text-align: left;\">ais</td><td style = \"text-align: left;\">Pseudomonas sp</td><td style = \"text-align: right;\">262</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ais</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">306</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">unclassified Pseudomonas (no rank), Pseudomonas (genus), Pseudomonadaceae (family), Pseudomonadales (order), Gammaproteobacteria (class), Proteobacteria (phylum), Bacteria (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">196821 (no rank), 286 (genus), 135621 (family), 72274 (order), 1236 (class), 1224 (phylum), 2 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">27449</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MFPRLPTLALGALLLASTPLLAAQPVTTLTVLSSGGIMGTIREVAPAYEKATGVKLDIAAAPSMGDTPQAIPNRLARNEPADVVLMVGSALDKLVASGQVAKDSRVDLGQSFIAMAVRQGAPKPDISNMDAFKQTLEKAQSVAYSDSASGVYLSRILFPRMQLDKSFMAKARMIPAEPVGAVVARGEAQLGFQQLSELKAVPGIDIVGLIPDQAQKMTLYSGAMVSKSQHPEAARALLQYLASKDAAKAIEDSGLKPVPAQP</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=trans-aconitate = cis-aconitate; Xref=Rhea:RHEA:17265, ChEBI:CHEBI:15708, ChEBI:CHEBI:16383; EC=5.3.3.7; Evidence={ECO:0000269|PubMed:26293748};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">5.3.3.7</td><td style = \"text-align: left;\">FUNCTION: Involved in assimilation of trans-aconitic acid. Preference for cis-aconitic acid is 14-fold higher than for trans-aconitic acid. Not active on intermediates of tricarboxylic acid (TCA) cycle including citric acid, succinic acid, fumaric acid, and 2-oxoglutaric acid or on other dicarboxilic acids including itaconic acid, formic acid, citraconic acid or maleic acid. {ECO:0000269|PubMed:26293748}.</td><td style = \"font-style: italic; text-align: left;\">ACTIVITY REGULATION: Activated more than 1.5 fold by Ca(2+), Mg(2+), Mn(2+), Ni(2+), Fe(2+), DDT and 1,10-phenanthroline. Strongly inhibited by Ag(+) and Hg(+). Inhibited by addition of 20% (v/v) glycerol. No effect by addition of NADH or NADPH. {ECO:0000269|PubMed:26293748}.</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  Kinetic parameters: KM=5.9 mM for cis-aconitic acid {ECO:0000269|PubMed:26293748}; KM=80 mM for trans-aconitic acid {ECO:0000269|PubMed:26293748}; Note=kcat and kcat/KM are 4500 sec(-1) and 760 sec(-1) mM(-1), respectively for cis-aconitic acid. kcat and kcat/KM are 15000 sec(-1) and 190 sec(-1) mM(-1), respectively for trans-aconitic acid. {ECO:0000269|PubMed:26293748};</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  pH dependence: Optimum pH is 6.0. {ECO:0000269|PubMed:26293748};</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">RHEA:17265</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  Temperature dependence: Optimum temperature is 37 degrees Celsius. {ECO:0000269|PubMed:26293748};</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Direct protein sequencing;Isomerase;Signal</td><td style = \"text-align: left;\">KW-0903; KW-0413; KW-0732</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI0005360B17</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); BIOPHYSICOCHEMICAL PROPERTIES (1); BIOTECHNOLOGY (1); CATALYTIC ACTIVITY (1); FUNCTION (1); SUBUNIT (1)</td><td style = \"text-align: left;\">Chain (1); Signal (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">SUBUNIT: Monomer. {ECO:0000269|PubMed:26293748}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">cellular response to calcium ion [GO:0071277]; cellular response to iron ion [GO:0071281]; cellular response to magnesium ion [GO:0071286]; cellular response to manganese ion [GO:0071287]; cellular response to nickel ion [GO:0071289]; cellular response to silver ion [GO:0071292]; response to DDT [GO:0046680]; response to mercury ion [GO:0046689]</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">aconitate delta-isomerase activity [GO:0047614]</td><td style = \"text-align: left;\">GO:0046680; GO:0046689; GO:0047614; GO:0071277; GO:0071281; GO:0071286; GO:0071287; GO:0071289; GO:0071292</td><td style = \"text-align: left;\">aconitate delta-isomerase activity [GO:0047614]; cellular response to calcium ion [GO:0071277]; cellular response to iron ion [GO:0071281]; cellular response to magnesium ion [GO:0071286]; cellular response to manganese ion [GO:0071287]; cellular response to nickel ion [GO:0071289]; cellular response to silver ion [GO:0071292]; response to DDT [GO:0046680]; response to mercury ion [GO:0046689]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">BIOTECHNOLOGY: May be useful in development of a bioprocess for effective production of trans-aconitic acid. {ECO:0000303|PubMed:26293748}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 23..262; /note=&quot;Aconitate isomerase&quot;; /id=&quot;PRO_0000434591&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SIGNAL 1..22; /evidence=&quot;ECO:0000269|PubMed:26293748&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">26293748</td><td style = \"text-align: left;\">2015-11-11</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">42</td><td style = \"text-align: right;\">41</td><td style = \"text-align: left;\">A0A0A1HA03</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">708C1_FAGES</td><td style = \"text-align: left;\">UDP-glycosyltransferase 708C1, EC 2.4.1.360 (C-glucosyltransferase a, FeCGTa) (UDP-glucose:2-hydroxyflavanone C-glucosyltransferase)</td><td style = \"text-align: left;\">UGT708C1 CGTa</td><td style = \"text-align: left;\">Fagopyrum esculentum (Common buckwheat) (Polygonum fagopyrum)</td><td style = \"text-align: right;\">457</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">UGT708C1</td><td style = \"font-style: italic; text-align: left;\">CGTa</td><td style = \"text-align: right;\">3617</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Fagopyrum (genus), Fagopyreae (tribe), Polygonoideae (subfamily), Polygonaceae (family), Caryophyllales (order), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">3616 (genus), 1110383 (tribe), 1110380 (subfamily), 3615 (family), 3524 (order), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: right;\">50375</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MMGDLTTSFPATTLTTNDQPHVVVCSGAGMGHLTPFLNLASALSSAPYNCKVTLLIVIPLITDAESHHISSFFSSHPTIHRLDFHVNLPAPKPNVDPFFLRYKSISDSAHRLPVHLSALSPPISAVFSDFLFTQGLNTTLPHLPNYTFTTTSARFFTLMSYVPHLAKSSSSSPVEIPGLEPFPTDNIPPPFFNPEHIFTSFTISNAKYFSLSKGILVNTFDSFEPETLSALNSGDTLSDLPPVIPIGPLNELEHNKQEELLPWLDQQPEKSVLYVSFGNRTAMSSDQILELGMGLERSDCRFIWVVKTSKIDKDDKSELRKLFGEELYLKLSEKGKLVKWVNQTEILGHTAVGGFLSHCGWNSVMEAARRGVPILAWPQHGDQRENAWVVEKAGLGVWEREWASGIQAAIVEKVKMIMGNNDLRKSAMKVGEEAKRACDVGGSSATALMNIIGSLKR</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">ACT_SITE 32; /note=&quot;Proton acceptor&quot;; /evidence=&quot;ECO:0000305|PubMed:32699169&quot;; ACT_SITE 129; /note=&quot;Charge relay&quot;; /evidence=&quot;ECO:0000305|PubMed:32699169&quot;</td><td style = \"text-align: left;\">BINDING 31..34; /ligand=&quot;UDP-alpha-D-glucose&quot;; /ligand_id=&quot;ChEBI:CHEBI:58885&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169, ECO:0007744|PDB:6LLZ&quot;; BINDING 150; /ligand=&quot;UDP-alpha-D-glucose&quot;; /ligand_id=&quot;ChEBI:CHEBI:58885&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169, ECO:0007744|PDB:6LLZ&quot;; BINDING 340..343; /ligand=&quot;UDP-alpha-D-glucose&quot;; /ligand_id=&quot;ChEBI:CHEBI:58885&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169, ECO:0007744|PDB:6LLZ&quot;; BINDING 358..366; /ligand=&quot;UDP-alpha-D-glucose&quot;; /ligand_id=&quot;ChEBI:CHEBI:58885&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169, ECO:0007744|PDB:6LLZ&quot;; BINDING 382..383; /ligand=&quot;UDP-alpha-D-glucose&quot;; /ligand_id=&quot;ChEBI:CHEBI:58885&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169, ECO:0007744|PDB:6LLZ&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=a 3&apos;-hydro-2&apos;-hydroxy-beta-oxodihydrochalcone + UDP-alpha-D-glucose = a 3&apos;-(beta-D-glucopyranosyl)-2&apos;-hydroxy-beta-oxodihydrochalcone + H(+) + UDP; Xref=Rhea:RHEA:51504, ChEBI:CHEBI:15378, ChEBI:CHEBI:58223, ChEBI:CHEBI:58885, ChEBI:CHEBI:142482, ChEBI:CHEBI:142483; EC=2.4.1.360; Evidence={ECO:0000269|PubMed:25142187, ECO:0000269|PubMed:32699169}; PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:51505; Evidence={ECO:0000269|PubMed:25142187, ECO:0000269|PubMed:32699169};</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.4.1.360</td><td style = \"text-align: left;\">FUNCTION: UDP-glucose-dependent glucosyltransferase catalyzing the C-glucosylation of 2-hydroxyflavanones (2-hydroxynaringenin, 2-hydroxyeriodictyol and 2-hydroxypinocembrin) and phloretin (PubMed:25142187). No activity with flavanones, flavones or flavonols (PubMed:25142187). Exhibits C-glycosylation activity toward 2&apos;,4&apos;,6&apos;-trihydroxyacetophenone and phloretin using UDP-glucose as sugar donor (PubMed:32699169). Can use UDP-galactose as sugar donor, but catalytic efficiency is 14-fold lower toward UDP-galactose than toward UDP-glucose (PubMed:32699169). {ECO:0000269|PubMed:25142187, ECO:0000269|PubMed:32699169}.</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  Kinetic parameters: KM=4.4 uM for 2-hydroxynaringenin {ECO:0000269|PubMed:25142187}; KM=40 uM for 2-hydroxypinocembrin {ECO:0000269|PubMed:25142187}; KM=36.5 uM for 2-phenyl-2&apos;,4&apos;,6&apos;-trihydroxyactophenone {ECO:0000269|PubMed:25142187}; KM=3.73 uM for phloretin {ECO:0000269|PubMed:32699169}; KM=14.66 uM for 2&apos;,4&apos;,6&apos;-trihydroxyacetophenone {ECO:0000269|PubMed:32699169}; KM=58.1 uM for UDP-glucose with 2-hydroxypinocembrin as acceptor {ECO:0000269|PubMed:25142187}; KM=56.95 uM for UDP-glucose with phloretin as acceptor {ECO:0000269|PubMed:32699169}; KM=42.55 uM for UDP-glucose with 2&apos;,4&apos;,6&apos;-trihydroxyacetophenone as acceptor {ECO:0000269|PubMed:32699169};</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  pH dependence: Optimum pH is 6.5-7.0. {ECO:0000269|PubMed:25142187};</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:51504</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  Temperature dependence: Optimum temperature is 45-50 degrees Celsius. {ECO:0000269|PubMed:25142187};</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;Direct protein sequencing;Glycosyltransferase;Transferase</td><td style = \"text-align: left;\">KW-0002; KW-0903; KW-0328; KW-0808</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000535F106</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES (1); CATALYTIC ACTIVITY (1); DEVELOPMENTAL STAGE (1); FUNCTION (1); SIMILARITY (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Active site (2); Beta strand (19); Binding site (5); Chain (1); Helix (25); Mutagenesis (8); Region (1); Turn (2)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">DEVELOPMENTAL STAGE: Expressed in germinating seeds and during cotyledon development. {ECO:0000269|PubMed:25142187}.</td><td style = \"font-style: italic; text-align: left;\">TISSUE SPECIFICITY: Expressed in cotyledons. Not detected in flowers, leaves, roots and hypocotyls. {ECO:0000269|PubMed:25142187}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">UDP-glucosyltransferase activity [GO:0035251]</td><td style = \"text-align: left;\">GO:0035251</td><td style = \"text-align: left;\">UDP-glucosyltransferase activity [GO:0035251]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">MUTAGEN 102; /note=&quot;Y-&gt;F: Decreases affinity for phloretin 2.9-fold and increases affinity for UDP-glucose 2-fold.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169&quot;; MUTAGEN 130; /note=&quot;F-&gt;A: Decreases affinity for phloretin 2.1-fold and affinity for UDP-glucose 1.5-fold.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169&quot;; MUTAGEN 150; /note=&quot;T-&gt;A: Decreases affinity for phloretin 2.7-fold and increases affinity for UDP-glucose 1.2-fold.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169&quot;; MUTAGEN 151; /note=&quot;T-&gt;A: Decreases affinity for phloretin 2-fold and affinity for UDP-glucose 4.9-fold.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169&quot;; MUTAGEN 198; /note=&quot;F-&gt;A: Decreases affinity for phloretin 6.4-fold and increases affinity for UDP-glucose 1.4-fold.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169&quot;; MUTAGEN 382; /note=&quot;D-&gt;E: Decreases affinity for phloretin 2.8-fold and affinity for UDP-glucose 1.2-fold.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169&quot;; MUTAGEN 383; /note=&quot;Q-&gt;A: Decreases affinity for phloretin 3.5-fold and affinity for UDP-glucose 3.6-fold.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169&quot;; MUTAGEN 383; /note=&quot;Q-&gt;H: Decreases affinity for phloretin 3.7-fold and affinity for UDP-glucose 2.1-fold.&quot;; /evidence=&quot;ECO:0000269|PubMed:32699169&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 1..457; /note=&quot;UDP-glycosyltransferase 708C1&quot;; /id=&quot;PRO_0000436254&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">X-ray crystallography (4)</td><td style = \"font-style: italic; text-align: left;\">STRAND 21..25; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 51..59; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 79..85; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 93..95; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 119..121; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 123..128; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 139..141; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 145..151; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 170..172; /evidence=&quot;ECO:0007829|PDB:6LLG&quot;; STRAND 212..218; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 234..237; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 243..245; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 272..276; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 301..305; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 335..339; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 352..357; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 374..376; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 395..398; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; STRAND 419..421; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;</td><td style = \"font-style: italic; text-align: left;\">HELIX 30..44; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 63..75; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 97..108; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 109..111; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 112..117; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 130..132; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 133..138; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 153..161; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 162..164; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 184..186; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 189..192; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 197..206; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 207..211; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 225..232; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 261..265; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 285..298; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 317..323; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 325..334; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 343..347; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 361..370; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 383..392; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 402..404; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 406..418; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 422..438; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; HELIX 443..454; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;</td><td style = \"font-style: italic; text-align: left;\">TURN 46..48; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;; TURN 221..223; /evidence=&quot;ECO:0007829|PDB:7CYW&quot;</td><td style = \"text-align: left;\">25142187; 32699169</td><td style = \"text-align: left;\">2016-05-11</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">43</td><td style = \"text-align: right;\">42</td><td style = \"text-align: left;\">A0A0A1I6E7</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">NDB4S_ANDCR</td><td style = \"text-align: left;\">Antimicrobial peptide AcrAP1</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Androctonus crassicauda (Arabian fat-tailed scorpion)</td><td style = \"text-align: right;\">74</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: right;\">122909</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Androctonus (genus), Buthidae (family), Buthoidea (superfamily), Buthida (parvorder), Scorpiones (order), Arachnida (class), Chelicerata (subphylum), Arthropoda (phylum), Panarthropoda (no rank), Ecdysozoa (no rank), Protostomia (no rank), Bilateria (no rank), Eumetazoa (no rank), Metazoa (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">6857 (genus), 6856 (family), 70336 (superfamily), 259437 (parvorder), 6855 (order), 6854 (class), 6843 (subphylum), 6656 (phylum), 88770 (no rank), 1206794 (no rank), 33317 (no rank), 33213 (no rank), 6072 (no rank), 33208 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">8679</td><td style = \"font-style: italic; text-align: left;\">MASS SPECTROMETRY: Mass=1961.76; Method=Electrospray; Evidence={ECO:0000269|PubMed:25332684};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MEIKYLLTVFLVLLIVSDHCQAFLFSLIPHAISGLISAFKGRRKRDLDGQIDRFRNFRKRDAELEELLSKLPIY</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">FUNCTION: Has antimicrobial activity against the Gram-positive bacteria S.aureus (MIC=8 uM) and the yeast C.albicans (MIC=16 uM). Causes hemolysis on horse erythrocytes (64 uM for 100% hemolysis). Minimum bactericidal concentrations have also been tested against S.aureus and is four-fold higher (MBC=32 uM). {ECO:0000269|PubMed:25332684}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Amidation;Antibiotic;Antimicrobial;Cleavage on pair of basic residues;Fungicide;Membrane;Secreted;Signal;Target cell membrane;Target membrane</td><td style = \"text-align: left;\">KW-0027; KW-0044; KW-0929; KW-0165; KW-0295; KW-0472; KW-0964; KW-0732; KW-1052; KW-1053</td><td style = \"text-align: left;\">MISCELLANEOUS: Does not show antimicrobial activity against the Gram-negative bacteria E.coli (MIC&gt;250 uM). Does not show effect on four different human cancer cell lines. {ECO:0000269|PubMed:25332684}.</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00052A686E</td><td style = \"text-align: left;\">FUNCTION (1); MASS SPECTROMETRY (1); MISCELLANEOUS (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Modified residue (1); Mutagenesis (4); Peptide (1); Propeptide (1); Signal (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Expressed by the venom gland. {ECO:0000305}.</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">defense response to bacterium [GO:0042742]; defense response to fungus [GO:0050832]; killing of cells of another organism [GO:0031640]</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; membrane [GO:0016020]</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">GO:0005576; GO:0016020; GO:0031640; GO:0042742; GO:0050832</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; membrane [GO:0016020]; defense response to bacterium [GO:0042742]; defense response to fungus [GO:0050832]; killing of cells of another organism [GO:0031640]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">MUTAGEN 26; /note=&quot;S-&gt;K: Shows important increase in antimicrobial activity against E.coli (MIC=8 uM) and slight increase against S.aureus (MIC=4 uM) and C.albicans (MIC=4 uM), as well as slight increase in hemolytic activity (32 uM for 100% hemolysis). Also shows anti-proliferative effects on four cancer cell lines; when associated with K-30; K-33 and K-37.&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;; MUTAGEN 30; /note=&quot;H-&gt;K: Shows important increase in antimicrobial activity against E.coli (MIC=8 uM) and slight increase against S.aureus (MIC=4 uM) and C.albicans (MIC=4 uM), as well as slight increase in hemolytic activity (32 uM for 100% hemolysis). Also shows anti-proliferative effects on four cancer cell lines; when associated with K-26; K-33 and K-37.&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;; MUTAGEN 33; /note=&quot;S-&gt;K: Shows important increase in antimicrobial activity against E.coli (MIC=8 uM) and slight increase against S.aureus (MIC=4 uM) and C.albicans (MIC=4 uM), as well as slight increase in hemolytic activity (32 uM for 100% hemolysis). Also shows anti-proliferative effects on four cancer cell lines; when associated with K-26; K-30 and K-37.&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;; MUTAGEN 37; /note=&quot;S-&gt;K: Shows important increase in antimicrobial activity against E.coli (MIC=8 uM) and slight increase against S.aureus (MIC=4 uM) and C.albicans (MIC=4 uM), as well as slight increase in hemolytic activity (32 uM for 100% hemolysis). Also shows anti-proliferative effects on four cancer cell lines; when associated with K-26; K-30 and K-33.&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Secreted {ECO:0000269|PubMed:25332684}. Target cell membrane {ECO:0000250}. Note=Forms a helical membrane channel in the prey. {ECO:0000250}.</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">MOD_RES 40; /note=&quot;Lysine amide&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;</td><td style = \"font-style: italic; text-align: left;\">PEPTIDE 23..40; /note=&quot;Antimicrobial peptide AcrAP1&quot;; /id=&quot;PRO_0000432592&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">PROPEP 46..74; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;; /id=&quot;PRO_0000432593&quot;</td><td style = \"text-align: left;\">SIGNAL 1..22; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">25332684</td><td style = \"text-align: left;\">2015-04-01</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">44</td><td style = \"text-align: right;\">43</td><td style = \"text-align: left;\">A0A0A1I6N9</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">NDB4T_ANDCR</td><td style = \"text-align: left;\">Antimicrobial peptide AcrAP2</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Androctonus crassicauda (Arabian fat-tailed scorpion)</td><td style = \"text-align: right;\">74</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">122909</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Androctonus (genus), Buthidae (family), Buthoidea (superfamily), Buthida (parvorder), Scorpiones (order), Arachnida (class), Chelicerata (subphylum), Arthropoda (phylum), Panarthropoda (no rank), Ecdysozoa (no rank), Protostomia (no rank), Bilateria (no rank), Eumetazoa (no rank), Metazoa (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">6857 (genus), 6856 (family), 70336 (superfamily), 259437 (parvorder), 6855 (order), 6854 (class), 6843 (subphylum), 6656 (phylum), 88770 (no rank), 1206794 (no rank), 33317 (no rank), 33213 (no rank), 6072 (no rank), 33208 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">8655</td><td style = \"font-style: italic; text-align: left;\">MASS SPECTROMETRY: Mass=1937.39; Method=Electrospray; Evidence={ECO:0000269|PubMed:25332684};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MEIKYLLTVFLVLLIVSDHCQAFLFSLIPNAISGLLSAFKGRRKRNLDGQIDRFRNFRKRDAELEELLSKLPIY</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">FUNCTION: Has antimicrobial activity against the Gram-positive bacteria S.aureus (MIC=8 uM) and the yeast C.albicans (MIC=16 uM). Causes hemolysis on horse erythrocytes (64 uM for 100% hemolysis). Minimum bactericidal concentrations have also been tested against S.aureus and is four-fold higher (MBC=32 uM). {ECO:0000269|PubMed:25332684}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Amidation;Antibiotic;Antimicrobial;Cleavage on pair of basic residues;Fungicide;Membrane;Secreted;Signal;Target cell membrane;Target membrane</td><td style = \"text-align: left;\">KW-0027; KW-0044; KW-0929; KW-0165; KW-0295; KW-0472; KW-0964; KW-0732; KW-1052; KW-1053</td><td style = \"font-style: italic; text-align: left;\">MISCELLANEOUS: Does not show antimicrobial activity against the Gram-negative bacteria E.coli (MIC&gt;250 uM). Does not show effect on four different human cancer cell lines. {ECO:0000269|PubMed:25332684}.</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00052A3EB8</td><td style = \"text-align: left;\">FUNCTION (1); MASS SPECTROMETRY (1); MISCELLANEOUS (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Modified residue (1); Mutagenesis (4); Peptide (1); Propeptide (1); Signal (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">TISSUE SPECIFICITY: Expressed by the venom gland. {ECO:0000305}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">defense response to bacterium [GO:0042742]; defense response to fungus [GO:0050832]; killing of cells of another organism [GO:0031640]</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; membrane [GO:0016020]</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">GO:0005576; GO:0016020; GO:0031640; GO:0042742; GO:0050832</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; membrane [GO:0016020]; defense response to bacterium [GO:0042742]; defense response to fungus [GO:0050832]; killing of cells of another organism [GO:0031640]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">MUTAGEN 26; /note=&quot;S-&gt;K: Shows important increase in antimicrobial activity against E.coli (MIC=8 uM) and slight increase against S.aureus (MIC=4 uM) and C.albicans (MIC=4 uM), as well as slight increase in hemolytic activity (32 uM for 100% hemolysis). Also shows anti-proliferative effects on four cancer cell lines; when associated with K-30; K-33 and K-37.&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;; MUTAGEN 30; /note=&quot;N-&gt;K: Shows important increase in antimicrobial activity against E.coli (MIC=8 uM) and slight increase against S.aureus (MIC=4 uM) and C.albicans (MIC=4 uM), as well as slight increase in hemolytic activity (32 uM for 100% hemolysis). Also shows anti-proliferative effects on four cancer cell lines; when associated with K-26; K-33 and K-37.&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;; MUTAGEN 33; /note=&quot;S-&gt;K: Shows important increase in antimicrobial activity against E.coli (MIC=8 uM) and slight increase against S.aureus (MIC=4 uM) and C.albicans (MIC=4 uM), as well as slight increase in hemolytic activity (32 uM for 100% hemolysis). Also shows anti-proliferative effects on four cancer cell lines; when associated with K-26; K-30 and K-37.&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;; MUTAGEN 37; /note=&quot;S-&gt;K: Shows important increase in antimicrobial activity against E.coli (MIC=8 uM) and slight increase against S.aureus (MIC=4 uM) and C.albicans (MIC=4 uM), as well as slight increase in hemolytic activity (32 uM for 100% hemolysis). Also shows anti-proliferative effects on four cancer cell lines; when associated with K-26; K-30 and K-33.&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Secreted {ECO:0000269|PubMed:25332684}. Target cell membrane {ECO:0000250}. Note=Forms a helical membrane channel in the prey. {ECO:0000250}.</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">MOD_RES 40; /note=&quot;Lysine amide&quot;; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;</td><td style = \"font-style: italic; text-align: left;\">PEPTIDE 23..40; /note=&quot;Antimicrobial peptide AcrAP2&quot;; /id=&quot;PRO_0000432594&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">PROPEP 46..74; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;; /id=&quot;PRO_0000432595&quot;</td><td style = \"font-style: italic; text-align: left;\">SIGNAL 1..22; /evidence=&quot;ECO:0000269|PubMed:25332684&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">25332684</td><td style = \"text-align: left;\">2015-04-01</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">45</td><td style = \"text-align: right;\">44</td><td style = \"text-align: left;\">A0A0A7EPL0</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">PIAL1_ARATH</td><td style = \"text-align: left;\">E4 SUMO-protein ligase PIAL1, EC 2.3.2.- (Protein EMBRYO DEFECTIVE 3001) (Protein INHIBITOR OF ACTIVATED STAT-LIKE 1)</td><td style = \"text-align: left;\">PIAL1 EMB3001 At1g08910 F7G19.21</td><td style = \"text-align: left;\">Arabidopsis thaliana (Mouse-ear cress)</td><td style = \"text-align: right;\">847</td><td style = \"font-style: italic; text-align: left;\">At1g08910</td><td style = \"font-style: italic; text-align: left;\">F7G19.21</td><td style = \"text-align: left;\">PIAL1</td><td style = \"font-style: italic; text-align: left;\">EMB3001</td><td style = \"text-align: right;\">3702</td><td style = \"font-style: italic; text-align: left;\">UP000006548: Chromosome 1</td><td style = \"text-align: left;\">Arabidopsis (genus), Camelineae (tribe), Brassicaceae (family), Brassicales (order), malvids (no rank), rosids (no rank), Pentapetalae (no rank), Gunneridae (no rank), eudicotyledons (no rank), Mesangiospermae (no rank), Magnoliopsida (class), Spermatophyta (no rank), Euphyllophyta (no rank), Tracheophyta (no rank), Embryophyta (no rank), Streptophytina (subphylum), Streptophyta (phylum), Viridiplantae (kingdom), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">3701 (genus), 980083 (tribe), 3700 (family), 3699 (order), 91836 (no rank), 71275 (no rank), 1437201 (no rank), 91827 (no rank), 71240 (no rank), 1437183 (no rank), 3398 (class), 58024 (no rank), 78536 (no rank), 58023 (no rank), 3193 (no rank), 131221 (subphylum), 35493 (phylum), 33090 (kingdom), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  Event=Alternative splicing; Named isoforms=2; Name=1; IsoId=A0A0A7EPL0-1; Sequence=Displayed; Name=2; IsoId=A0A0A7EPL0-2; Sequence=VSP_057991;</td><td style = \"font-style: italic; text-align: left;\">VAR_SEQ 318..342; /note=&quot;PSWRCPHCNQSVCYTDIRVDQKLRK -&gt; HHGAARI (in isoform 2)&quot;; /id=&quot;VSP_057991&quot;</td><td style = \"font-style: italic; text-align: left;\">Sequence=AAB70421.1; Type=Erroneous gene model prediction; Evidence={ECO:0000305};</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">93475</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MVIPATSRFGFRAEFNTKEFQASCISLANEIDAAIGRNEVPGNIQELALILNNVCRRKCDDYQTRAVVMALMISVKSACQLGWFPERETQELLAIIDLMWNGFSCPENVTSCVNSPVTLISQVIERFYPCVKLGHILVSFEAKPESKMMMKDFHISKKMPHSPKQKVGLFVVRTEDISRSNCIVHPQGVSFLLNGKGIDKRVNISMESGPQLPTNVTALLNLGANLLQAIGCFGGSYLIAIAFMDVIPLPNKPLLKDYVHPEVVGSNSDCDIIEGPSRISLSCPISRTRIKLPVKGHVCKHLQCFDFWNYVNMNTRRPSWRCPHCNQSVCYTDIRVDQKLRKILEEVGRNAADVVISADGTWMVETENDEDVELVPETTHDHGDPNSFINLGPTVKNPARDENEMETSTQVEEHNPCLSEIQGPSNDTHRPASDYTMLNQSHTSTNTLPQLPRTLNAFDGQQFVNLPQVINTRDSPASQALPMTFSPTPSPQDILATNAANFGTSMPAAQSSQFQGSHVTSLGNCEGRTSDLMARWNHIYGRVQTQFPPAPLSHHHYSMQNQSPSPAQQRPVPSYIAHPQTFHVNYGENADQRWMPSSIAHPQTLPVNYGGNTNQRPIPSSIAHPQTLPVNYRGNTDHRSTPYSITHLQTLLNYGGNADQRPMPSSITNLQTLPATYGGYAHQRPMSSSITHPRTSPVNYGGTPDQRPMPSSITHPQTLPVSYGGTTDQILNPGGAMGQFSSREFMNLTPANTENWRPQSRMRGSVAPGTGYDHMIIHPTRPVHPQAQTPPAPLSTSYDGADEIQAFIGHPSYPVSNNETQAGTSSLPVAEGLGYSGSFWSMPPETW</td><td style = \"font-style: italic; text-align: left;\">SEQUENCE CAUTION:  Sequence=AAB70421.1; Type=Erroneous gene model prediction; Evidence={ECO:0000305};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">BINDING 299; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00452&quot;; BINDING 301; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00452&quot;; BINDING 322; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00452&quot;; BINDING 325; /ligand=&quot;Zn(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:29105&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00452&quot;</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.3.2.-</td><td style = \"text-align: left;\">FUNCTION: E4-type SUMO ligase that promotes SUMO chain formation in a SCE1-dependent manner and thus contributes to a pathway for proteolytic removal of sumoylation substrates. Involved in stress responses (e.g. osmotic, salt and abscisic acid ABA) and sulfur metabolism. {ECO:0000269|PubMed:25415977}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PATHWAY: Protein modification; protein sumoylation. {ECO:0000269|PubMed:25415977}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Alternative splicing;Ligase;Metal-binding;Nucleus;Reference proteome;Repeat;Transferase;Ubl conjugation pathway;Zinc;Zinc-finger</td><td style = \"text-align: left;\">KW-0025; KW-0436; KW-0479; KW-0539; KW-1185; KW-0677; KW-0808; KW-0833; KW-0862; KW-0863</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at transcript level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000152B3D0</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS (2); DISRUPTION PHENOTYPE (1); FUNCTION (1); PATHWAY (1); SEQUENCE CAUTION (1); SIMILARITY (1); SUBCELLULAR LOCATION (1); TISSUE SPECIFICITY (1)</td><td style = \"text-align: left;\">Alternative sequence (1); Binding site (4); Chain (1); Region (1); Repeat (7); Zinc finger (1)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">TISSUE SPECIFICITY: Expressed in leaves, stems and flowers, and, at low levels, in siliques and old leaves. {ECO:0000269|PubMed:25415977}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">positive regulation of sulfur metabolic process [GO:0051176]; protein sumoylation [GO:0016925]; response to abscisic acid [GO:0009737]; response to osmotic stress [GO:0006970]; response to salt stress [GO:0009651]</td><td style = \"text-align: left;\">nucleus [GO:0005634]</td><td style = \"text-align: left;\">ligase activity [GO:0016874]; SUMO transferase activity [GO:0019789]; zinc ion binding [GO:0008270]</td><td style = \"text-align: left;\">GO:0005634; GO:0006970; GO:0008270; GO:0009651; GO:0009737; GO:0016874; GO:0016925; GO:0019789; GO:0051176</td><td style = \"text-align: left;\">nucleus [GO:0005634]; ligase activity [GO:0016874]; SUMO transferase activity [GO:0019789]; zinc ion binding [GO:0008270]; positive regulation of sulfur metabolic process [GO:0051176]; protein sumoylation [GO:0016925]; response to abscisic acid [GO:0009737]; response to osmotic stress [GO:0006970]; response to salt stress [GO:0009651]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">DISRUPTION PHENOTYPE: No obvious growth difference under standard greenhouse conditions. Altered sulfur metabolism. Reduced growth in high osmotic pressure (mannitol) and in response to abscisic acid (ABA), but enhanced growth and fitness in high salt (NaCl) condition. Abnormal steady state levels of SUMO conjugates in various conditions. {ECO:0000269|PubMed:25415977}.</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Nucleus {ECO:0000305}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 1..847; /note=&quot;E4 SUMO-protein ligase PIAL1&quot;; /id=&quot;PRO_0000434951&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">25415977; 11130712; 27862469</td><td style = \"text-align: left;\">2015-12-09</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">46</td><td style = \"text-align: right;\">45</td><td style = \"text-align: left;\">A0A0A7EQR3</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">GCE_CERUI</td><td style = \"text-align: left;\">4-O-methyl-glucuronoyl methylesterase, EC 3.1.1.117 (Glucuronoyl esterase, GE)</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Cerrena unicolor (Canker rot fungus) (Daedalea unicolor)</td><td style = \"text-align: right;\">474</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: right;\">90312</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Cerrena (genus), Cerrenaceae (family), Polyporales (order), Agaricomycetes incertae sedis (no rank), Agaricomycetes (class), Agaricomycotina (subphylum), Basidiomycota (phylum), Dikarya (subkingdom), Fungi (kingdom), Opisthokonta (no rank), Eukaryota (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">90311 (genus), 2028206 (family), 5303 (order), 355688 (no rank), 155619 (class), 5302 (subphylum), 5204 (phylum), 451864 (subkingdom), 4751 (kingdom), 33154 (no rank), 2759 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">49910</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MFKPSFVALALVSYATAQASAPQWGQCGGIGWTGPTACPSGWACQQLNAYYSQCLQGAAPAPARTTAAPPPPPATTAAPPPPTTSAPTGSSPVAGACGAIASTVPNYNNAKLPDPFTFANGTALRTKADWSCRRAEISALIQNYEAGTLPPKPPVVTASFSKSGNTGTLAITAGLSNSQTIKFSPTISYPSGTPPANGWPLIIAYEGGSIPIPAGVATLTYSNSDMAQQNSASSRGQGLFYQLYGSTHSASAMTAWVWGVSRIIDALEMTPTAQINTQRIGVTGCSRDGKGALMAGAFEERIALTIPQESGSGGDACWRLSKYEIDNGNQVQDAVEIVGENVWFSTNFNNYVQKLPTVPEDHHLLAAMVAPRAMISFENTDYLWLSPMSSFGCMTAAHTVWQGLGIADSHGFAQVGGHAHCAWPSSLTPQLNAFINRFLLDQSATTNVFTTNNQFGKVQWNAANWITWTTPTLT</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">ACT_SITE 286; /note=&quot;Nucleophile&quot;; /evidence=&quot;ECO:0000250|UniProtKB:G2QJR6&quot;; ACT_SITE 420; /note=&quot;Proton donor/acceptor&quot;; /evidence=&quot;ECO:0000250|UniProtKB:G2QJR6&quot;</td><td style = \"text-align: left;\">BINDING 290; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:G2QJR6&quot;; BINDING 332; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:G2QJR6&quot;; BINDING 340; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:G2QJR6&quot;; BINDING 384; /ligand=&quot;substrate&quot;; /evidence=&quot;ECO:0000250|UniProtKB:G2QJR6&quot;</td><td style = \"font-style: italic; text-align: left;\">CATALYTIC ACTIVITY: Reaction=a 4-O-methyl-alpha-D-glucuronosyl ester derivative + H2O = 4-O-methyl-alpha-D-glucuronate derivative + an alcohol + H(+); Xref=Rhea:RHEA:67452, ChEBI:CHEBI:15377, ChEBI:CHEBI:15378, ChEBI:CHEBI:30879, ChEBI:CHEBI:171667, ChEBI:CHEBI:171668; EC=3.1.1.117; Evidence={ECO:0000269|PubMed:25425346, ECO:0000269|PubMed:26712478}; PhysiologicalDirection=left-to-right; Xref=Rhea:RHEA:67453; Evidence={ECO:0000305|PubMed:25425346, ECO:0000305|PubMed:26712478};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">3.1.1.117</td><td style = \"text-align: left;\">FUNCTION: Glucuronoyl esterase which may play a significant role in biomass degradation, as it is considered to disconnect hemicellulose from lignin through the hydrolysis of the ester bond between 4-O-methyl-D-glucuronic acid residues of glucuronoxylans and aromatic alcohols of lignin (PubMed:25425346, PubMed:26712478). {ECO:0000269|PubMed:25425346, ECO:0000269|PubMed:26712478}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES:  Kinetic parameters: KM=4.6 mM for benzyl (methyl 4-O-methyl-alpha-D-glucopyranoside) uronate {ECO:0000269|PubMed:25425346}; KM=80 mM for benzyl (methyl alpha-D-glucopyranoside) uronate {ECO:0000269|PubMed:25425346}; KM=55 mM for phenylpropyl (methyl alpha-D-glucopyranoside) uronate {ECO:0000269|PubMed:25425346}; KM=8.9 mM for phenyl (methyl alpha-D-glucopyranoside) uronate {ECO:0000269|PubMed:25425346}; KM=4.6 mM for threo-l-[4-(benzyloxy)-3-methoxyphenyl]-3-hydroxy-2- (2-methoxyphenoxy) propyl (methyl 4-O-methyl-alpha-D-glucopyranosid) uronate {ECO:0000269|PubMed:26712478};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">RHEA:67452</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;Disulfide bond;Glycoprotein;Hydrolase;Lignin degradation;Secreted;Serine esterase;Signal</td><td style = \"text-align: left;\">KW-0002; KW-1015; KW-0325; KW-0378; KW-0439; KW-0964; KW-0719; KW-0732</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI0005412FED</td><td style = \"text-align: left;\">BIOPHYSICOCHEMICAL PROPERTIES (1); CATALYTIC ACTIVITY (1); FUNCTION (1); PTM (1); SIMILARITY (1); SUBCELLULAR LOCATION (1)</td><td style = \"text-align: left;\">Active site (2); Beta strand (11); Binding site (4); Chain (1); Compositional bias (1); Disulfide bond (2); Domain (1); Glycosylation (1); Helix (18); Motif (1); Region (1); Signal (1); Turn (3)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">carbohydrate metabolic process [GO:0005975]; lignin catabolic process [GO:0046274]</td><td style = \"text-align: left;\">extracellular region [GO:0005576]</td><td style = \"text-align: left;\">carboxylic ester hydrolase activity [GO:0052689]; cellulose binding [GO:0030248]</td><td style = \"text-align: left;\">GO:0005576; GO:0005975; GO:0030248; GO:0046274; GO:0052689</td><td style = \"text-align: left;\">extracellular region [GO:0005576]; carboxylic ester hydrolase activity [GO:0052689]; cellulose binding [GO:0030248]; carbohydrate metabolic process [GO:0005975]; lignin catabolic process [GO:0046274]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">SUBCELLULAR LOCATION: Secreted {ECO:0000269|PubMed:25425346}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 18..474; /note=&quot;4-O-methyl-glucuronoyl methylesterase&quot;; /id=&quot;PRO_5002038279&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">DISULFID 285..421; /evidence=&quot;ECO:0000250|UniProtKB:G2QJR6&quot;; DISULFID 317..393; /evidence=&quot;ECO:0000250|UniProtKB:G2QJR6&quot;</td><td style = \"font-style: italic; text-align: left;\">CARBOHYD 120; /note=&quot;N-linked (GlcNAc...) asparagine&quot;; /evidence=&quot;ECO:0000255|PROSITE-ProRule:PRU00498&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PTM: N-glycosylated (PubMed:25425346). {ECO:0000269|PubMed:25425346}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">SIGNAL 1..17; /evidence=&quot;ECO:0000255&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">X-ray crystallography (6)</td><td style = \"font-style: italic; text-align: left;\">STRAND 154..163; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 166..174; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 180..183; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 186..188; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 199..205; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 216..221; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 275..285; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 303..309; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 342..344; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 372..379; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; STRAND 410..415; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;</td><td style = \"font-style: italic; text-align: left;\">HELIX 127..144; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 223..226; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 232..234; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 239..244; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 252..268; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 271..273; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 287..298; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 318..326; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 334..340; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 346..350; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 355..357; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 362..367; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 383..385; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 387..403; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 407..409; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 425..427; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 428..438; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; HELIX 462..464; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;</td><td style = \"font-style: italic; text-align: left;\">TURN 312..315; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; TURN 351..353; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;; TURN 368..371; /evidence=&quot;ECO:0007829|PDB:6RU1&quot;</td><td style = \"text-align: left;\">25425346; 26712478</td><td style = \"text-align: left;\">2016-04-13</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">47</td><td style = \"text-align: right;\">46</td><td style = \"text-align: left;\">A0A0A7GEY4</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">GGPPS_GEOAI</td><td style = \"text-align: left;\">Geranylgeranyl pyrophosphate synthase, GGPP synthase, GGPPSase, EC 2.5.1.- (Dimethylallyltranstransferase, EC 2.5.1.1) (Farnesyl diphosphate synthase) (Farnesyltranstransferase, EC 2.5.1.29) (Geranylgeranyl diphosphate synthase) (Geranyltranstransferase, EC 2.5.1.10)</td><td style = \"text-align: left;\">GACE_1337</td><td style = \"text-align: left;\">Geoglobus acetivorans</td><td style = \"text-align: right;\">319</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">GACE_1337</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">565033</td><td style = \"text-align: left;\">UP000030624: Chromosome</td><td style = \"text-align: left;\">Geoglobus (genus), Archaeoglobaceae (family), Archaeoglobales (order), Archaeoglobi (class), Euryarchaeota (phylum), Archaea (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">190818 (genus), 2232 (family), 2231 (order), 183980 (class), 28890 (phylum), 2157 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">35918</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MISEIIKDRAKLVNEKIEELLKEQEPEGLYRAARHYLKAGGKRLRPVITLLSAEALGEDYRKAIHAAIAIETVHNFTLVHDDIMDEDEMRRGVKTVHTLFGIPTAILAGDTLYAEAFEILSMSDAPPENIVRAVSKLARVCVEICEGQFMDMSFEERDSVGESEYLEMVRKKTGVLIGISASIPAVLFGKDESVEKALWNYGIYSGIGFQIHDDLLDISGKGKIGKDWGSDILEGKKTLIVIKAFEEGIELETFGKGRASEEELERDIKKLFDCGAVDYARERAREYIEMAKKNLEVIDESPSRNYLVELADYLIERDH</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">BINDING 42; /ligand=&quot;isopentenyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:128769&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 45; /ligand=&quot;isopentenyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:128769&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 74; /ligand=&quot;isopentenyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:128769&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 81; /ligand=&quot;Mg(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:18420&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 81; /ligand=&quot;Mg(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:18420&quot;; /ligand_label=&quot;2&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 85; /ligand=&quot;Mg(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:18420&quot;; /ligand_label=&quot;1&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 85; /ligand=&quot;Mg(2+)&quot;; /ligand_id=&quot;ChEBI:CHEBI:18420&quot;; /ligand_label=&quot;2&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 90; /ligand=&quot;dimethylallyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:57623&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 91; /ligand=&quot;isopentenyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:128769&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 172; /ligand=&quot;dimethylallyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:57623&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 173; /ligand=&quot;dimethylallyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:57623&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 210; /ligand=&quot;dimethylallyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:57623&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 226; /ligand=&quot;dimethylallyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:57623&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;; BINDING 236; /ligand=&quot;dimethylallyl diphosphate&quot;; /ligand_id=&quot;ChEBI:CHEBI:57623&quot;; /evidence=&quot;ECO:0000250|UniProtKB:Q12051&quot;</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=dimethylallyl diphosphate + isopentenyl diphosphate = (2E)-geranyl diphosphate + diphosphate; Xref=Rhea:RHEA:22408, ChEBI:CHEBI:33019, ChEBI:CHEBI:57623, ChEBI:CHEBI:58057, ChEBI:CHEBI:128769; EC=2.5.1.1; Evidence={ECO:0000269|PubMed:30062607}; CATALYTIC ACTIVITY: Reaction=(2E)-geranyl diphosphate + isopentenyl diphosphate = (2E,6E)-farnesyl diphosphate + diphosphate; Xref=Rhea:RHEA:19361, ChEBI:CHEBI:33019, ChEBI:CHEBI:58057, ChEBI:CHEBI:128769, ChEBI:CHEBI:175763; EC=2.5.1.10; Evidence={ECO:0000269|PubMed:30062607}; CATALYTIC ACTIVITY: Reaction=(2E,6E)-farnesyl diphosphate + isopentenyl diphosphate = (2E,6E,10E)-geranylgeranyl diphosphate + diphosphate; Xref=Rhea:RHEA:17653, ChEBI:CHEBI:33019, ChEBI:CHEBI:58756, ChEBI:CHEBI:128769, ChEBI:CHEBI:175763; EC=2.5.1.29; Evidence={ECO:0000269|PubMed:30062607};</td><td style = \"text-align: left;\">COFACTOR: Name=Mg(2+); Xref=ChEBI:CHEBI:18420; Evidence={ECO:0000250|UniProtKB:Q12051}; Note=Binds 2 Mg(2+) ions per subunit. {ECO:0000250|UniProtKB:Q12051};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.5.1.-; 2.5.1.1; 2.5.1.10; 2.5.1.29</td><td style = \"text-align: left;\">FUNCTION: Catalyzes the addition of 3 molecules of isopentenyl diphosphate (IPP) onto dimethylallyl diphosphate (DMAPP) to form geranylgeranyl pyrophosphate (GGPP). Catalyzes the synthesis of geranylgeranyl pyrophosphate as a major product and of farnesyl pyrophosphate in smaller amounts. {ECO:0000269|PubMed:30062607}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">PATHWAY: Isoprenoid biosynthesis; geranyl diphosphate biosynthesis; geranyl diphosphate from dimethylallyl diphosphate and isopentenyl diphosphate: step 1/1. {ECO:0000305}.; PATHWAY: Isoprenoid biosynthesis; farnesyl diphosphate biosynthesis; farnesyl diphosphate from geranyl diphosphate and isopentenyl diphosphate: step 1/1. {ECO:0000305}.; PATHWAY: Isoprenoid biosynthesis; geranylgeranyl diphosphate biosynthesis; geranylgeranyl diphosphate from farnesyl diphosphate and isopentenyl diphosphate: step 1/1. {ECO:0000305}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:22408 RHEA:19361 RHEA:17653</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;Isoprene biosynthesis;Magnesium;Metal-binding;Transferase</td><td style = \"text-align: left;\">KW-0002; KW-0414; KW-0460; KW-0479; KW-0808</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI000540F586</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY (3); COFACTOR (1); FUNCTION (1); PATHWAY (3); SIMILARITY (1); SUBUNIT (1)</td><td style = \"text-align: left;\">Beta strand (1); Binding site (14); Chain (1); Helix (16)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">SUBUNIT: Homodimer. {ECO:0000269|PubMed:30062607}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">farnesyl diphosphate biosynthetic process [GO:0045337]; geranyl diphosphate biosynthetic process [GO:0033384]; geranylgeranyl diphosphate biosynthetic process [GO:0033386]</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">dimethylallyltranstransferase activity [GO:0004161]; farnesyltranstransferase activity [GO:0004311]; geranyltranstransferase activity [GO:0004337]; metal ion binding [GO:0046872]</td><td style = \"text-align: left;\">GO:0004161; GO:0004311; GO:0004337; GO:0033384; GO:0033386; GO:0045337; GO:0046872</td><td style = \"text-align: left;\">dimethylallyltranstransferase activity [GO:0004161]; farnesyltranstransferase activity [GO:0004311]; geranyltranstransferase activity [GO:0004337]; metal ion binding [GO:0046872]; farnesyl diphosphate biosynthetic process [GO:0045337]; geranyl diphosphate biosynthetic process [GO:0033384]; geranylgeranyl diphosphate biosynthetic process [GO:0033386]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 1..319; /note=&quot;Geranylgeranyl pyrophosphate synthase&quot;; /id=&quot;PRO_0000448754&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">X-ray crystallography (1)</td><td style = \"font-style: italic; text-align: left;\">STRAND 88..90; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;</td><td style = \"font-style: italic; text-align: left;\">HELIX 2..20; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 27..38; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 44..55; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 60..63; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 64..85; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 96..100; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 102..120; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 127..156; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 162..172; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 174..187; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 192..216; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 230..233; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 239..246; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 265..273; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 276..295; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;; HELIX 302..316; /evidence=&quot;ECO:0007829|PDB:5JFQ&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">25416759; 30062607</td><td style = \"text-align: left;\">2019-12-11</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">48</td><td style = \"text-align: right;\">47</td><td style = \"text-align: left;\">A0A0A7HFE1</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">CAS10_STRTR</td><td style = \"text-align: left;\">CRISPR system single-strand-specific deoxyribonuclease Cas10/Csm1 (subtype III-A), ssDNase Cas10, EC 3.1.-.- (Cyclic oligoadenylate synthase, EC 2.7.7.-) (StCas10)</td><td style = \"text-align: left;\">cas10 csm1</td><td style = \"text-align: left;\">Streptococcus thermophilus</td><td style = \"text-align: right;\">758</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">cas10</td><td style = \"font-style: italic; text-align: left;\">csm1</td><td style = \"text-align: right;\">1308</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Streptococcus (genus), Streptococcaceae (family), Lactobacillales (order), Bacilli (class), Firmicutes (phylum), Terrabacteria group (no rank), Bacteria (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">1301 (genus), 1300 (family), 186826 (order), 91061 (class), 1239 (phylum), 1783272 (no rank), 2 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">86822</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MKKEKIDLFYGALLHDIGKVIQRATGERKKHALVGADWFDEIADNQVISDQIRYHMANYQSDKLGNDHLAYITYIADNIASGVDRRQSNEESDEDASAKIWDTYTNQADIFNVFGAQTDKRYFKPTVLNLKSKPNFASATYEPFSKGDYAAIATRIKNELAEFEFNQAQIDSLLNLFEAILSFVPSSTNSKEIADISLAEHSRLTAAFALAIYDYLEDKGRHNYKEDLFTKASAFYEEEAFLLASFDLSGIQDFIYNIATSGAAKQLKARSLYLDFMSEYIADSLLDKLGLNRANLLYVGGGHAYFVLANTEKTVETLVQFEKDFNQFLLANFQTRLYVAFGWGSFAAKDIMSELNSPESYRQIYQKASRMISEKKISRYDYRTLMLLNRGGKSSERECEICHSVENLVSYHDQKVCDICRGLYQFSKEIAHDHFIITENEGLPIGPNACLKGVAFEKLSQESFSRVYVKNDYKAGTIKATHVFVGDYQCDEIHKYAALSKNEDGLGIKRLAVVRLDVDDLGAAFMAGFSRQGNGQYSTLSRSATFSRSMSLFFKVYINQFASDKKLSIIYAGGDDVFAIGSWQDIIAFTVELRQNFIKWTNGKLTLSAGIGLFADKTPISLMAHQTGELEEAAKGNEKDSISLFSSDYTFKFDRFITNVYDDKLEQIRYFFNHQDERGKNFIYKLIELLRNYESEEKMNVARLAYYLTRLEELTDKDERDKFKQFKKLFFKWYTNNESDRKEAELALLLYVYEIRKD</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CATALYTIC ACTIVITY: Reaction=6 ATP = cyclic hexaadenylate + 6 diphosphate; Xref=Rhea:RHEA:58276, ChEBI:CHEBI:30616, ChEBI:CHEBI:33019, ChEBI:CHEBI:142456; Evidence={ECO:0000269|PubMed:28663439};</td><td style = \"font-style: italic; text-align: left;\">COFACTOR: Name=a divalent metal cation; Xref=ChEBI:CHEBI:60240; Evidence={ECO:0000269|PubMed:27105119, ECO:0000269|PubMed:28663439}; Note=Degradation of ssDNA requires Mn(2+), Co(2+) or Ni(2+); Mg(2+), Ca(2+) or Zn(2+) do not activate ssDNase activity (PubMed:27105119). Formation of cOA requires Mn(2+), Co(2+) or Zn(2+), Mg(2+) is not as efficient (PubMed:28663439). {ECO:0000269|PubMed:27105119, ECO:0000269|PubMed:28663439};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">2.7.7.-; 3.1.-.-</td><td style = \"text-align: left;\">FUNCTION: CRISPR (clustered regularly interspaced short palindromic repeat) is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain spacers, sequences complementary to antecedent mobile elements, and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA). The type III-A Csm effector complex binds crRNA and acts as a crRNA-guided RNase, DNase and cyclic oligoadenylate synthase; binding of target RNA cognate to the crRNA is required for all activities. In a heterologous host this Csm effector complex restricts ssRNA phage MS2, suggesting it may target RNA viruses in vivo. {ECO:0000269|PubMed:25458845}.; FUNCTION: Csm functions as a non-specific ssDNase. Base-pairing between crRNA and target RNA to form a ternary Csm complex activates a ssDNase activity; target RNA cleavage suppresses the ssDNase, a temporal control that prevents uncontrolled DNA degradation. Viral RNA transcripts probably tether the Csm complex to the viral genome, recruiting Cas10 ssDNA activity which is able to degrade DNA in the transcription bubble, spatially controlling the DNase activity. {ECO:0000269|PubMed:27105119}.; FUNCTION: This subunit has a weak ssDNase activity that is dramatically activated by the ternary Csm effector complex (the crRNA, Cas proteins and a cognate target ssRNA). Target RNA and ssDNA are cleaved simultaneously, although RNase activity (of Csm3) is much faster. RNA cleavage by Csm3 is not required for ssDNase activity as Csm complex with inactive Csm3 still has ssDNase activity; however as the cleaved target RNA products dissociate away ssDNase activity decreases. Self-recognition, with subsequent repression of the ssDNase activity, occurs when the 5&apos; handle of the crRNA bases pairs with the 3&apos; flanking sequence of the target RNA (which would occur if the CRISPR locus were transcribed as an anti-pre-crRNA). This protein has low activity on dsDNA which is not stimulated by the Csm complex. {ECO:0000269|PubMed:27105119}.; FUNCTION: This subunit is a single-strand-specific deoxyribonuclease (ssDNase) which digests both linear and circular ssDNA; it has both exo- and endonuclease activity. {ECO:0000250|UniProtKB:B6YWB8}.; FUNCTION: When associated with the ternary Csm effector complex (the crRNA, Cas proteins and a cognate target ssRNA) synthesizes cyclic oligoadenylates (cOA) from ATP, producing cyclic triadenylate (cA3) up to cyclic hexaadenylate (cA6), which is the active cOA. The enzyme is also able to cyclize pppA3 up to pppA6. cOAs are second messengers that induce an antiviral state important for defense against invading nucleic acids. Synthesis of cOA can occur with AMP plus ATP, 2&apos;dATP or 3&apos;dATP (but no other nucleotides), and requires a free 3&apos;-OH ribose moiety. {ECO:0000269|PubMed:28663439}.</td><td style = \"font-style: italic; text-align: left;\">ACTIVITY REGULATION: ssDNase activity is activated by target RNA binding to the Csm-crRNA complex and is inhibited by EDTA. {ECO:0000269|PubMed:27105119}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">RHEA:58276</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;Antiviral defense;ATP-binding;Endonuclease;Exonuclease;Hydrolase;Nuclease;Nucleotide-binding;RNA-binding;Transferase</td><td style = \"text-align: left;\">KW-0002; KW-0051; KW-0067; KW-0255; KW-0269; KW-0378; KW-0540; KW-0547; KW-0694; KW-0808</td><td style = \"font-style: italic; text-align: left;\">MISCELLANEOUS: Encoded in a type III-A CRISPR locus. {ECO:0000269|PubMed:25458845}.</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI0002177493</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); CATALYTIC ACTIVITY (1); COFACTOR (1); DOMAIN (1); FUNCTION (5); MISCELLANEOUS (1); SIMILARITY (1); SUBUNIT (1)</td><td style = \"text-align: left;\">Beta strand (31); Chain (1); Domain (1); Helix (34); Mutagenesis (2); Region (1); Turn (11)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">SUBUNIT: Part of the Csm effector complex that includes at least Cas10(1), Csm2(3), Csm3(5), Csm4(1), Csm5(1) and mature crRNA (PubMed:25458845, PubMed:27105119, PubMed:28663439). The Csm complex is elongated and slightly twisted with a maximal length of 215 Angstroms and a diameter of 75-80 Angstroms (PubMed:25458845). It has been modeled to have a central protein filamant of Csm3 subunits along which the dsRNA helix of paired crRNA and target RNA binds. The filament is capped at one end by Cas10 and Csm4 and at the other end by Csm5; ssDNA is thought to bind to the N-terminal HD domain of Cas10 (Probable). Csm with a precursor crRNA does not include Csm5, while Cas6, the enzyme probably involved in pre-crRNA processing, is found associated with a subset of the Csm complex (PubMed:25458845). {ECO:0000269|PubMed:25458845, ECO:0000269|PubMed:27105119, ECO:0000269|PubMed:28663439, ECO:0000305|PubMed:25458845, ECO:0000305|PubMed:27105119}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">defense response to virus [GO:0051607]</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">ATP binding [GO:0005524]; endonuclease activity [GO:0004519]; exonuclease activity [GO:0004527]; RNA binding [GO:0003723]; transferase activity [GO:0016740]</td><td style = \"text-align: left;\">GO:0003723; GO:0004519; GO:0004527; GO:0005524; GO:0016740; GO:0051607</td><td style = \"text-align: left;\">ATP binding [GO:0005524]; endonuclease activity [GO:0004519]; exonuclease activity [GO:0004527]; RNA binding [GO:0003723]; transferase activity [GO:0016740]; defense response to virus [GO:0051607]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">MUTAGEN 16; /note=&quot;D-&gt;A: Dramatically decreased ssDNase activity. Wild-type synthesis of cOA.&quot;; /evidence=&quot;ECO:0000269|PubMed:27105119, ECO:0000269|PubMed:28663439&quot;; MUTAGEN 575..576; /note=&quot;DD-&gt;AA: Wild-type ssDNase activity. No synthesis of cOA.&quot;; /evidence=&quot;ECO:0000269|PubMed:27105119, ECO:0000269|PubMed:28663439&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 1..758; /note=&quot;CRISPR system single-strand-specific deoxyribonuclease Cas10/Csm1 (subtype III-A)&quot;; /id=&quot;PRO_0000446112&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">Electron microscopy (2)</td><td style = \"font-style: italic; text-align: left;\">STRAND 55..58; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 106..108; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; STRAND 114..116; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 130..132; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 138..141; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; STRAND 185..187; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 193..197; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 225..227; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 241..247; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 306..308; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 341..346; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 351..354; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 376..378; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 400..403; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 412..416; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 433..436; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; STRAND 439..443; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 446..448; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 452..456; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; STRAND 472..474; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 503..505; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 511..514; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 533..535; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 565..571; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 574..576; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 578..581; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 610..614; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 637..640; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; STRAND 645..648; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 694..697; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 723..725; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;</td><td style = \"font-style: italic; text-align: left;\">HELIX 6..13; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 18..24; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 31..42; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 47..53; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 73..80; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 111..113; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 153..158; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 174..177; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 181..183; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 198..218; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 261..266; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 267..289; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 293..295; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; HELIX 312..325; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 328..331; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 348..350; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 358..375; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 385..388; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 422..427; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 496..499; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 540..554; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 557..561; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 583..601; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 620..633; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 634..636; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; HELIX 654..657; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 660..663; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 665..671; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 673..675; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; HELIX 683..691; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 698..713; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 718..721; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 726..732; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 738..754; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;</td><td style = \"font-style: italic; text-align: left;\">TURN 15..17; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 178..180; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 190..192; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 235..237; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 252..256; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 382..384; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 419..421; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 428..431; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 523..525; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 562..564; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 677..680; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;</td><td style = \"text-align: left;\">25458845; 27105119; 28663439</td><td style = \"text-align: left;\">2019-01-16</td><td style = \"text-align: right;\">&ctdot;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">49</td><td style = \"text-align: right;\">48</td><td style = \"text-align: left;\">A0A0A7HIF0</td><td style = \"text-align: left;\">reviewed</td><td style = \"text-align: left;\">CSM3_STRTR</td><td style = \"text-align: left;\">CRISPR system Cms endoribonuclease Csm3, Csm3 RNase, EC 3.1.-.- (CRISPR type III A-associated RAMP protein Csm3)</td><td style = \"text-align: left;\">csm3</td><td style = \"text-align: left;\">Streptococcus thermophilus</td><td style = \"text-align: right;\">220</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">csm3</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">1308</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">Streptococcus (genus), Streptococcaceae (family), Lactobacillales (order), Bacilli (class), Firmicutes (phylum), Terrabacteria group (no rank), Bacteria (superkingdom), cellular organisms (no rank)</td><td style = \"text-align: left;\">1301 (genus), 1300 (family), 186826 (order), 91061 (class), 1239 (phylum), 1783272 (no rank), 2 (superkingdom), 131567 (no rank)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">ALTERNATIVE PRODUCTS:  </td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">24569</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">MTFAKIKFSAQIRLETGLHIGGSDAFAAIGAIDSPVIKDPITNIPIIPGSSLKGKMRTLLAKVYNEKVAEKPSDDSDILSRLFGNSKDKRFKMGRLIFRDAFLSNADELDSLGVRSYTEVKFENTIDRITAEANPRQIERAIRNSTFDFELIYEITDENENQVEEDFKVIRDGLKLLELDYLGGSGSRGYGKVAFEKLKATTVFGNYDVKTLNELLTAEV</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: right;\">1</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">COFACTOR: Name=a metal cation; Xref=ChEBI:CHEBI:25213; Evidence={ECO:0000269|PubMed:25458845}; Note=Endonucleolytic cleavage of target ssRNA by the Csm complex requires a divalent metal ion; Mg(2+) has the best activity in vitro, but Mn(2+), Ca(2+), Zn(2+), Ni(2+), and Co(2+) also support cleavage. {ECO:0000269|PubMed:25458845, ECO:0000305|PubMed:27105119};</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3.1.-.-</td><td style = \"text-align: left;\">FUNCTION: CRISPR (clustered regularly interspaced short palindromic repeat) is an adaptive immune system that provides protection against mobile genetic elements (viruses, transposable elements and conjugative plasmids). CRISPR clusters contain spacers, sequences complementary to antecedent mobile elements, and target invading nucleic acids. CRISPR clusters are transcribed and processed into CRISPR RNA (crRNA). The type III-A Csm effector complex binds crRNA and acts as a crRNA-guided RNase, DNase and cyclic oligoadenylate synthase; binding of target RNA cognate to the crRNA is required for all activities. In a heterologous host this Csm effector complex restricts ssRNA phage MS2, suggesting it may target RNA viruses in vivo. {ECO:0000269|PubMed:25458845}.; FUNCTION: Csm functions as a non-specific ssDNase. Base-pairing between crRNA and target RNA to form a ternary Csm complex activates a ssDNase activity; target RNA cleavage suppresses the ssDNase, a temporal control that prevents uncontrolled DNA degradation. Viral RNA transcripts probably tether the Csm complex to the viral genome, recruiting Cas10 ssDNA activity which is able to degrade DNA in the transcription bubble, spatially controlling the DNase activity. {ECO:0000269|PubMed:27105119}.; FUNCTION: This subunit has the target ssRNA endonuclease activity; it cleaves multiple sites in the target RNA at 6 nucleotide intervals. The number of cleavage sites in the target RNA correlates with the number of Csm3 subunits in the Csm effector complex (PubMed:25458845). In the Csm complex target RNA and ssDNA are cleaved simultaneously, although RNase activity (of Csm3) is much faster. RNA cleavage by Csm3 is not required for ssDNase activity as Csm complex with inactive Csm3 still has ssDNase activity; however as the cleaved target RNA products dissociate away ssDNase activity decreases (PubMed:27105119). {ECO:0000269|PubMed:25458845, ECO:0000269|PubMed:27105119}.</td><td style = \"font-style: italic; text-align: left;\">ACTIVITY REGULATION: Target ssRNase is inhibited by EDTA. {ECO:0000269|PubMed:25458845}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: right;\">5.0</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">3D-structure;Antiviral defense;Endonuclease;Hydrolase;Nuclease;RNA-binding</td><td style = \"text-align: left;\">KW-0002; KW-0051; KW-0255; KW-0378; KW-0540; KW-0694</td><td style = \"font-style: italic; text-align: left;\">MISCELLANEOUS: Encoded in a type III-A CRISPR locus. {ECO:0000269|PubMed:25458845}.</td><td style = \"text-align: left;\">Evidence at protein level</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">UPI00005115AF</td><td style = \"text-align: left;\">ACTIVITY REGULATION (1); COFACTOR (1); FUNCTION (3); MISCELLANEOUS (1); SIMILARITY (1); SUBUNIT (1)</td><td style = \"text-align: left;\">Beta strand (13); Chain (1); Helix (5); Mutagenesis (6); Turn (2)</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">SUBUNIT: Part of the Csm effector complex that includes at least Cas10(1), Csm2(3), Csm3(5), Csm4(1), Csm5(1) and mature crRNA (PubMed:25458845, PubMed:27105119, PubMed:28663439). The Csm complex is elongated and slightly twisted with a maximal length of 215 Angstroms and a diameter of 75-80 Angstroms (PubMed:25458845). It has been modeled to have a central protein filamant of Csm3 subunits along which the dsRNA helix of paired crRNA and target RNA binds. The filament is capped at one end by Cas10 and Csm4 and at the other end by Csm5; ssDNA is thought to bind to the N-terminal HD domain of Cas10 (Probable). Csm with a precursor crRNA does not include Csm5, while Cas6, the enzyme probably involved in pre-crRNA processing, is found associated with a subset of the Csm complex (PubMed:25458845). {ECO:0000269|PubMed:25458845, ECO:0000269|PubMed:27105119, ECO:0000269|PubMed:28663439, ECO:0000305|PubMed:25458845, ECO:0000305|PubMed:27105119}.</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">defense response to virus [GO:0051607]</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">endonuclease activity [GO:0004519]; RNA binding [GO:0003723]</td><td style = \"text-align: left;\">GO:0003723; GO:0004519; GO:0051607</td><td style = \"text-align: left;\">endonuclease activity [GO:0004519]; RNA binding [GO:0003723]; defense response to virus [GO:0051607]</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">MUTAGEN 19; /note=&quot;H-&gt;A: Wild-type degradation of target ssRNA by the Csm complex.&quot;; /evidence=&quot;ECO:0000269|PubMed:25458845&quot;; MUTAGEN 33; /note=&quot;D-&gt;A: No degradation of target ssRNA by the Csm complex, complex assembles normally and binds ssRNA. 10(3) to 10(4) decreased growth of an RNA phage in vivo. No change in ssDNase activity of the ternary Csm complex.&quot;; /evidence=&quot;ECO:0000269|PubMed:25458845, ECO:0000269|PubMed:27105119&quot;; MUTAGEN 100; /note=&quot;D-&gt;A: Nearly wild-type degradation of target ssRNA by the Csm complex, crRNA is shorter, Csm complex is altered.&quot;; /evidence=&quot;ECO:0000269|PubMed:25458845&quot;; MUTAGEN 119; /note=&quot;E-&gt;A: Wild-type degradation of target ssRNA by the Csm complex.&quot;; /evidence=&quot;ECO:0000269|PubMed:25458845&quot;; MUTAGEN 123; /note=&quot;E-&gt;A: Wild-type degradation of target ssRNA by the Csm complex.&quot;; /evidence=&quot;ECO:0000269|PubMed:25458845&quot;; MUTAGEN 139; /note=&quot;E-&gt;A: Wild-type degradation of target ssRNA by the Csm complex.&quot;; /evidence=&quot;ECO:0000269|PubMed:25458845&quot;</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">CHAIN 1..220; /note=&quot;CRISPR system Cms endoribonuclease Csm3&quot;; /id=&quot;PRO_0000446119&quot;</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: right;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">missing</td><td style = \"font-style: italic; text-align: left;\">Electron microscopy (2)</td><td style = \"font-style: italic; text-align: left;\">STRAND 5..16; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 23..25; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 27..30; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 40..42; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 70..73; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; STRAND 95..97; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 117..122; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 128..133; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 137..141; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 149..154; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 183..185; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; STRAND 187..189; /evidence=&quot;ECO:0007829|PDB:6NUD&quot;; STRAND 193..204; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;</td><td style = \"font-style: italic; text-align: left;\">HELIX 50..60; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 80..83; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 105..107; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 109..112; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; HELIX 163..177; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;</td><td style = \"font-style: italic; text-align: left;\">TURN 61..63; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;; TURN 77..79; /evidence=&quot;ECO:0007829|PDB:6NUE&quot;</td><td style = \"text-align: left;\">25458845; 27105119; 28663439</td><td style = \"text-align: left;\">2019-04-10</td><td style = \"text-align: right;\">&ctdot;</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cccccc}\n\t& Column1 & Entry & Reviewed & Entry Name & Protein names & \\\\\n\t\\hline\n\t& Int64 & String15 & String15 & String15 & String & \\\\\n\t\\hline\n\t1 & 0 & A0A024B7W1 & reviewed & POLG\\_ZIKVF & Genome polyprotein [Cleaved into: Capsid protein C (Capsid protein) (Core protein); Protein prM (Precursor membrane protein); Peptide pr (Peptide precursor); Small envelope protein M (Matrix protein); Envelope protein E; Non-structural protein 1, NS1; Non-structural protein 2A, NS2A; Serine protease subunit NS2B (Flavivirin protease NS2B regulatory subunit) (Non-structural protein 2B); Serine protease NS3, EC 3.4.21.91, EC 3.6.1.15, EC 3.6.4.13 (Flavivirin protease NS3 catalytic subunit) (Non-structural protein 3); Non-structural protein 4A, NS4A; Peptide 2k; Non-structural protein 4B, NS4B; RNA-directed RNA polymerase NS5, EC 2.1.1.56, EC 2.1.1.57, EC 2.7.7.48 (NS5) ] & $\\dots$ \\\\\n\t2 & 1 & A0A024SC78 & reviewed & CUTI1\\_HYPJR & Cutinase, EC 3.1.1.74 & $\\dots$ \\\\\n\t3 & 2 & A0A024SH76 & reviewed & GUX2\\_HYPJR & Exoglucanase 2, EC 3.2.1.91 (1,4-beta-cellobiohydrolase) (Cellobiohydrolase 6A, Cel6A) (Exocellobiohydrolase II, CBHII) (Exoglucanase II) & $\\dots$ \\\\\n\t4 & 3 & A0A026W182 & reviewed & ORCO\\_OOCBI & Odorant receptor coreceptor & $\\dots$ \\\\\n\t5 & 4 & A0A044RE18 & reviewed & BLI\\_ONCVO & Endoprotease bli, EC 3.4.21.75 (Blisterase) & $\\dots$ \\\\\n\t6 & 5 & A0A059TC02 & reviewed & CCR1\\_PETHY & Cinnamoyl-CoA reductase 1, PhCCR1, EC 1.2.1.44 (Coniferylaldehyde synthase, EC 1.2.1.-) & $\\dots$ \\\\\n\t7 & 6 & A0A060A682 & reviewed & HAP2\\_TETTH & Hapless 2 (Generative cell specific-1) & $\\dots$ \\\\\n\t8 & 7 & A0A061ACU2 & reviewed & PIEZ1\\_CAEEL & Piezo-type mechanosensitive ion channel component 1 & $\\dots$ \\\\\n\t9 & 8 & A0A061I403 & reviewed & FICD\\_CRIGR & Protein adenylyltransferase FICD, EC 2.7.7.n1 (AMPylator FICD) (De-AMPylase FICD, EC 3.1.4.-) (FIC domain-containing protein) (Huntingtin-interacting protein E) & $\\dots$ \\\\\n\t10 & 9 & A0A067CMC7 & reviewed & HTP3\\_SAPPC & Endonuclease Htp3, EC 3.1.31.- (Host targeting protein 3) (RxLR effector protein Htp3) & $\\dots$ \\\\\n\t11 & 10 & A0A067XGX8 & reviewed & AROG2\\_PETHY & Phospho-2-dehydro-3-deoxyheptonate aldolase 2, chloroplastic, EC 2.5.1.54 (3-deoxy-D-arabino-heptulosonate 7-phosphate synthase 2, DAHP synthase 2, PhDAHP2) (Phospho-2-keto-3-deoxyheptonate aldolase 2) & $\\dots$ \\\\\n\t12 & 11 & A0A067XH53 & reviewed & AROG1\\_PETHY & Phospho-2-dehydro-3-deoxyheptonate aldolase 1, chloroplastic, EC 2.5.1.54 (3-deoxy-D-arabino-heptulosonate 7-phosphate synthase 1, DAHP synthase 1, PhDAHP1) (Phospho-2-keto-3-deoxyheptonate aldolase 1) & $\\dots$ \\\\\n\t13 & 12 & A0A067XR63 & reviewed & XTH7\\_DIOKA & Xyloglucan endotransglucosylase protein 7, XET protein 7, EC 2.4.1.207 (DkXTH7) (Xyloglucan endotransglucosylase/hydrolase protein 7, XTH protein 7) & $\\dots$ \\\\\n\t14 & 13 & A0A067XRK9 & reviewed & XTH6\\_DIOKA & Xyloglucan endotransglucosylase protein 6, XET protein 6, EC 2.4.1.207 (DkXTH6) (Xyloglucan endotransglucosylase/hydrolase protein 6, XTH protein 6) & $\\dots$ \\\\\n\t15 & 14 & A0A067YMX8 & reviewed & XTH8\\_DIOKA & Xyloglucan endotransglucosylase protein 8, XET protein 8, EC 2.4.1.207 (DkXTH8) (Xyloglucan endotransglucosylase/hydrolase protein 8, XTH protein 8) & $\\dots$ \\\\\n\t16 & 15 & A0A068J840 & reviewed & UGT1\\_PANGI & UDP-glycosyltransferase 1, UGTPg1, EC 2.4.1.363 & $\\dots$ \\\\\n\t17 & 16 & A0A072UR65 & reviewed & CHT5B\\_MEDTR & Class V chitinase CHIT5b, MtCHIT5b, EC 3.2.1.14 & $\\dots$ \\\\\n\t18 & 17 & A0A075BSX9 & reviewed & HLNO\\_SHIS7 & (S)-6-hydroxynicotine oxidase, (S)-6HN oxidase, EC 1.5.3.5 (6-hydroxy-L-nicotine oxidase, 6-HLNO) (L-hydroxynicotine oxidase, LHNO) & $\\dots$ \\\\\n\t19 & 18 & A0A075F7E9 & reviewed & LERK1\\_ORYSI & G-type lectin S-receptor-like serine/threonine-protein kinase LECRK1, OsLecRK1, EC 2.7.11.1 (OsRLCK134) & $\\dots$ \\\\\n\t20 & 19 & A0A075F932 & reviewed & SYT1\\_ANSCY & Synaptotagmin-1 (Synaptotagmin I, SytI) & $\\dots$ \\\\\n\t21 & 20 & A0A075FBG7 & reviewed & ELS\\_MARVU & 9,13-epoxylabda-14-ene synthase, chloroplastic, EC 4.2.3.189 (Manoyl oxide synthase, EC 4.2.3.190) (Miltiradiene synthase, EC 4.2.3.131) & $\\dots$ \\\\\n\t22 & 21 & A0A075QQ08 & reviewed & IF4E1\\_TOBAC & Eukaryotic translation initiation factor 4E-1, NteIF4E1, eIF4E-1 (Eukaryotic translation initiation factor 4E2-T, eIF4E2-T) (eIF-4F 25 kDa subunit) (eIF-4F p26 subunit) (mRNA cap-binding protein) & $\\dots$ \\\\\n\t23 & 22 & A0A075TMP8 & reviewed & PATI\\_PENEN & Cytochrome P450 monooxygenase patI, EC 1.-.-.- (Patulin biosynthesis cluster protein I) (m-hydroxybenzyl alcohol hydroxylase) & $\\dots$ \\\\\n\t24 & 23 & A0A075TRC0 & reviewed & PATK\\_PENEN & 6-methylsalicylic acid synthase, 6MSAS, EC 2.3.1.165 (Non-reducing polyketide synthase patK) (Patulin biosynthesis cluster protein K) & $\\dots$ \\\\\n\t25 & 24 & A0A075TRK9 & reviewed & PATE\\_PENEN & Patulin synthase, EC 1.1.-.- (Dehydrogenase patE) (Patulin biosynthesis cluster protein E) & $\\dots$ \\\\\n\t26 & 25 & A0A075TRL5 & reviewed & PATH\\_PENEN & Cytochrome P450 monooxygenase patH, EC 1.-.-.- (Patulin biosynthesis cluster protein H) (m-cresol hydrolase) & $\\dots$ \\\\\n\t27 & 26 & A0A078CGE6 & reviewed & M3KE1\\_BRANA & MAP3K epsilon protein kinase 1, BnM3KE1, EC 2.7.11.1 & $\\dots$ \\\\\n\t28 & 27 & A0A087WPF7 & reviewed & AUTS2\\_MOUSE & Autism susceptibility gene 2 protein homolog & $\\dots$ \\\\\n\t29 & 28 & A0A087X1C5 & reviewed & CP2D7\\_HUMAN & Putative cytochrome P450 2D7, EC 1.14.14.1 & $\\dots$ \\\\\n\t30 & 29 & A0A088MIT0 & reviewed & BRKP2\\_PHYNA & Bradykinin-related peptides [Cleaved into: Nattererimorphin; [Val1,Thr6]-bradykinyl-Ser,Pro,Ala; Bradykinin; Des-Arg9-bradykinin; [Hyp3]-bradykinyl-Val,Asp ] & $\\dots$ \\\\\n\t$\\dots$ & $\\dots$ & $\\dots$ & $\\dots$ & $\\dots$ & $\\dots$ &  \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 17
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Entry identifiers in sample file belong to Uniprot Database"
      ],
      "metadata": {
        "id": "6WIdrfuPQJsj"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "db = \"uniprot:\"\n",
        "dbentry = string.(db, df.Entry)\n",
        "entry = join(dbentry, \"+\")\n",
        "\n",
        "kegg_conv_uniprot = KEGGAPI.conv(\"genes\", entry)\n",
        "DataFrame(\n",
        "  kegg_conv_uniprot.data,\n",
        "  kegg_conv_uniprot.colnames\n",
        ")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 498
        },
        "id": "12ovg35x-e6M",
        "outputId": "8ea37bbd-52c3-415a-d283-1d759677cedb"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m19×2 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m Target ID     \u001b[0m\u001b[1m Source ID            \u001b[0m\n",
              "     │\u001b[90m String        \u001b[0m\u001b[90m String               \u001b[0m\n",
              "─────┼─────────────────────────────────────\n",
              "   1 │ up:A0A024SC78  trr:M419DRAFT_76732\n",
              "   2 │ up:A0A024SH76  trr:M419DRAFT_122470\n",
              "   3 │ up:A0A061ACU2  cel:CELE_C10C5.1\n",
              "   4 │ up:A0A067CMC7  spar:SPRG_03573\n",
              "   5 │ up:A0A068J840  ag:AIE12479\n",
              "   6 │ up:A0A072UR65  mtr:25493984\n",
              "   7 │ up:A0A075BSX9  shz:shn_30305\n",
              "   8 │ up:A0A075FBG7  ag:AIE77094\n",
              "   9 │ up:A0A075QQ08  nta:107829212\n",
              "  10 │ up:A0A087X1C5  hsa:1564\n",
              "  11 │ up:A0A088MLT8  mmu:100505386\n",
              "  12 │ up:A0A089QRB9  mtu:Rv1180\n",
              "  13 │ up:A0A089QRB9  mtu:Rv1181\n",
              "  14 │ up:A0A096MJY4  rno:499497\n",
              "  15 │ up:A0A096P2H4  panu:101021046\n",
              "  16 │ up:A0A0A1H8I4  ag:BAP90747\n",
              "  17 │ up:A0A0A1HA03  ag:BAP90360\n",
              "  18 │ up:A0A0A7EPL0  ath:AT1G08910\n",
              "  19 │ up:A0A0A7GEY4  gac:GACE_1337"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>19×2 DataFrame</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">Target ID</th><th style = \"text-align: left;\">Source ID</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">up:A0A024SC78</td><td style = \"text-align: left;\">trr:M419DRAFT_76732</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">2</td><td style = \"text-align: left;\">up:A0A024SH76</td><td style = \"text-align: left;\">trr:M419DRAFT_122470</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">3</td><td style = \"text-align: left;\">up:A0A061ACU2</td><td style = \"text-align: left;\">cel:CELE_C10C5.1</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">4</td><td style = \"text-align: left;\">up:A0A067CMC7</td><td style = \"text-align: left;\">spar:SPRG_03573</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">5</td><td style = \"text-align: left;\">up:A0A068J840</td><td style = \"text-align: left;\">ag:AIE12479</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">6</td><td style = \"text-align: left;\">up:A0A072UR65</td><td style = \"text-align: left;\">mtr:25493984</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">7</td><td style = \"text-align: left;\">up:A0A075BSX9</td><td style = \"text-align: left;\">shz:shn_30305</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">8</td><td style = \"text-align: left;\">up:A0A075FBG7</td><td style = \"text-align: left;\">ag:AIE77094</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">9</td><td style = \"text-align: left;\">up:A0A075QQ08</td><td style = \"text-align: left;\">nta:107829212</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">10</td><td style = \"text-align: left;\">up:A0A087X1C5</td><td style = \"text-align: left;\">hsa:1564</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11</td><td style = \"text-align: left;\">up:A0A088MLT8</td><td style = \"text-align: left;\">mmu:100505386</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">12</td><td style = \"text-align: left;\">up:A0A089QRB9</td><td style = \"text-align: left;\">mtu:Rv1180</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">13</td><td style = \"text-align: left;\">up:A0A089QRB9</td><td style = \"text-align: left;\">mtu:Rv1181</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">14</td><td style = \"text-align: left;\">up:A0A096MJY4</td><td style = \"text-align: left;\">rno:499497</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">15</td><td style = \"text-align: left;\">up:A0A096P2H4</td><td style = \"text-align: left;\">panu:101021046</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">16</td><td style = \"text-align: left;\">up:A0A0A1H8I4</td><td style = \"text-align: left;\">ag:BAP90747</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">17</td><td style = \"text-align: left;\">up:A0A0A1HA03</td><td style = \"text-align: left;\">ag:BAP90360</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">18</td><td style = \"text-align: left;\">up:A0A0A7EPL0</td><td style = \"text-align: left;\">ath:AT1G08910</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">19</td><td style = \"text-align: left;\">up:A0A0A7GEY4</td><td style = \"text-align: left;\">gac:GACE_1337</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& Target ID & Source ID\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & up:A0A024SC78 & trr:M419DRAFT\\_76732 \\\\\n\t2 & up:A0A024SH76 & trr:M419DRAFT\\_122470 \\\\\n\t3 & up:A0A061ACU2 & cel:CELE\\_C10C5.1 \\\\\n\t4 & up:A0A067CMC7 & spar:SPRG\\_03573 \\\\\n\t5 & up:A0A068J840 & ag:AIE12479 \\\\\n\t6 & up:A0A072UR65 & mtr:25493984 \\\\\n\t7 & up:A0A075BSX9 & shz:shn\\_30305 \\\\\n\t8 & up:A0A075FBG7 & ag:AIE77094 \\\\\n\t9 & up:A0A075QQ08 & nta:107829212 \\\\\n\t10 & up:A0A087X1C5 & hsa:1564 \\\\\n\t11 & up:A0A088MLT8 & mmu:100505386 \\\\\n\t12 & up:A0A089QRB9 & mtu:Rv1180 \\\\\n\t13 & up:A0A089QRB9 & mtu:Rv1181 \\\\\n\t14 & up:A0A096MJY4 & rno:499497 \\\\\n\t15 & up:A0A096P2H4 & panu:101021046 \\\\\n\t16 & up:A0A0A1H8I4 & ag:BAP90747 \\\\\n\t17 & up:A0A0A1HA03 & ag:BAP90360 \\\\\n\t18 & up:A0A0A7EPL0 & ath:AT1G08910 \\\\\n\t19 & up:A0A0A7GEY4 & gac:GACE\\_1337 \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 24
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### 1.3 Convert KEGG identifiers to outside database\n",
        "\n",
        "To obtain the outside database identifier of a KEGG protein the function conv uses the DB identifier of the desire database and the KEGG gene identifier.\n",
        "\n",
        "Several identifiers from the same database can be run at once.\n",
        "\n",
        "Only those identifiers with a hit in the database are return."
      ],
      "metadata": {
        "id": "X4NA3_KXOBvx"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "@time ncbi_conv_kegg = KEGGAPI.conv(\"ncbi-proteinid\", \"mtr:25493984\")\n",
        "DataFrame(\n",
        "  ncbi_conv_kegg.data,\n",
        "  ncbi_conv_kegg.colnames\n",
        ")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 137
        },
        "id": "lNE_AQe-1Rf-",
        "outputId": "95aceb5d-5f09-4c71-defc-df74547e4a80"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.657951 seconds (351 allocations: 82.250 KiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m1×2 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m Target ID    \u001b[0m\u001b[1m Source ID                   \u001b[0m\n",
              "     │\u001b[90m String       \u001b[0m\u001b[90m String                      \u001b[0m\n",
              "─────┼───────────────────────────────────────────\n",
              "   1 │ mtr:25493984  ncbi-proteinid:XP_013458146"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>1×2 DataFrame</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">Target ID</th><th style = \"text-align: left;\">Source ID</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">mtr:25493984</td><td style = \"text-align: left;\">ncbi-proteinid:XP_013458146</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& Target ID & Source ID\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & mtr:25493984 & ncbi-proteinid:XP\\_013458146 \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 25
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "PkPJ2EYNRq9L"
      },
      "source": [
        "### 2. Gene gene information\n",
        "\n",
        "To obtain gene information at KEGG database the function \"find\" uses the string \"genes\" and the KEGG gene identifier."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "bS7fvW57RzIA",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 137
        },
        "outputId": "0b7cd7d3-5893-479c-e6a9-52ce97e5a52e"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.576392 seconds (234 allocations: 13.109 KiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m1×2 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m ID           \u001b[0m\u001b[1m Gene Name                \u001b[0m\n",
              "     │\u001b[90m String       \u001b[0m\u001b[90m String                   \u001b[0m\n",
              "─────┼────────────────────────────────────────\n",
              "   1 │ mtr:25493984  class V chitinase CHIT5b"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>1×2 DataFrame</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">ID</th><th style = \"text-align: left;\">Gene Name</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">mtr:25493984</td><td style = \"text-align: left;\">class V chitinase CHIT5b</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& ID & Gene Name\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & mtr:25493984 & class V chitinase CHIT5b \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 32
        }
      ],
      "source": [
        "@time kegg_find_genes = KEGGAPI.find(\"genes\", \"mtr:25493984\")\n",
        "DataFrame(\n",
        "  kegg_find_genes.data,\n",
        "  kegg_find_genes.colnames\n",
        ")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "CRRA4WoYUE2N"
      },
      "source": [
        "### 3. Get Enzyme sequences, nucleotide and amino acid."
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### 3.1 Get nucleotide sequence and save to fasta file\n",
        "\n",
        "With the \"kegg_get\" function user can get nucleotide sequence of one or more gene using an array with KEGG protein id and the string \"ntseq\".\n",
        "\n",
        "The output of the function can be save to file using the function FastaWriter from the FastaOI package."
      ],
      "metadata": {
        "id": "L85F_gDZArMP"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "TFmbRV3QUK-7",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9c96e25d-92d9-4c40-ec59-ba375bba764b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.624419 seconds (362 allocations: 239.680 KiB)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 1 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.040651 seconds (13.35 k allocations: 735.603 KiB, 71.29% compilation time)\n"
          ]
        }
      ],
      "source": [
        "# Nucleotide sequence\n",
        "@time kegg_ntseq = KEGGAPI.kegg_get([\"mtr:25493984\", \"shz:shn_30305\"], \"ntseq\")\n",
        "\n",
        "@time FastaWriter(\"ntseq.fasta\") do fw\n",
        "    for ch in kegg_ntseq[2]\n",
        "        write(fw, ch)\n",
        "    end\n",
        "end"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### 3.2 Get amino acid sequence and save to fasta file\n",
        "\n",
        "With the \"kegg_get\" function user can get amino acid sequence of one or more gene using an array with KEGG protein id and the string \"aaseq\".\n",
        "\n",
        "The output of the function can be save to file using the function FastaWriter from the FastaOI package."
      ],
      "metadata": {
        "id": "PLL86q8TDGXJ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "# Nucleotide sequence\n",
        "@time kegg_aaseq = KEGGAPI.kegg_get([\"mtr:25493984\", \"shz:shn_30305\"], \"aaseq\")\n",
        "\n",
        "@time FastaWriter(\"aaseq.fasta\") do fw\n",
        "    for ch in kegg_aaseq[2]\n",
        "        write(fw, ch)\n",
        "    end\n",
        "end"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ITw2upzC7gZz",
        "outputId": "47297f08-7a5a-45f8-b3df-c5386c019d6b"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.139374 seconds (228 allocations: 16.734 KiB)\n",
            "  0.027636 seconds (5.37 k allocations: 275.713 KiB, 96.65% compilation time)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 1 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "bdFRKq9rmCJm"
      },
      "source": [
        "### 4. Ortholog group\n",
        "\n",
        "To identify the ortholog related to the enzyme of interest, the function link takes as input the string \"ko\", and the KEGG gene identifier."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 137
        },
        "id": "sqHLBoPXlcdo",
        "outputId": "8296533e-c7a1-49eb-e63b-05d456c35b43"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.639537 seconds (351 allocations: 82.141 KiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m1×2 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m Target ID    \u001b[0m\u001b[1m Source ID \u001b[0m\n",
              "     │\u001b[90m String       \u001b[0m\u001b[90m String    \u001b[0m\n",
              "─────┼─────────────────────────\n",
              "   1 │ mtr:25493984  ko:K01183"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>1×2 DataFrame</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">Target ID</th><th style = \"text-align: left;\">Source ID</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">mtr:25493984</td><td style = \"text-align: left;\">ko:K01183</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& Target ID & Source ID\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & mtr:25493984 & ko:K01183 \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 69
        }
      ],
      "source": [
        "@time kegg_ko = KEGGAPI.link(\"ko\", \"mtr:25493984\")\n",
        "DataFrame(\n",
        "  kegg_ko.data,\n",
        "  kegg_ko.colnames\n",
        ")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "F7qyez2zm63f"
      },
      "source": [
        "### 5. Reaction(s) catalyzed by gene of interest\n",
        "\n",
        "To obtain the reactions associated to a gene, and a KEGG orthogroup, the input of the \"link\" function are the string \"reaction\", and KEGG ortholog number as \"KXXXXX\"."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 158
        },
        "id": "p9yYxzWIlnJk",
        "outputId": "cc3c76f4-006e-4fd3-9ee7-0833e2754ca5"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.170478 seconds (219 allocations: 12.547 KiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m2×2 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m Target ID \u001b[0m\u001b[1m Source ID \u001b[0m\n",
              "     │\u001b[90m String    \u001b[0m\u001b[90m String    \u001b[0m\n",
              "─────┼──────────────────────\n",
              "   1 │ ko:K01183  rn:R01206\n",
              "   2 │ ko:K01183  rn:R02334"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>2×2 DataFrame</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">Target ID</th><th style = \"text-align: left;\">Source ID</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">rn:R01206</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">2</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">rn:R02334</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& Target ID & Source ID\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & ko:K01183 & rn:R01206 \\\\\n\t2 & ko:K01183 & rn:R02334 \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 70
        }
      ],
      "source": [
        "@time kegg_reaction = KEGGAPI.link(\"reaction\", \"K01183\")\n",
        "DataFrame(\n",
        "  kegg_reaction.data,\n",
        "  kegg_reaction.colnames\n",
        ")"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### 6. Reaction information\n",
        "\n",
        "To obtain the reactions information, the \"kegg_get\" function requires an array of reaction KEGG identifier as \"rn:RXXXXX\""
      ],
      "metadata": {
        "id": "2dl_tODz-Plh"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "OfzKtE12sHZv",
        "outputId": "80d569a1-4247-4cae-8695-6d1bd0880282"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.136026 seconds (224 allocations: 16.945 KiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "2-element Vector{String}:\n",
              " \"ENTRY       R01206             \" ⋯ 527 bytes ⋯ \" endochitinase B [EC:3.2.1.14]\"\n",
              " \"ENTRY       R02334             \" ⋯ 480 bytes ⋯ \" endochitinase B [EC:3.2.1.14]\""
            ]
          },
          "metadata": {},
          "execution_count": 178
        }
      ],
      "source": [
        "@time kegg_reaction_info = KEGGAPI.kegg_get(kegg_reaction.data[2])\n",
        "kegg_reaction_info[2]"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "6CRIqMnlUocC"
      },
      "source": [
        "### 7. Pathway(s) catalyzed by gene of interest\n",
        "\n",
        "To obtain pathways associated to a gene the input of the \"link\" function are the string \"pathway\", and KEGG gene identifier."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 158
        },
        "id": "N5RsRiXGU0Yk",
        "outputId": "ae130018-df16-4638-81a8-a196fd48520d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.180780 seconds (220 allocations: 12.594 KiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m2×2 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m Target ID    \u001b[0m\u001b[1m Source ID     \u001b[0m\n",
              "     │\u001b[90m String       \u001b[0m\u001b[90m String        \u001b[0m\n",
              "─────┼─────────────────────────────\n",
              "   1 │ mtr:25493984  path:mtr00520\n",
              "   2 │ mtr:25493984  path:mtr01100"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>2×2 DataFrame</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">Target ID</th><th style = \"text-align: left;\">Source ID</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">mtr:25493984</td><td style = \"text-align: left;\">path:mtr00520</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">2</td><td style = \"text-align: left;\">mtr:25493984</td><td style = \"text-align: left;\">path:mtr01100</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& Target ID & Source ID\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & mtr:25493984 & path:mtr00520 \\\\\n\t2 & mtr:25493984 & path:mtr01100 \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 119
        }
      ],
      "source": [
        "@time kegg_pathways = KEGGAPI.link(\"pathway\", \"mtr:25493984\")\n",
        "DataFrame(\n",
        "  kegg_pathways.data,\n",
        "  kegg_pathways.colnames\n",
        ")"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### 8. Obtain pathway information\n",
        "\n",
        "To collect information about a pathway, the function find requieres as input the string \"pathway\" and the KEGG pathway identifer as \"path:mapXXXXX\"."
      ],
      "metadata": {
        "id": "BC5c3R47yg8Q"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 137
        },
        "id": "lpNGpuKXvz2J",
        "outputId": "565f8980-7db4-4edf-c7b7-35fb1c61b6d2"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.218293 seconds (218 allocations: 12.688 KiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m1×2 DataFrame\u001b[0m\n",
              "\u001b[1m Row \u001b[0m│\u001b[1m ID            \u001b[0m\u001b[1m Pathway                           \u001b[0m\n",
              "     │\u001b[90m String        \u001b[0m\u001b[90m String                            \u001b[0m\n",
              "─────┼──────────────────────────────────────────────────\n",
              "   1 │ path:map00520  Amino sugar and nucleotide sugar…"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>1×2 DataFrame</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">ID</th><th style = \"text-align: left;\">Pathway</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">path:map00520</td><td style = \"text-align: left;\">Amino sugar and nucleotide sugar metabolism</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& ID & Pathway\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & path:map00520 & Amino sugar and nucleotide sugar metabolism \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 130
        }
      ],
      "source": [
        "@time kegg_pathway_find = KEGGAPI.find(\"pathway\", \"path:map00520\")\n",
        "DataFrame(\n",
        "  kegg_pathway_find.data,\n",
        "  kegg_pathway_find.colnames\n",
        ")"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### 9. Download pathway of interest.\n",
        "\n",
        "The get_image function is to download a any image, the imput is the pathway number as path:mapXXXXX\n",
        "\n",
        "The save_image function is to save the figure in a png file. The input is a string wiht the name of the file and the extension \".png\""
      ],
      "metadata": {
        "id": "NRBETF35zJvd"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5tloIfcrADrB",
        "outputId": "da65e86c-ca62-4505-a44f-cbdd75d65c11"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.570293 seconds (309 allocations: 360.461 KiB)\n",
            "  0.016000 seconds (10.67 k allocations: 570.570 KiB, 97.55% compilation time)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\"aminoacid.png\""
            ]
          },
          "metadata": {},
          "execution_count": 100
        }
      ],
      "source": [
        "# save figure pathway\n",
        "@time kegg_image = KEGGAPI.get_image(\"path:map00520\")\n",
        "@time KEGGAPI.save_image(kegg_image, \"aminoacid.png\")"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### 10. Visualize saved pathway"
      ],
      "metadata": {
        "id": "ZP3H-DB14Alj"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "7lH90PuIAuCe"
      },
      "outputs": [],
      "source": [
        "Pkg.add(\"TestImages\")\n",
        "Pkg.add(\"Images\")\n",
        "Pkg.add(\"FileIO\")\n",
        "Pkg.add(\"Colors\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "kjwejygRBPxV"
      },
      "outputs": [],
      "source": [
        "using Images, TestImages, Colors\n",
        "img = load(\"aminoacid.png\")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "59081rvC_sxD"
      },
      "source": [
        "### 11. Ortholog genes\n",
        "\n",
        "Identify all genes related to the KEGG ortholog group using the link function. The input is the string \"genes\" and the KEGG ortholog group as \"KXXXXX\"."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 662
        },
        "id": "l5dj0_xe_f6l",
        "outputId": "573a9678-e330-4b9c-c686-318e0af11991"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  2.599635 seconds (58.20 k allocations: 7.468 MiB)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "\u001b[1m11522×2 DataFrame\u001b[0m\n",
              "\u001b[1m   Row \u001b[0m│\u001b[1m Target ID \u001b[0m\u001b[1m Source ID        \u001b[0m\n",
              "       │\u001b[90m String    \u001b[0m\u001b[90m String           \u001b[0m\n",
              "───────┼─────────────────────────────\n",
              "     1 │ ko:K01183  hsa:27159\n",
              "     2 │ ko:K01183  hsa:1118\n",
              "     3 │ ko:K01183  ptr:457641\n",
              "     4 │ ko:K01183  ptr:457114\n",
              "     5 │ ko:K01183  pps:100977638\n",
              "     6 │ ko:K01183  pps:100991992\n",
              "     7 │ ko:K01183  ggo:101143527\n",
              "     8 │ ko:K01183  ggo:101149661\n",
              "     9 │ ko:K01183  ggo:101149317\n",
              "    10 │ ko:K01183  pon:100436700\n",
              "    11 │ ko:K01183  pon:100447861\n",
              "   ⋮   │     ⋮             ⋮\n",
              " 11513 │ ko:K01183  hlo:J0X27_09085\n",
              " 11514 │ ko:K01183  hlo:J0X27_09090\n",
              " 11515 │ ko:K01183  nbg:DV706_00185\n",
              " 11516 │ ko:K01183  nbg:DV706_17085\n",
              " 11517 │ ko:K01183  nas:GCU68_15905\n",
              " 11518 │ ko:K01183  nkr:NKOR_05530\n",
              " 11519 │ ko:K01183  niw:Nisw_02425\n",
              " 11520 │ ko:K01183  nip:NsoK4_05940\n",
              " 11521 │ ko:K01183  nic:DSQ20_02680\n",
              " 11522 │ ko:K01183  niu:DSQ19_04165\n",
              "\u001b[36m                   11501 rows omitted\u001b[0m"
            ],
            "text/html": [
              "<div><div style = \"float: left;\"><span>11522×2 DataFrame</span></div><div style = \"float: right;\"><span style = \"font-style: italic;\">11497 rows omitted</span></div><div style = \"clear: both;\"></div></div><div class = \"data-frame\" style = \"overflow-x: scroll;\"><table class = \"data-frame\" style = \"margin-bottom: 6px;\"><thead><tr class = \"header\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">Row</th><th style = \"text-align: left;\">Target ID</th><th style = \"text-align: left;\">Source ID</th></tr><tr class = \"subheader headerLastRow\"><th class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\"></th><th title = \"String\" style = \"text-align: left;\">String</th><th title = \"String\" style = \"text-align: left;\">String</th></tr></thead><tbody><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">1</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">hsa:27159</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">2</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">hsa:1118</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">3</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">ptr:457641</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">4</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">ptr:457114</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">5</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">pps:100977638</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">6</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">pps:100991992</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">7</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">ggo:101143527</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">8</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">ggo:101149661</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">9</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">ggo:101149317</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">10</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">pon:100436700</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">pon:100447861</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">12</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">pon:100437818</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">13</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">nle:100600050</td></tr><tr><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td><td style = \"text-align: right;\">&vellip;</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11511</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">npl:FGF80_09185</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11512</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">haly:HYG82_13115</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11513</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">hlo:J0X27_09085</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11514</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">hlo:J0X27_09090</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11515</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">nbg:DV706_00185</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11516</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">nbg:DV706_17085</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11517</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">nas:GCU68_15905</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11518</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">nkr:NKOR_05530</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11519</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">niw:Nisw_02425</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11520</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">nip:NsoK4_05940</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11521</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">nic:DSQ20_02680</td></tr><tr><td class = \"rowNumber\" style = \"font-weight: bold; text-align: right;\">11522</td><td style = \"text-align: left;\">ko:K01183</td><td style = \"text-align: left;\">niu:DSQ19_04165</td></tr></tbody></table></div>"
            ],
            "text/latex": "\\begin{tabular}{r|cc}\n\t& Target ID & Source ID\\\\\n\t\\hline\n\t& String & String\\\\\n\t\\hline\n\t1 & ko:K01183 & hsa:27159 \\\\\n\t2 & ko:K01183 & hsa:1118 \\\\\n\t3 & ko:K01183 & ptr:457641 \\\\\n\t4 & ko:K01183 & ptr:457114 \\\\\n\t5 & ko:K01183 & pps:100977638 \\\\\n\t6 & ko:K01183 & pps:100991992 \\\\\n\t7 & ko:K01183 & ggo:101143527 \\\\\n\t8 & ko:K01183 & ggo:101149661 \\\\\n\t9 & ko:K01183 & ggo:101149317 \\\\\n\t10 & ko:K01183 & pon:100436700 \\\\\n\t11 & ko:K01183 & pon:100447861 \\\\\n\t12 & ko:K01183 & pon:100437818 \\\\\n\t13 & ko:K01183 & nle:100600050 \\\\\n\t14 & ko:K01183 & nle:100603319 \\\\\n\t15 & ko:K01183 & nle:100600588 \\\\\n\t16 & ko:K01183 & hmh:116467948 \\\\\n\t17 & ko:K01183 & hmh:116468357 \\\\\n\t18 & ko:K01183 & hmh:116457378 \\\\\n\t19 & ko:K01183 & mcc:703286 \\\\\n\t20 & ko:K01183 & mcc:703284 \\\\\n\t21 & ko:K01183 & mcc:705382 \\\\\n\t22 & ko:K01183 & mcf:102117287 \\\\\n\t23 & ko:K01183 & mcf:102138420 \\\\\n\t24 & ko:K01183 & mcf:102117900 \\\\\n\t25 & ko:K01183 & mthb:126929805 \\\\\n\t26 & ko:K01183 & mthb:126939002 \\\\\n\t27 & ko:K01183 & mthb:126950786 \\\\\n\t28 & ko:K01183 & mni:105479216 \\\\\n\t29 & ko:K01183 & mni:105479217 \\\\\n\t30 & ko:K01183 & mni:105484764 \\\\\n\t$\\dots$ & $\\dots$ & $\\dots$ \\\\\n\\end{tabular}\n"
          },
          "metadata": {},
          "execution_count": 168
        }
      ],
      "source": [
        "@time kegg_ko_genes = KEGGAPI.link(\"genes\", \"K01183\")\n",
        "DataFrame(\n",
        "  kegg_ko_genes.data,\n",
        "  kegg_ko_genes.colnames\n",
        ")"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "### 12. Save to file ortholog genes sequence for downstream analysis."
      ],
      "metadata": {
        "id": "NSnTR3h_0cMD"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### 12.1 Get nucleotide sequence and save to fasta file\n",
        "\n",
        "With the \"kegg_get\" function user can get nucleotide sequence of one or more gene using an array with KEGG protein id and the string \"ntseq\".\n",
        "\n",
        "The output of the function can be save to file using the function FastaWriter from the FastaOI package."
      ],
      "metadata": {
        "id": "DfBpGvgxWPeh"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "# Nucleotide sequence\n",
        "@time kegg_ntseq = KEGGAPI.kegg_get(kegg_ko_genes.data[2][1:50], \"ntseq\")\n",
        "\n",
        "@time FastaWriter(\"MSA_ntseq.fasta\") do fw\n",
        "    for ch in kegg_ntseq[2]\n",
        "        write(fw, ch)\n",
        "    end\n",
        "end"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "azwkzYxsVXxD",
        "outputId": "66477b4f-7602-461f-b30c-c3ade1b71734"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  1.822281 seconds (1.47 k allocations: 639.531 KiB)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 1 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 2 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 3 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 4 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 5 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 6 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 7 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 8 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 9 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 10 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 11 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 12 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 13 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 14 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 15 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 16 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 17 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 18 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 19 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 20 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 21 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 22 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 23 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 24 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 25 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 27 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 28 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 29 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 30 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 31 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 32 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 34 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 35 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 36 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 37 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 38 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 39 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 40 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 41 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 42 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 43 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 44 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 45 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 46 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 47 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 48 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 49 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 50 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.040468 seconds (13.13 k allocations: 853.924 KiB, 59.39% compilation time)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "#### 12.2 Get amino acid sequence and save to fasta file\n",
        "\n",
        "With the \"kegg_get\" function user can get amino acid sequence of one or more gene using an array with KEGG protein id and the string \"aaseq\".\n",
        "The output of the function can be save to file using the function FastaWriter from the FastaOI package."
      ],
      "metadata": {
        "id": "-UjsPRk6WhYJ"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "# Nucleotide sequence\n",
        "@time kegg_aaseq = KEGGAPI.kegg_get(kegg_ko_genes.data[2][1:50], \"aaseq\")\n",
        "\n",
        "@time FastaWriter(\"MSA_aaseq.fasta\") do fw\n",
        "    for ch in kegg_aaseq[2]\n",
        "        write(fw, ch)\n",
        "    end\n",
        "end"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xCPEVmyaWrhc",
        "outputId": "cde2427b-7b5b-427f-af2f-6e955d927564"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  1.354897 seconds (1.33 k allocations: 197.695 KiB)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 1 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 2 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 3 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 4 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 5 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 6 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 7 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 8 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 9 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 10 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 11 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 12 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 13 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 14 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 15 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 16 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 17 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 18 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 19 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 20 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  0.032210 seconds (13.14 k allocations: 824.627 KiB, 75.70% compilation time)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 21 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 22 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 23 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 24 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 25 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 27 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 28 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 29 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 30 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 31 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 32 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 34 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 35 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 36 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 37 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 38 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 39 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 40 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 41 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 42 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 43 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 44 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 45 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 46 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 47 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 48 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 49 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n",
            "\u001b[33m\u001b[1m┌ \u001b[22m\u001b[39m\u001b[33m\u001b[1mWarning: \u001b[22m\u001b[39mdescription line longer than 80 characters (entry 50 of FASTA input)\n",
            "\u001b[33m\u001b[1m└ \u001b[22m\u001b[39m\u001b[90m@ FastaIO ~/.julia/packages/FastaIO/LA9pk/src/FastaIO.jl:446\u001b[39m\n"
          ]
        }
      ]
    }
  ],
  "metadata": {
    "colab": {
      "provenance": [],
      "authorship_tag": "ABX9TyPUD+PmLP1wimAvFEVybm0h",
      "include_colab_link": true
    },
    "kernelspec": {
      "display_name": "julia 1.8.2",
      "name": "julia"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}